%=========================================
%
% Tom Jenkins PhD Thesis 2018 Written in LaTeX
%
%=========================================

% Document type
\documentclass[a4paper,twoside,12pt]{article}

%----------
% Preamble
%----------

%% Basic preamble
\usepackage{amsmath} % contains advanced math extensions for LaTeX
\usepackage{geometry}
\geometry{left=30mm, % left and right margins
          top=20mm} % top and bottom margins
\usepackage[onehalfspacing]{setspace} % spacing
\usepackage{graphicx} % manage external pictures
\usepackage{pdfpages} % Add PDFs to document
\raggedbottom % No space between subsection and text
\usepackage{afterpage}

% Arial font (this code works!) http://cambio.name/personal/content/how-use-arial-font-latex
\renewcommand{\rmdefault}{phv}
\renewcommand{\sfdefault}{phv}
\usepackage[T1]{fontenc} %\textsterling

%% Referencing
\usepackage{natbib} % additional citation options and styles
\bibpunct{(}{)}{;}{a}{}{} % (Jones et al. 2000) citation

% Figure settings
\usepackage{chngcntr}
\usepackage{caption}
\captionsetup[figure]{labelfont=bf, font=small} % label 'figure x:' in bold
  % \makeatletter % figure at top of page
  % \setlength{\@fptop}{0pt} 
  % \makeatother
\usepackage{subfigure} % enable use of 'subfigure'
  
% Table settings
\usepackage{longtable}
\captionsetup[table]{singlelinecheck=false} % Place table caption on the left
\captionsetup[table]{labelfont=bf, font=small} % label 'table x:' in bold
\usepackage{multirow} % formatting abnormal table arrangements
\usepackage{booktabs, fixltx2e} % package to place footnotes on table ; \multicolumn{ncol}{centering/right/left}
\usepackage[flushleft]{threeparttable} % footnote beneath table
\usepackage{pdflscape} % landscape pages in pdf

% Define new table column
\usepackage{array}
\newcolumntype{L}{>{\raggedright\arraybackslash}p{5cm}} % left align 5cm
\newcolumntype{R}{>{\raggedright\arraybackslash}p{7.5cm}} % left align 7.5cm
\newcolumntype{x}{>{\centering\arraybackslash}p{1cm}} % centre align 1cm
\usepackage{color, colortbl} % enable colouring rows and columns
\definecolor{Grey}{gray}{0.95} % grey

%% Import csv file data
\usepackage{csvsimple}

%% Removes badboxing warnings
\hfuzz=20pt
\vfuzz=20pt
\hbadness=2000
\vbadness=\maxdimen

%% Justified text
\tolerance=1
\emergencystretch=\maxdimen
\hyphenpenalty=10000
\hbadness=10000

%% Hyperlinks for table of contents and text
\usepackage{hyperref}
  \hypersetup{
    colorlinks,
    citecolor=black,
    filecolor=black,
    linkcolor=black,
    urlcolor=black}

%% Table of contents settings
\setcounter{secnumdepth}{0} % sections not labelled as numbers but are included in Table of Contents
\renewcommand*\contentsname{Table of Contents} % Edit content name

%%
\usepackage[T1]{fontenc} % access \textquotedbl
\usepackage{textcomp}    % access \textquotesingle

%% Package allows track changes to document
% \usepackage{changes}
\usepackage[final]{changes}
\definechangesauthor[name={Tom}, color=blue]{Tom}
% \added[id=Tom, comment={Write comment here}]{Write new text here}

%-----------------
% End of preamble
%-----------------


%=================
%
% Start of thesis
%
%=================
\begin{document}


%------------
% Title Page
%------------
\begin{titlepage}

\begin{figure}[h!]
  \centering
  \includegraphics[width=2.2in, height=1in]{Chapter_figures/General_intro/Exeter_logo.jpg}
\end{figure}

\begin{center}
  % \Large \textbf{Marine Protected Area network: selecting appropriate taxa and assessing genetic connectivity in two benthic marine invertebrates} \\
    % \Large \textbf{Population genetic structure in two benthic invertebrates: implications for the designation and connectivity of Marine Protected Areas} \\
\Large \textbf{Connectivity between MPAs: selecting appropriate taxa and assessing genetic connectivity in two benthic marine invertebrates}

\vspace{120pt}
\large Submitted by Thomas Lowther Jenkins to the University of Exeter as a thesis for the degree of Doctor of Philosophy in Biological Sciences, October 2018. \\
\end{center}

\vspace{108pt}
\begin{flushleft}
\normalsize This thesis is available for Library use on the understanding that it is copyright material and that no quotation from the thesis may be published without proper acknowledgement. \\
  \vspace{12pt}  
  I certify that all material in this thesis which is not my own work has been identified and that no material has previously been submitted and approved for the award of a degree by this or any other University. \\
  \vspace{36pt}  
(Signature) ..............................................................................................
\end{flushleft}

\thispagestyle{plain} % add page number to title page
\end{titlepage}


%----------
% Abstract
%----------
\newpage
\section{Abstract}
\setcounter{page}{3} % Start page numbering here
Connectivity is fundamental for the persistence of many populations of marine species and is formally identified as one of five key principles for designing an ecologically coherent network of \replaced{Marine Protected Areas (MPAs)}{MPAs} in European waters.  However, the process of assessing connectivity between MPAs, and which taxa to include in assessments of connectivity, is challenging.  Managers of MPAs have typically concentrated their efforts on species that are endangered or rare, or on so-called `umbrella', `keystone' or `flagship' species; however, these species may not always be the best candidates for assessing connectivity of a MPA network.  In this thesis, a meta-analysis was firstly conducted to study genetic patterns across a broad range of coastal marine taxa in the northeast Atlantic.  This meta-analysis provided insights into the biological and methodological information needed to ascertain which taxa may be considered as good candidates for assessing genetic connectivity between MPAs across Britain and the wider northeast Atlantic.  The knowledge gained from this literature survey facilitated the design of a set of criteria that identified ideal traits of a candidate species for assessments of genetic connectivity between MPAs; subsequently, based on these criteria, two species were selected to assess connectivity between MPAs in the British network: the pink sea fan (\textit{Eunicella verrucosa}) and the European lobster (\textit{Homarus gammarus}).  Using 13 microsatellites and 3,743 SNPs, the results for the pink sea fan indicated the presence of three distinct genetic groups, partitioned between sites from western Ireland, southern Portugal and Britain-France.  For the European lobster, 86 SNPs indicated strong genetic differentiation between the northeast Atlantic, the middle Mediterranean and the eastern Mediterranean (Aegean Sea).  In addition, there was a pronounced genetic cline across the northeast Atlantic, suggesting that connectivity in the European lobster follows a stepping-stone model of dispersal, which was supported by simulations of larval dispersal. Taken together, the results from these two studies suggests that the MPA network in Britain is sufficient to maintain connectivity in the pink sea fan and the European lobster, and possibly other species living in comparable habitats with similar life histories and dispersal traits.  Moreover, the criteria applied in this thesis to select species appears to facilitate the identification of ideal surrogate taxa to assess connectivity between MPAs, which could easily be applied to assessments of MPA network connectivity in other seas and oceans around the world.

%-------------------
% Table of Contents
%-------------------
\newpage
\phantomsection
\addcontentsline{toc}{section}{Table of Contents}
\tableofcontents

%-------------------
% List of Figures
%-------------------
\newpage
\phantomsection
\addcontentsline{toc}{section}{List of Figures}
\listoffigures

%-------------------
% List of Tables
%-------------------
\phantomsection
\addcontentsline{toc}{section}{List of Tables}
\listoftables

%-------------------
% List of changes
%-------------------
% \newpage
% \phantomsection
% \addcontentsline{toc}{section}{List of Changes}
% \listofchanges

%-------------------
% Acknowledgements
%-------------------
\newpage
\section{Acknowledgements}
I know it is a cliche in these acknowledgement sections, but there are so many people who have helped to make this thesis a reality. Collaboration is, to me, one of the main traits of successful science, and this has been central to each component of research carried out in this thesis. This is particularly relevant to all of the people from across the UK and Europe who kindly gave up their time to collect tissue samples of lobster for me, and for my colleagues and friends who trained me in wet-lab molecular biology techniques.

Before I dive into the plethora of acknowledgements, I'd like to start by thanking the primary funders of my PhD, the NERC GW4+ DTP, the University of Exeter and Natural England (in particular Emma Dade and James Highfield), for which this PhD would not have been possible without your support. In addition, a special thank you to the Genetics Society and the British Ecological Society who have generously supported my attendence to several conferences and training workshops, and extra fieldwork opportunities, which has enhanced both my PhD and my personal development as an independent researcher.

Secondly, I would like to thank my PhD supervisor, Jamie Stevens, for initially entrusting me with this PhD! And for being an incredibly dedicated and motivating supervisor; our insightful discussions have always made me think critically about the science. Also, thank you to all of the members of the Molecular Ecology and Evolution Group (MEEG) for your ideas and advice in and out of the lab. In particular, I would like to thank Josie Paris for your help and support in bioinformatics and Guy Freeman for the exceptional species illustrations used in some of the figures in this thesis. In addition, a massive thanks to Lauren Ames and Graham Thomas for passing on your wet-lab tips and tricks -- they won't be forgotten! And a thank you to Rita Castilho, whose knowledge and expertise was extremely valuable for analysing, interpreting and publishing data from chapter two.

Thirdly, and most importantly for the lobster project, a huge thank you to all the people who kindly collected or provided me with lobster tissue samples for genetic analysis: James Wood (Bridlington), Jonas Seafood Ltd (Cromer), D.R.Collin \& Sons (Eyemouth), Donald MacLennan (Outer Hebrides), Jack Emmerson (Isle of Man), W Harvey \& Sons (Isles of Scilly), Sion Williams (Llyn Peninsula), Dawson Shearer and Matt Coleman (Orkney), Charlie Ellis (Padstow), DASH Shellfish (Pembrokeshire), Mark Hamilton (Shetland), Monteum Ltd (Shoreham-by-sea), Douglas Craigie (Sula Sgeir), Peter Gay and Don Thompson (Channel Islands), Fabienne Legall (Île de Ré), Roland Krone (Helgoland), Owen Doyle (Arranmore Island and Donegal Bay), Ron Randalls (Cork Harbour), John Hickey (Hook Peninsula), Majbritt Bolton-Warberg (Kilkieran Bay), Declan Nee (Mullet Peninsula), Vera O'Donovan (Ventry), Alexandros Triantafyllidis (Greece), Sara Saba and Ulisse Murru (Sardinia), Tonje Knutsen (Flodevigen and Tvedestrand), Carl Andre (Gullmarfjord, Kavra and Singlefjord), Ellika Faust (Lysekill) and Gonzalo Perez Benavente (Vigo). This study would not have been possible without you. I would like to especially thank the team at the National Lobster Hatchery for your support and enthusiasm, particularly Charlie Ellis for helping me collect lobster samples and for providing insightful ideas and comments for chapter six.  

Finally, I never would have got this far if it wasn't for my friends, girlfriend, family, coffee and Marvel films. First, a massive shout out to past and present members of coffee club -- relinquishing my admin duties and deciding a successor will be difficult, but I trust the true heir will be revealed at some point. Also, a shout out to my friend and fellow PhD submitter, Matt Jones, whose competitive FIFA duals and banter have kept me sane. Second, thank you to Abbi, who has continued to stick with me over the last few years, and particularly the last several months! Lastly, to my amazing family, Mum, Dad and Bubus, who have supported me throughout this perpetual student phase, even when you've had no idea what I'm talking about! Spending weekends at home and Facetiming in the evenings has kept me going and I can't thank you enough for your support.



%---------------
% Abbreviations
%---------------
\newpage
\section{Abbreviations}
% \csvreader[no head, tabbing]{Tables/Abbreviations.csv}{1=\m,2=\p,3=\n}{\m & \p & \n}

\begin{tabular}{p{2cm}l}
AU & Adaptive unit \\ 
BAP & Biodiversity Action Plan \\
BIM & Bord Lascaigh Mhara \\
bp & Base pair \\
CBD & Convention on Biological Diversity \\
CU & Conservation unit \\
DAPC & Discriminant Analysis of Principal Components \\
DEFRA & Department for Environment Food and Rural Affairs \\
DNA & Deoxyribonucleic acid \\
EEZ & Exclusive economic zone \\
ESU & Evolutionary significant unit \\
HWE & Hardy-Weinberg Equilibrium \\
IBD & Isolation-by-distance \\
IFC & Integrated Fluidic Circuit \\
IFCA & Inshore Fisheries and Conservation Authorities \\
IUCN & International Union for Conservation of Nature \\
IUU & Illegal unreported and regulated fishing \\
JNCC & Joint Nature Conservation Committee \\
Ka & Thousand years ago \\
LD & Linkage disequilibrium \\
LFU & Lobster Fishery Unit \\
LGM & Last Glacial Maximum \\
Ma & Million years ago \\
MCMC & Markov chain Monte Carlo \\
MCZ & Marine Conservation Zone \\
MPA & Marine Protected Area \\
MSFD & Marine Strategy Framework Directive \\
mtDNA & Mitochondrial DNA \\
MU & Management unit \\
NCMPA & Nature Conservation Marine Protected Area \\
NE & Natural England \\
NERC & Natural Environment Research Council \\
NTZ & No Take Zone \\
OSPAR & Oslo/Paris Convention \\
PCR & Polymerase chain reaction \\
PLD & Pelagic larval duration \\


\end{tabular}

\newpage
\begin{tabular}{p{2cm}l}
RADseq & Restriction-site associated DNA sequencing \\
RNA & Ribonucleic acid \\
SAC & Special Areas of Conservation \\
SNP & Single nucleotide polymorphism \\
SPA & Special Protected Area \\
UK & United Kingdom \\
\end{tabular}


%-------------
%
% Chapter 1: General Introduction
% 
%-------------
\newpage
\section{Chapter 1: General introduction}
\rule{\textwidth}{0.8pt}
\vspace{1pt}

\noindent Understanding the biology, ecology and evolution of marine fauna and flora has never been more readily achievable.  Previously, scientific knowledge of marine organisms routinely trailed their terrestrial counterparts, which likely arose from the obvious obstacles associated with observing, sampling and studying organisms in marine systems.  However, decades of technological advances now permit the exploration of many marine and estuarine environments, substantially narrowing the knowledge gap between marine and terrestrial systems.

The study of connectivity is one such discipline that has benefited from these technological advances.  Connectivity research links a number of different fields in ecology and evolution including animal behaviour, population dynamics, genetic structure analysis and adaptation to local environments \citep{Kool2013}.  The discipline has also emerged as a key component of applied science and conservation; for example, understanding connectivity has been vital for designing networks of protected areas \citep{Jones2007}, tracking pathways of invasive species \citep{Perez-Portela2012}, managing fisheries resources \citep{Fogarty2007,Kough2013}, and monitoring the effects of climate change \citep{Gerber2014}.

This thesis takes advantage of the recent developments in both marine connectivity and DNA sequence technology to investigate patterns of genetic diversity and connectivity in two coastal marine invertebrates.  Although this PhD ultimately aims to generate novel population genetic data for the pink sea fan (\textit{Eunicella verrucosa}) and the European lobster (\textit{Homarus gammarus}), the overarching theme of this thesis is steered towards informing marine conservation and management.  Therefore, as well as exploring the population biology and ecology of these two species, a central premise is to integrate and translate the findings into usable forms of evidence that can inform practitioners in the fields of marine protected area designation and implementation and fisheries management.  


%%% 1.1 Marine connectivity
\subsection{1.1 Marine connectivity}
Marine connectivity is the study of dispersal and immigration between populations of marine organisms. In particular, it has been defined as the extent to which populations inhabiting different parts of a species' range are linked by the movement of eggs, propagules, larvae, juveniles or adults \citep{Palumbi2003}.  High connectivity between two populations implies that many individuals are exchanged between these populations, whereas limited or no exchanges implies restricted or no connectivity. This definition, however, does not consider that an individual may disperse to a new population but die upon (or shortly after) reaching its destination.  Realised connectivity (also used interchangeably with functional or effective connectivity) is where an individual successfully navigates across habitat patches and concludes in successful settlement and reproduction in the new population \citep{Watson2010,Kool2013}. \citet{Lowe2010} also distinguished demographic and genetic connectivity \hyperref[section1.1.4]{(section 1.1.4)}, which essentially identifies and explores the influence of, and links between, short temporal scale population dynamics and large-scale evolutionary processes \citep{Hidalgo2017}. 

Marine connectivity is fundamental for the persistence of many populations of marine species.  It is particularly important for sessile and non-motile marine fauna that rely primarily on ocean currents to disperse to nearby or distant populations \citep{Cowen2009}.  This ability to travel potentially vast distances via ocean currents likely explains the large ranges associated with many marine fauna; it also increases the capacity for species to expand their range by colonising new habitat patches that become available.  Indeed, connectivity can be extremely important for the long-term stability of meta-populations and for recolonising habitats that have experienced local extirpation \citep{Hanski1998}.  Hence, one of the aims of marine conservation is to establish marine reserves of adequate size and spacing to help maintain natural connectivity by reducing human disturbance in key or vulnerable areas \citep{Fogarty2007,Botsford2009}.

Populations persist when self-recruitment and immigration equal or exceed mortality and emigration \citep{Cowen2009}.  Populations are sinks when the net import of individuals is greater than the net export of individuals; conversely, source populations export more individuals than they receive.  Sinks that have low local recruitment (i.e. low survival and birth rates of residents) are assumed to benefit considerably from immigrant subsidy to maintain stable populations \citep{Runge2006}.  This is unlikely the case for pseudo-sinks, however, which can be independently viable populations, but appear as `true' sinks because asymmetrical immigration can depress fecundity or increase mortality because of density-dependent processes \citep{Watkinson1995}.  Yet, distinguishing pseudo-sink populations from genuine sinks is seldom straightforward.  Nevertheless, identifying source and sink populations can be very important for designing networks of protected areas \citep{Jones2007} and for the spatial management of marine fisheries \citep{Fogarty2007}.  


\subsubsection{1.1.1 Models of connectivity}
Simple but logical models attempt to conceptualise connectivity (Fig. \ref{connectivitymodels}), forming a theoretical baseline for investigating connectivity in natural populations.  The three most widely considered models are the island model \citep[Fig. \ref{connectivitymodels}A;][]{Wright1932}, the hierarchical island model \citep[Fig. \ref{connectivitymodels}B;][]{Slatkin1991} and the stepping-stone model \citep[Fig. \ref{connectivitymodels}B;][]{Kimura1964}.  In the island model, all populations are connected via a constant migration rate, whereas in the hierarchical island model, a group of populations exchange migrants with each other at a much higher rate compared with other populations or groups of populations \citep{Slatkin1991}.  In other words, in the hierarchical island model, connectivity within a subset of populations is higher compared with some other external populations.  In comparison to these two models, the stepping-stone model considers the spatial arrangement of populations and proposes that the migration rate is stronger between populations that are spatially closer together.  In effect, the model suggests that in each generation, an individual can migrate one `step' in any direction to an adjacent population, and then the process repeats itself in subsequent generations \citep{Kimura1964}.  This model is somewhat realistic for species with large ranges and is usually associated with isolation-by-distance (IBD), a term coined by Wright in the 1940s to describe the distribution of genetic variation over a geographic region \citep{Wright1943}. 

%% Figure: Models of connectivity 
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/General_intro/Models_of_Connectivity_Figure.pdf}
\caption [Models of connectivity]
{Three models of connectivity: (A) the island model, (B) the hierarchical island model and (C) the stepping-stone model.  Blue circles represent hypothetical populations.  Arrows denote migration rates; red arrows represent much less migration than grey arrows.}
\label{connectivitymodels}
\end{figure}
%% end of figure

For marine species with pelagic larval phases, the island model in essence describes a situation in which larvae are drawn from a well-mixed pool of larvae that have originated from multiple populations across the species geographical range.  This is likely due to high dispersal capacity of the larvae, coupled with adequate oceanography to facilitate mixing, which ultimately results in high connectivity over large spatial scales.  In comparison, the stepping-stone model describes a scenario in which the larval dispersal capacity is insufficient to travel from one habitat patch to a distant habitat patch in a single dispersal event.  Instead, larvae more often disperse to neighbouring populations which may lead to the development of IBD patterns, whereby connectivity is stronger between populations that are closer together than between populations that are further apart, resulting in higher genetic similarity between populations that are spatially closer to one another.  


\subsubsection{1.1.2 Drivers of connectivity}
The marine environment presents many opportunities for dispersal within and between populations.  Benthic marine organisms are typically sedentary or immobile as adults and, therefore, rely on the natural oceanography around them to disperse their eggs or larvae \citep{Cowen2009}.  During this pelagic larval phase, the larvae form a temporary local population in the plankton, and then drop out of the water column after a period of drifting to find suitable habitat for settlement.  In the early days, this high potential for long-distance dispersal led to the belief that marine populations were universally well-connected and `open' over ecological time scales \citep{Cowen2000}.  This was initially supported by a number of genetic studies that reported large homogenous populations over regional to basin-wide spatial scales \citep[][and references therein]{Cowen2007}.  However, recent research on many different marine taxa, and reconsideration of previous evidence using more modern genetic markers, suggests that limited dispersal and connectivity may be more prevalent in marine environments than previously thought \citep{Hauser2008,Hellberg2009}, challenging the paradigm that populations of marine species are universally open over ecological time scales.

The bipartite life cycle of many benthic marine organisms means that the total distance travelled during dispersal is primarily driven by environmental and ontogenic factors during the pelagic larval phase, which is highly variable across taxa and can last anywhere from hours to months \citep{Shanks2009}.  As eggs or larvae are expelled into the water column above the seafloor, they are immediately subjected to the velocity and direction of the ambient ocean currents. Larvae that have limited ability to swim against water masses are completely dependent on the ocean currents to disperse away from their natal origin.  Logically, this suggests that interspecific larvae released from the same habitat with a similar pelagic larval duration (PLD) may be expected to have equivalent dispersal potential.  However, because of the diverse life history strategies employed by marine organisms \hyperref[section1.1.3]{(section 1.1.3)}, and the influence of biotic (e.g. predation, spawning timing) and abiotic factors (e.g. habitat availability), dispersal distance and connectivity can vary substantially between species \citep{Bradbury2008,Cowen2009}.  For example, nutrient availability and sea temperatures play a critical role in dictating the PLD of some species \citep[e.g.][]{McCormick1995}, which could also mean that projected increases in sea temperature with global climate change may have tangible effects on future patterns of marine connectivity \citep{OConnor2007}.

A break in connectivity can occur when certain conditions restrict dispersal between populations.  For example, glaciers during the Pleistocene epoch isolated many terrestrial populations of northwest American taxa, preventing movement between populations \citep{Shafer2010}. In the marine environment, the formation of a land barrier would instantly prevent movement between populations either side of this barrier; such an example exists in Central America, the Isthmus of Panama (Fig. \ref{panama}), which separates the Pacific and Atlantic oceans.  The Isthmus of Panama formed around 2.8 million years ago (Ma) during the Cenozoic era \citep{ODea2016}.  This introduced a land barrier to marine species but interestingly removed a marine barrier to terrestrial species from North and South America (Fig. \ref{panama}); this meant that migration and gene flow was nullified in the marine realm but became possible in the terrestrial realm.  For marine species, this vicariant event resulted in reproductive isolation and independent evolution between populations of the two oceans, leading to gradual divergence over time and the beginnings of allopatric speciation in some taxa \citep{Lessios2008}.

%% Figure: Isthmus of Panama
\begin{figure}[h!]
\centering
\includegraphics[width=0.8\textwidth]{Chapter_figures/General_intro/Isthmus_of_Panamap.png}
\caption [The Isthmus of Panama]
{The Isthmus of Panama. The formation of the Isthmus of Panama introduced a land barrier to populations of marine species but removed a marine barrier to terrestrial species from North and South America. Image created by Andrew Z. Colvin under a CC BY-SA 4.0 licence.}
\label{panama}
\end{figure}
%% end of figure

More subtle barriers can also impede dispersal in the marine environment, such as eddies, fronts, deep-water and environmental gradients (e.g. salinity and temperature).  The magnitude of the effect of these barriers to dispersal and connectivity can vary depending on a species' life history and the strength of the barrier.  For example, the Almeria-Oran front in the Mediterranean Sea has been found to constrain connectivity in some, but not all, species \citep{Patarnello2007,Pascual2017}.  Moreover, deep-water has been suggested to hinder connectivity between the central and eastern Pacific in sedentary shallow water species whose larvae cannot stay in the plankton long enough to bridge the stretch of deep-water (Lessios 2012).  On the other hand, the circular currents of eddies may promote self-recruitment and connectivity between habitats located within the eddy system \citep{Sponaugle2005}.

Lastly, there are mounting reports of invasive species that have likely arisen from unnatural dispersal, which is possibly a consequence of intentional (e.g. aquaculture) or unintentional (e.g. lionfish in the Caribbean via ballast water) human-mediated translocations \citep{Lowry2013}.  Unfortunately, invasive species can have negative impacts on biodiversity, ecosystem functioning and ecosystem services \citep[e.g. lionfish,][]{Green2012}, which has led to the implementation of national and international programs to mitigate current and future impacts \citep{Pysek2010}.  From a conservation genetics perspective, human-mediated translocations can also make inferences about population structure difficult because it can produce abnormal phylogeographic patterns that no longer reflect natural processes \citep{Ni2012}.  In addition, continuing with the theme of human-mediated dispersal, there has also been an increase in `ocean sprawl' over the last few decades \citep{Firth2016}; ocean sprawl is the proliferation of artificial structures in the sea. These structures, including offshore marine renewable energy devices and wrecks, have been suggested to provide artificial stepping stones for marine fauna \citep{Krone2011,Adams2014}; however, this scheme is still in its infancy and more research is needed to quantify the effects (positive or negative) of ocean sprawl on ecological connectivity in the marine environment \citep{Bishop2017}.  In summary, when studying benthic marine organisms, it is also important to consider potential routes of anthropogenic dispersal and its possible impacts when exploring patterns of dispersal and connectivity.


\subsubsection{1.1.3 Dispersal, recruitment and life history strategies}
\label{section1.1.3}
Dispersal is defined by \citet{Cayuela2018} as the movement of individuals from their natal patch of birth to their first breeding site.  Effective dispersal is similarly defined, except that the disperser spends enough time at the new site to successfully reproduce and transmit its genes to the next generation.  In population genetic studies, ``dispersal'' and ``migration'' are often considered synonyms; however, \citet{Cayuela2018} highlight that these are two distinct concepts, in which dispersal (effective or non-effective) is usually a one-way process, whereas migration implicates recurrent, two-way movements.

Dispersal success can vary considerably across space and time as the marine environment is not uniform, but a highly dynamic heterogenous oceanic landscape (i.e. seascape) which presents challenges for dispersing individuals.  Benthic marine organisms have evolved a diverse array of life history strategies and behaviours (e.g. diel vertical migration) to deal with these challenges \citep{Levin2006}, which interact with oceanography to ultimately define dispersal success \citep{Cowen2009}.  Some modes of reproduction and development are, theoretically, expected to hamper or increase larval dispersal (Table \ref{dispersaltable}); the most common of these for benthic marine species are briefly discussed.

%% Table: Dispersal potential
\afterpage{
\begin{table}
\small
\caption[Life history strategies and expected dispersal potential]{Common life history strategies of benthic marine species and their expected effects on dispersal potential.}
\label{dispersaltable}
\begin{tabular}[l]{lp{9cm}p{2cm}}
\hline
\textbf{Strategy} & \textbf{Description} & \textbf{Dispersal potential}\\
% \textbf{} & \textbf{} & \textbf{} & \textbf{potential}\\
\hline
\textbf{Development} & & \\
Direct & Develop inside parents or from eggs. Larvae may look like adult form straight after hatching. & Low \\
Brooder & Eggs fertilised inside maternal adult. Larvae ejected and settle nearby. & Low \\
Surface brooder & Eggs fertilised inside maternal adult. Larvae ejected and are phototaxic. & Low \\ % but potential higher potential than a standard brooder
Broadcast spawner & Discharges gametes into the water column. & High \\[3pt]
\textbf{Pelagic phase} & & \\
No PLD & No pelagic larval duration.  & Low \\
Lecithotrophic & Disperse via ocean currents, provided a yolk sac for nutrition.  & Medium \\
Planktotrophic & Disperse via ocean currents, feed while in the plankton.  & High \\
\hline
\end{tabular}
\end{table}
}
%% end of table

Direct developing organisms lack a pelagic larval phase, with offspring emerging from the parent as larvae and then settling after expulsion, or offspring are devoid of a larval stage altogether and emerge as a miniature version of the adults.  For example, brooders release sperm (assumed to be negatively buoyant) which fertilise eggs inside the maternal adult and larvae are then ejected and settle nearby; this is the case for some sexually reproducing corals \citep[e.g. \textit{Corallium rubrum},][]{Ledoux2010a}.  A similar strategy, surface brooding, is where larvae show phototaxis behaviour (attraction or repulsion to light), whereby larvae swim towards or away from light at the ocean surface; such a strategy is adopted by larvae of the red gorgonian (\textit{Paramuricea clavata}), which display negative phototaxis and remain in suspension for only a few minutes before descending to the seafloor to settle \citep{Mokhtar-Jamai2011}.  A slightly different method of direct development occurs in oviparous skates and rays, in which the eggs are internally fertilised and egg cases are deposited on the seafloor where, after a period of incubation, fully developed young emerge \citep[e.g. \textit{Raja clavata},][]{Chevolot2006}.  In comparison to these direct strategies, broadcast spawners synchronously discharge their gametes into the water column, which are externally fertilised and then swept away by the surrounding currents \citep[e.g. the octocoral \textit{Eunicella verrucosa},][]{Munro2004}.  Because of the duration spent drifting in ocean currents, broadcast spawning is expected to have much higher dispersal potential than direct modes of development (Table \ref{dispersaltable}).  During the pelagic phase, the meroplanktonic larvae are either lecithotrophic (provided with a yolk sac) or planktotrophic (feed in the plankton); the latter is expected to have higher dispersal potential because larvae can potentially stay in the ocean currents for longer and thus travel further.  However, long-distance dispersal of planktotrophic and lecithotrophic larvae can be highly dependent on larval swimming capacity and behaviour \citep{Nanninga2018}.

To complete the process of dispersal, larvae must navigate across the seascape matrix \citep[termed the `transience' step by][]{Cayuela2018} and conclude in successful settlement and recruitment into the new population.  Though, post-transience there are several factors that can influence settlement and recruitment. For example, long-distance dispersers run the risk of being carried beyond suitable habitat or beyond tolerable environmental conditions (not to mention an increased risk of predation), particularly if they originated from a location at the periphery of a species' range.  Moreover, if the seascape contains a mosaic of suitable habitats then the likelihood of larvae finding an adequate habitat patch decreases compared to seascapes with continuous habitat patches; such information is extremely valuable for conservation managers when protecting species that are found in both types of habitats.  Larvae also have to deal with biotic pressures immediately after settlement, such as predation and competition within and between species.  For instance, in areas where there is high competition for space and resources, only the fittest individuals and species will survive and proliferate.  These factors constantly interact and contribute to shaping the ecological structure of the community, but also influence intraspecific recruitment, evolutionary processes, and connectivity within and between spatially discrete populations \citep{Cowen2009}. 


\subsubsection{1.1.4 Genetic connectivity}
\label{section1.1.4}
Genetic connectivity is defined by \citet{Lowe2010} as the degree to which gene flow affects evolutionary processes within populations.  From an ecological perspective, this definition implies that individuals must disperse to a new population and successfully contribute their genes into the local gene pool to facilitate genetic connectivity.  As a result, genetic methods of assessing connectivity represent a form of realised (or effective) connectivity, whereby only the contributions of dispersers that survive and successfully reproduce are usually quantified.  This means, however, that genetic methods do not always account for immigrants that join a population but do not reproduce.

Demographic connectivity refers to how the number of exchanges between populations (via immigration and emigration) affects population growth and vital rates \citep{Lowe2010}.  Alternatively, it has been defined as the movement of individuals between populations, the extent of which is large enough to be demographically significant, where `significance' is context dependent \citep{Leis2006}.  Unlike genetic connectivity, demographic connectivity is a function of the relative contribution of net immigration (immigrants -- emigrants) to total recruitment (local recruitment + net immigration).  In expanding populations, therefore, net immigration can be very high but still only represent a small proportion of total recruitment and, vice versa, when populations are declining, net immigration can be low but represent a large proportion of total recruitment \citep{Lowe2010}.  The latter has been presented as the `rescue effect' for populations that are near extinction because critically they rely on immigrant subsidy for persistence \citep{Brown1977}.  This demonstrates the importance of demographic connectivity for maintaining stability in populations with low local recruitment \citep{Runge2006} and for (re)colonising unoccupied habitat patches in meta-population systems \citep{Hanski1998}.

The level of exchange required to maintain demographic stability is in orders of magnitude higher than to maintain genetic homogeneity \citep{Lowe2010}.  As few as ten effective migrants per generation may be enough to maintain drift connectivity, that is, sufficient gene flow to preserve similar allele frequencies between populations \citep{Lowe2010}.  From a conservation perspective, a reduction in demographic connectivity (i.e. decrease in immigrant subsidy) can be cause for concern because it may reduce population viability; similarly, from a fisheries management perspective, insufficient demographic connectivity is equally concerning because it may result in decreases in yield.  This may impact the population (or stock) persistence, which is of central importance to managers of marine reserves and resources, and which may not be fully answerable with genetic data, unless combined with other data such as population growth rates, movement behaviour, reproductive success or biophysical modelling \citep{Lowe2010,Breusing2016}.  However, genetic connectivity allows insights into the degree of realised connectivity between two populations, which can also be very useful for managers who are interested in the contribution of immigrants that survive, reproduce and add to the local gene pool.  Furthermore, gene flow can play an important role in reducing inbreeding and purging deleterious mutations, as well as spreading advantageous alleles and introducing new adaptive variants into the population \citep{Frankham2015}. In fact, as few as one migrant per generation may be enough to reduce the harmful effects of inbreeding, termed inbreeding connectivity by \citet{Lowe2010}, thereby maintaining fitness by counteracting the loss of genetic diversity \citep{Frankham2015}. \\  

\clearpage
\noindent \textit{Estimating genetic connectivity} \\
Gene flow between spatially discrete populations can be estimated using indirect and direct approaches \citep{Gagnaire2015}, both of which rely on the development of adequate genetic markers \hyperref[sec:geneticmarkers]{(section 1.2.2)}.  Indirect approaches focus on estimating the amount of genetic divergence between populations and, in contrast, direct approaches attempt to detect migrants by using multi-locus genotypic information to assign individuals to their location or parents of origin \citep{Lowe2010,Gagnaire2015}.  These direct approaches are, by design, quite similar to some non-genetic methods \hyperref[sec:nongenetics]{(section 1.2.1)} that are used for identifying immigrants among populations.  

Indirect approaches typically use genetic indices, such as \textit{F}\textsubscript{st} \citep{Weir1984} and \textit{D} \citep{Jost2008}, and population structure analyses to estimate the degree of genetic divergence among populations.  These methods are sometimes based on a number of assumptions (e.g. Hardy-Weinberg equilibrium) relating to population dynamics (e.g. constant population size), life history (e.g. non-overlapping generations) and the contribution of evolutionary forces (e.g. negligible effect of selection) \citep{Gagnaire2015}, which are infrequently met in natural systems.  Moreover, it is important to be aware that low levels of genetic differentiation (suggesting migration is above the threshold required for genetic connectivity) do not guarantee demographic connectivity, a phenomenon recently defined as `crinkled connectivity' \citep{Ovenden2013}.  To further complicate interpretations, populations can be genetically similar in the absence of gene flow due to large effective population sizes (\textit{N}\textsubscript{e}), which can mitigate the influence of genetic drift; unfortunately, however, accurately estimating \textit{N}\textsubscript{e} in marine organisms has been notoriously difficult \citep{Hare2011}.  Nevertheless, these indirect approaches have proved vital for studying the processes of gene flow, drift and selection in natural populations \citep{Hellberg2009}.

Direct approaches can be broadly divided into methods of population assignment and parentage analysis \citep{Manel2005}; both approaches make fewer assumptions than indirect approaches but require sound knowledge of the species distribution \citep{Gagnaire2015}.  Individual assignment assigns an individual to a population or location in which their genotype has the highest probability of occurring.  Although devoid of model-based assumptions, this approach is highly sensitive to the degree of genetic differentiation between populations as the accuracy of assignment is proportional to the genetic differentiation between putative populations within a species (i.e. decreased genetic differentiation equals decreased accuracy) \citep{Christie2017}.  This has been a problem in many studies of benthic marine invertebrates because they often exhibit globally weak genetic differentiation \citep{Benestan2015}; however, this limitation \textit{may} be alleviated with the advent of genomics (\hyperref[sec:genomicsrev]{section 1.2.3}), which promises more powerful markers and higher resolution for detecting genetic differences between sampling sites \citep{Allendorf2010}.  In contrast to individual assignment, parentage analyses assign an individual to their biological parents based on their observed genotypes \citep{Jones2010}.  Although relatively insensitive to the amount of genetic differentiation, parentage assignment is highly sensitive to the proportion of potential parents sampled \citep{Christie2017}.  Therefore, for accurate assignment, a significant proportion of potential parents must be sampled, which is frequently logistically very difficult to achieve in the marine realm \citep[but see][]{Saenz-Agudelo2009,Harrison2012,Williamson2016}. \\  

\noindent \textit{Evolutionary processes influencing population structure} \\
Investigating genetic diversity and population genetic structure are often the first analyses to be undertaken in a conservation or fisheries genetics study.  As mentioned previously, this enables an indirect assessment of genetic connectivity by providing insights into patterns of gene flow, but these analyses also enable the study of other evolutionary processes acting on populations, such as drift and selection \citep{Hellberg2002}.  For example, when there are restrictions to gene flow, population allele frequencies can diverge over time due to drift (the random sampling of alleles from generation to generation); therefore, with the assumption of random mating, an allele could become fixed or completely purged from a population by chance.  Moreover, drift is stronger in small or bottlenecked populations because the sampling variance is greater when the \textit{N}\textsubscript{e} is smaller \citep{Charlesworth2009}.  Populations can also diverge when strong natural selection favours a particular mutation that increases the fitness or survivorship of the carriers, resulting in the allele sweeping to fixation in that particular population \citep{Nielsen2005}.  The absence of gene flow has been assumed to be favourable for local adaptation because gene flow can swamp locally adapted alleles \citep{Morjan2004}.  However, contrary to this, it is now widely acknowledged that local adaptation can develop in high gene flow scenarios \citep[e.g.][]{Cure2017,Diopere2017}, particularly when individuals selectively disperse towards habitats that inherently maximise their fitness \citep{Jacob2017}.  For studies where the primary goals are to assess inbreeding, \textit{N}\textsubscript{e} or connectivity, neutral markers are predominantly used because these markers are assumed to be driven by the interacting processes of gene flow and drift, and not selection (in which adaptive markers are more informative).  However, it is important to note that a recent barrier to gene flow may not be detectable in some markers because there has been insufficient time for the allele frequencies to diverge by drift \citep{Hedgecock2007}.  This could be circumvented to some degree by also using a direct approach of inferring connectivity (i.e. population or parentage assignment) or by collecting the relevant data needed to explore demographic connectivity.


%%% 1.2 Measuring connectivity
\subsection{1.2 Measuring connectivity}
\noindent Measuring connectivity is not trivial in the marine environment as the distribution and migratory pathways of marine organisms are concealed beneath the surface of the oceans, hidden from the human eye.  Nevertheless, there are a number of different techniques used to measure and quantify connectivity within and between populations in marine systems.  The choice of which technique(s) to employ depends on the study organism and the objectives of the research project, but, typically, both logistics and finance must be considered as these factors can be major limiting factors in the scope of marine connectivity studies.  As alluded to in the previous section, there are two types of approaches to measuring connectivity: direct and indirect methods.  Direct methods rely on recording movement by visual observations or by tracking the organism through tagging (genetic, chemical or physical) or assignment techniques. In comparison, indirect methods infer movement and connectivity but are not constrained by the need to recover tags or recapture the animal, and knowledge of the natal origin is not required. The remainder of this section considers the approaches used --both non-genetic and genetic-- to infer connectivity, with the majority of the section focusing on the genetic markers used as these are primary techniques used in this thesis.  

\subsubsection{1.2.1 Non-genetic methods}
\label{sec:nongenetics}
Direct observations are the most simplistic method of studying the movements of an organism.  This typically involves a capture-mark-recapture approach, whereby the organism is caught, uniquely marked and released, and when the organism is recaptured its patterns of movement can be assessed.  This method can be effective for monitoring the population sizes and movements of larger animals; however, a major drawback is that the organism must be recaptured.  Furthermore, this approach can only tell us where an individual was at a certain time at a certain place -- it does not tell us how the individual got to that place.

To determine the exact route of travel, novel techniques using electronic tagging devices have been deployed which enable real-time tracking of an organism, both horizontally (i.e. across seascape) and vertically (i.e. depth) \citep{Cooke2013}.  For example, satellite telemetry has been used to explore the three-dimensional movements of leatherback turtles (\textit{Dermochelys coriacea}) in the Atlantic and has provided insights into the spatial movements and diving behaviour of this species \citep{Hays2004}.  However, although electronic tagging has been extensively used in tracking marine vertebrates \citep{Costa2012,Hazen2012}, it is still in its infancy for marine invertebrates \citep{Fossette2016}.  Moreover, despite the technological advances in reducing the size of tags and increasing the amount of data gained per device \citep{Cooke2013}, it is still impossible to track larvae composing the meroplankton using this technique.  Pelagic larvae of benthic marine organisms typically range from 20 microns (microplankton) to two centimetres (macroplankton) in size, while the smallest satellite tag to-date known to provide useful data is $\sim$1.6 grams (Fig. \ref{birdtag}), which was used to track critically endangered spoon-billed sandpipers (www.saving-spoon-billed-sandpiper.com).  Therefore, for marine organisms that have a pelagic larval dispersal phase, other methods are needed to measure connectivity.

%% Figure: Bird tag image
\begin{figure}[h!]
\centering
\includegraphics[width=0.8\textwidth]{Chapter_figures/General_intro/Bird_tag.jpg}
\caption [Smallest satellite tag developed to-date]
{One of the smallest satellite tags developed to-date, used to track critically endangered spoon-billed sandpipers (\textit{Eurynorhynchus pygmeus}) (Photo by Chris Kelly).}
\label{birdtag}
\end{figure}
%% end of figure

Tracking individuals using geochemical differences across water bodies is another empirical non-genetic method used to study movement and connectivity in marine and freshwater organisms \citep{Thorrold2007}.  These techniques rely on using naturally occurring geochemical tags (e.g. otoliths in fish and stratoliths in some invertebrates), in which movement across different environmental gradients (e.g. temperature) leaves a distinct detectable signature in the calcified structures of these geochemical tags \citep{Thorrold2007}.  This method has been adopted in several fields of study to examine, for example, temporal and spatial patterns of connectivity and settlement in mytilid mussels \citep{Fodrie2011,Gomes2016}, and population connectivity of coastal fishes \citep{Fodrie2013,Williams2018}.  However, little or no information about connectivity can be extracted when recruits originate from locations with similar environmental signatures (i.e. similar sea chemistry) \citep{Fontes2009}.  Moreover, this method is only applicable to organisms that possess geochemical tags.


\subsubsection{1.2.2 Genetic markers: introduction}
\label{sec:geneticmarkers}
\noindent Genetic markers (or molecular markers) are any gene, DNA sequence or other molecular-based unit that can be used to study genetic variation at the individual, population or species level.  They are naturally occurring markers present in every individual and are, therefore, very useful for studying population structure and inferring connectivity.  The origin of genetic markers dates back to the 1970s, during which time allozymes were first used to investigate genetic variation and detect siblings in a range of species \citep{Allendorf2017}. By 1980, following the development of polymerase chain reaction (PCR) amplification, mitochondrial DNA (mtDNA) genes were being sequenced to explore phylogeography and population structure \citep{Allendorf2017}.  The next few decades saw technological advances give rise to more variable and increased numbers of loci (e.g. microsatellites and SNPs), followed by the genomic revolution in the latter 2000s \citep{Metzker2010,Allendorf2010,Davey2011}.  A comprehensive discussion of genetic markers and their uses in molecular ecology has been presented in several renowned review papers \citep{Sunnucks2000,Schlotterer2004,Hellberg2002,Allendorf2017}; however, some of the markers most commonly used for studying historical and contemporary population biology, and pertinent to the research presented in this thesis, are discussed below. \\

\noindent \textit{mtDNA} \\
Mitochondria have their own genome separate from nuclear DNA.  The robustness of mitochondrial DNA (mtDNA) as template DNA for subsequent PCR amplication, the variability of particular genes composing mtDNA and the general lack of recombination have contributed to the popularity of mtDNA over the last few decades \citep{Rowe2017}.  Moreover, the conserved arrangement of genes in mtDNA across the animal kingdom has led to the development of universal primers, which are now readily available.  In most animals mtDNA is maternally inherited because sperm mitochondria are usually destroyed post-fertilisation \citep{Rowe2017}.  This trait of uniparental inheritance means mtDNA can be treated as a single haplotype (with all sites sharing a common genealogy), allowing individual lineages to be tracked over time, which has useful applications in phylogeography \citep{Avise1987,Hickerson2010,Puebla2018}. However, this effectively means that these are single-locus markers, and because only the maternal lineage is investigated, there can be erroneous conclusions about genetic breaks in species where there is male-biased dispersal.  For example, in scenarios where females do not disperse or there is philopatry in females but not in males \citep[e.g. loggerhead sea turtles,][]{Casale2002}, mtDNA haplotypes may be highly differentiated when in fact there is male-mediated gene flow \citep{Stiebens2013}.  Another caveat of mtDNA is that copies of mtDNA can be translocated onto the nuclear genome, known as mtDNA pseudogenes or nuclear mitochondrial DNA segments (numts) \citep{Hazkani-Covo2010}.  These pseudogenes are non-functional and continue to evolve independently of mtDNA, which can introduce problems when primers unintentionally amplify numts in addition to the desired mtDNA fragment \citep{Calvignac2011}.  mtDNA has long been considered a neutral marker, but these assumptions are starting to be questioned, with one review suggesting mtDNA is far from neutrally evolving \citep{Galtier2009}. Nevertheless, mtDNA markers still have useful applications as a first look into genetic structure because drift is stronger compared to nuclear markers due to the lower \textit{N}\textsubscript{e} \citep{Puebla2018}, but the addition of complementary nuclear markers is generally thought to overcome the caveats of only using mtDNA markers for interpreting genetic patterns. \\ 

\noindent \textit{Microsatellites} \\
Microsatellites are short tandem repeats of typically 2-6 bp (base pairs).  They are codominant markers, meaning homozygotes and heterozygotes can be distinguished; this allows locus and population allele frequencies to be calculated.  Microsatellites are generally considered neutral markers because the repeat variations are thought to be non-functional; though, few have been thoroughly assessed and there are some known exceptions, such as the existence of some loci associated with human neurodegenerative diseases \citep{Vieira2016}. Due to their short repeat motif, microsatellite loci have high mutation rates, which can result in a large number of alleles of different sizes, making these markers highly informative for exploring genetic variation across individuals and populations \citep{Schlotterer2004}.  However, microsatellites have complex mutation behaviour and are generally species-specific \citep{Schlotterer2004}; moreover, difficulties in cross-calibrating microsatellite allele sizes between sequencing platforms and laboratories \citep[e.g.][]{Ellis2011} has limited their use in broad-scale studies.  Nevertheless, they have been, and still are, widely used in marine and terrestrial population genetics, although their usage has waned as new technology permits the high-throughput genotyping of other polymorphic markers.\\  

\noindent \textit{Single nucleotide polymorphisms} \\
Single nucleotide polymorphisms (SNPs) are single base changes at a position in the DNA sequence (Fig. \ref{snp}).  SNPs have become more popular over the last decade because they are abundant across the genome, have a simple mutation model and are eligible for high-throughput screening and automation \citep{Seeb2011,Helyar2011}.  It is also relatively straightforward to calibrate SNPs among laboratories and to assemble both spatial and temporal datasets from multiple studies \citep{Helyar2011}.  Although a single biallelic SNP locus contains less information (i.e. less allelic diversity per locus) than a microsatellite locus, individual SNPs can segregate strongly among populations \citep{Helyar2011}.  Some reports suggest that 100 neutral SNPs have approximately the same discriminatory power to detect population structure as 10-20 microsatellites \citep{Kalinowski2002}.  However, the most informative SNPs (i.e. those that show the greatest allele frequency variation among populations) have the potential to rival or even exceed the power of microsatellite markers \citep{Helyar2011}.  In combination with the rapid rise of genomics, this has led to an increase in the development of small panels of informative SNPs for a variety of marine and terrestrial species (e.g. salmonids, \citeauthor{Meek2016} \citeyear{Meek2016}; crustaceans, \citeauthor{Jenkins2018} \citeyear{Jenkins2018}; and molluscs, \citeauthor{Jiao2014} \citeyear{Jiao2014}).

%% Figure: SNP
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/General_intro/SNP_crop.pdf}
\caption [What is a single nucleotide polymorphism?]
{Single nucleotide polymorphisms (SNPs) at two positions in a DNA molecule. A SNP is a single base change at a  position in a DNA sequence.}
\label{snp}
\end{figure}
%% end of figure
  
\subsubsection{1.2.3 Genetic markers: the genomic revolution}
\label{sec:genomicsrev}
\noindent The advent of next-generation sequencing (NGS), also called massive parallel sequencing, has paved the way for large amounts of sequence data to be generated at more affordable costs \citep{Ellegren2008,Metzker2010,Mardis2011}. This introduced new bioinformatics challenges \citep{Pop2008}; however, year upon year increases in the read lengths generated by sequencing platforms \citep{Quail2012,Pillai2017} and improvements in algorithms and computational power, have somewhat alleviated these challenges.  With the enormous amounts of data that can now be gleaned from the genomes of model and non-model organisms, it has been said that we are in a genomic revolution \citep{Ellegren2014}, and with the continual advancement of sequencing technology, such as third-generation single-molecule real-time sequencing (e.g. Pacific Biosciences, Oxford Nanopore Technologies), it seems that the revolution is not slowing down just yet.

NGS technology has enabled the development of new methods of marker isolation for studies of population genetics, adaptation and conservation biology \citep{Andrews2014a}.  For example, exon-capture \citep{Hodges2007}, one such method made possible by NGS, targets and isolates SNPs from the exome, thereby permitting the study of protein-coding genomic regions \citep[reviewed in][]{Mamanova2010}.  Another method, next-generation cDNA sequencing (RNAseq), makes it possible to isolate SNPs \citep{Barbazuk2011} or to sequence entire transcriptomes from almost any individual or tissue \citep{Ozsolak2011,Todd2016}.  However, one of the most popular methods that emerged from NGS is restriction-site associated DNA sequencing (RADseq) \citep{Miller2007,Baird2008}, in which short regions of DNA adjacent to restriction enzyme cut sites are sequenced.  This is a type of reduced-representation sequencing (RRS) approach, in which a subset of markers are isolated from across the genome, the number of which will depend upon the size of the genome and the cutting frequency of the restriction enzyme used \citep{Davey2011}.
	
Since the conception of RADseq, a vast number of similar methods have been introduced, some of which have only minor tweaks or modifications to the traditional RADseq approach, while others have more pronounced differences \citep{Campbell2018}.  For example, nextRAD was recently developed by some of the original authors of RADseq \citep{Baird2008,Etter2011} but uses a combination of selective primers and transposomes to cut DNA instead of restriction enzymes \citep{Fu2017}.  One of the main advantages of RADseq and some of its derivatives is that no prior knowledge of the genome is required to perform the techniques \citep{Rowe2011}, which is a massive advantage for working with non-model species (species without a reference genome). \added{However, if a reference genome is available for the study species (or a closely related species), NGS reads can be aligned to the reference genome; this can have very useful benefits for RADseq studies with non-model species such as: (i) improved assembly and identification of SNP loci (by reducing potential effects of sequencing error); (ii) enhanced ability to filter paralogous or repetitive sequences and remove non-target DNA (contamination); (iii) allowing the physical position of loci to be considered (advantagous for mapping studies); and (iv) increased statistical power to detect genomic regions of interest, for example regions under divergent selection between populations \citep{Andrews2016}.} Another major advantage of these techniques is that thousands to tens of thousands of genome-wide SNPs can be discovered from across the genome in a single sequencing run \citep{Andrews2016}.  In comparison, typical microsatellite development usually consists of tens of markers, and there is no way of knowing if these microsatellites are distributed evenly across all of the chromosomes.  In studies of population genetics and local adaptation, the ability to potentially sample across all chromosomes is highly advantageous because it offers more opportunities to find informative loci (neutral or adaptive).  Of course, in non-model organisms, there is also no way of knowing whether the SNPs discovered from a RADseq approach are distributed evenly across the chromosomes; however, based on our knowledge of how restriction enzymes operate, and the sheer number of SNPs discovered using these approaches, it is likely that a large proportion of chromosomes will be represented, maximising the chances of finding some interesting loci.

RADseq approaches have now been applied to many different research fields to answer a variety of questions about ecology and evolution.  For example, they have been used in studies of species identification \citep{Maroso2018}, species delimitation \citep{Pante2015,Herrera2016}, hybridisation \citep{Faust2018}, stock management \citep{Mullins2018}, population assignment \citep{Drinan2018}, phylogeography \citep{Emerson2010}, local adaptation \citep{Harrisson2017}, connectivity \citep{VanWyngaarden2017,Xuereb2018} and growth-related traits \citep{Yu2018}.  One of the areas in which RADseq approaches have become particularly useful is in accurately resolving population structure at broad- and fine-scales.  In marine species, determining population structure has been challenging due to typically weak genetic differentiation and the limited resolution offered by traditional molecular markers \citep{Benestan2015}.  However, RADseq approaches enable the discovery of numerous genome-wide markers which maximises the power to detect subtle genetic differences among populations \citep{Funk2012}.  Such an approach has allowed researchers to resolve fine-scale population structure in range of marine species including great scallops \citep{Vendrami2017}, emperor penguins \citep{Younger2017}, American lobsters \citep{Benestan2015}, staghorn coral \citep{Drury2017} and starlet sea anemones \citep{Reitzel2013}.  On the other hand, it has also confirmed the existence of no population structure across certain spatial scales \citep[e.g.][]{Everett2016,Perez-Portela2018}, which is still an equally important finding for marine management as it implies genetic connectivity across the geographical scale of the samples analysed.

RADseq and genomics have also contributed to the discovery of `outlier' markers, which are candidate markers potentially under strong drift or divergent selection \citep{Lotterhos2015}.  From a connectivity perspective, these outlier markers typically have greater power to differentiate populations, which has promising applications for inferring connectivity using population assignment approaches \citep{Gagnaire2015}.  Indeed, the incorporation of gene-associated markers in assignment has already proven to be incredibly useful for fisheries management, where these markers are used as tools to help tackle illegal fishing \citep{Martinsohn2009,Nielsen2012}.  From a conservation perspective, outlier markers have the potential to revolutionise the delineation of conservation units \hyperref[section1.3.1]{(section 1.3.1)}\deleted{(section 1.3.1)} by identifying adaptive diversity in protected species \citep{Funk2012}.  Genomics appears to have been widely accepted by academic conservationists, with many recent review papers dedicated to discussing the opportunities provided by conservation genomics \citep[e.g.][]{Allendorf2010,Benestan2016,Flanagan2017,Barbosa2018}; however, a major challenge is the translation and integration of genomic data into conservation practice \citep{Shafer2015,Garner2016}.


\subsubsection{1.2.4 Biophysical modelling}
\noindent All previous methods described in this section provide an empirical assessment of connectivity, each with their own merits and caveats.  As an alternative, powerful biophysical models have been developed to assess marine connectivity; these utilise biological and hydrological data to simulate larval dispersal across seascapes \citep{Metaxas2009}.  Biophysical models use outputs of ocean models (e.g. current velocities, directions, etc.) as inputs to particle tracking algorithms that track individual particles (larvae) from a starting point to settlement \citep{Cowen2006}.

However, to accurately predict larval dispersal using these approaches, incorporating accurate biological parameters is critical.  For example, the length of time spent drifting is usually determined by an organism's PLD, which can be estimated from laboratory or (preferably) field studies \citep{Metaxas2009}.  However, many other factors can influence larval dispersal that could be integrated into the model, such as mortality \citep{Treml2015}, spawning time and periodicity \citep{Kough2015}, salinity and temperature \citep{Larez2000}, and settlement likelihood and larval behaviour \citep{Treml2012}.  Simulations that are performed with the incorporation of such species-specific data are usually termed individual-based models (IBMs) \citep{DeAngelis2014}.

Biophysical models provide insight into the larval dispersal potential and demographic connectivity of the species under study \citep{Thomas2013}.  They have also been used to estimate dispersal kernels, defined as the spatial probability distribution of dispersal distances based on repeated events, which theoretically capture the temporal variability of larval dispersal \citep{Siegel2003,Cowen2009}.  More recently, biophysical modelling has been combined with genetic studies to explore seascape genetics or seascape genomics \citep{Riginos2016,Selkoe2016}.  As with other naming conventions (i.e. population genetics/genomics, conservation genetics/genomics, etc.), seascape genomics is generally analogous to seascape genetics, with the exception that seascape genomic studies genotype markers using high-throughput genomic techniques such as RADseq.  Seascape genetic studies incorporate the fields of ecology, oceanography and geography to explore and interpret patterns of marine connectivity \citep{Christie2010,Selkoe2016}.  This approach has provided novel insights into spatial patterns of population structure and connectivity in many species including seagrass \citep[e.g.][]{Jahnke2018}, crustaceans \citep[e.g.][]{Thomas2013,Benestan2016b} and molluscs \citep[e.g.][]{Breusing2016,Sandoval-Castillo2018}, and is likely to have many applications in fisheries science, conservation and marine reserve design \citep{Selkoe2016,Mertens2018}. 


%% 1.3 Marine conservation
\subsection{1.3 Marine conservation}
The main aim of marine conservation is to restore and/or protect marine ecosystems to preserve biodiversity and to avoid overexploiting marine resources.  One of the first world-wide political movements to recognise the importance of conserving natural environments was founded in 1992: the Convention on Biological Diversity (CBD); this was was initially signed by 150 countries (currently 182 countries and the European Union) \citep{Ten-Kate2002}.  The objectives of the CBD are to (i) conserve biological diversity, (ii) ensure the sustainable use of its components, and (iii) enable fair and equitable sharing of benefits arising from the use of genetic resources \citep{Ten-Kate2002}.

The CBD movement coincided with the establishment of the Oslo/Paris (OSPAR) Convention, which was inaugurated to protect the marine environment of the northeast Atlantic.  The OSPAR Convention is a collaboration of fifteen governments and the European Union (EU) (also called contracting parties) whose purpose is to develop policy and international agreements to safeguard the OSPAR Maritime Area in the northeast Atlantic (Fig. \ref{ospar}).  In 1994, the OSPAR Maritime Area was divided into five regions for assessment and monitoring purposes: Arctic Waters (region I), Greater North Sea (region II), Celtic Seas (region III), Bay of Biscay and Iberian Coast (region IV), and Wider Atlantic (region V) (OSPAR Commission 2013).  Legislation within and across countries has followed the OSPAR Convention which empowers governments to deliver towards the OSPAR agreement. For example, the European Union introduced the EU Marine Strategy Framework Directive (MSFD) in 2008 and the UK introduced the Marine and Coastal Access Act in 2009.  The MSFD requires that each member state establishes `coherent and representative networks' of \replaced{Marine Protected Areas (MPAs)}{MPAs} by 2020.  Indeed, from a UK government perspective, its recent movement towards protecting marine environments in UK waters also contributes to its commitments to the CBD and other international agreements, thereby satisfying both politicians and conservationists.

%% Figure: OSPAR Maritime Area
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/General_intro/ospar_maritime_area.jpg}
\caption [OSPAR Maritime Area]
{The OSPAR Maritime Area.  This area is divided into five regions for assessment purposes: Arctic Waters (region I), Greater North Sea (region II), Celtic Seas (region III), Bay of Biscay and Iberian Coast (region IV), and Wider Atlantic (region V).}
\label{ospar}
\end{figure}
%% end of figure

\subsubsection{1.3.1 Conservation units}
\label{section1.3.1}
Delimiting conservation units (CUs) is an essential first step for managers and policymakers so that they know the boundaries of the populations they are trying to conserve \citep{Funk2012}.  Defining such boundaries allows the status of a population to be assessed, which can inform the development of a management strategy.  However, defining populations can be extremely challenging in habitats that are continuous over larges spaces \citep{Waples2006}, which can be a prevalent feature of marine environments.  Nevertheless, CUs are still relevant for marine species, particularly for the management of fishery stocks and for the protection of vulnerable species \citep{Heyden2014}.      

Conservation units have generally been discussed in terms of evolutionary significant units (ESUs), management units (MUs) and, recently, adaptive units (AUs) \citep{Funk2012,Barbosa2018}.  Definitions of ESUs are plentiful \citep[Box 1,][]{Funk2012}, though overall there is general agreement that an ESU represents a group of conspecifics that exhibit high genetic and ecological distinctiveness \citep{Ryder1986,Funk2012}, often due to allopatric or adaptive divergence \citep{Moritz1994}.  Conserving ESUs is, therefore, a high priority for management because the maintenance of different ESUs will maximise the evolutionary potential of a species to adapt to environmental change \citep{Funk2012}.  Populations that are demographically independent (i.e. where population growth rate is dependent on local recruitment and not immigration) are typically classed as MUs \citep{Moritz1994}.  Identifying these units can be particularly important in fisheries for delineating stocks with distinct population growth rates and demography \citep{Palsboll2007,Heyden2014}.  Whereas ESUs typically consider all genetic variation and MUs only consider neutral genetic variation \citep{Funk2012}, AUs specifically describe the adaptive differences between populations, which can be very important for prioritising conservation resources and for deciding which individuals to use as sources for supplementing depauperate populations \citep{Moritz1999}.  For example, supplementing a focal population with genetically incompatible sources (e.g. individuals adapted to a very different environment) may lead to outbreeding depression \citep{Frankham2011}.  For restocking programs, this knowledge is crucial so that hatchery managers ensure that the juveniles bred from their facility are compatible with the target population or area being stocked \citep{Ward2006}.

Allocating and prioritising conservation resources is not trivial.  For example, if resources were only available to conserve one or two populations, managers must weigh-up which populations are of highest value to the conservation of a species overall.  This is where CUs and other relevant data sources, such as measures of genetic diversity, are central for conservation managers.  For instance, stepping stone sites (and associated populations) that link isolated populations can be vital for long-distance dispersal, connectivity and range expansions \citep{Saura2014}; therefore, these areas may be prioritised to maintain linkages between ecosystems.  In the marine environment, such a prioritisation approach is often used to help design networks of Marine Protected Areas.    


\subsubsection{1.3.2 Marine Protected Areas}
Marine Protected Areas (MPAs) are areas of sea or ocean designated to protect habitats or species, or both.  At the time of writing, over 11,000 MPAs have been designated globally to protect the world's oceans (www.mpatlas.org).  This equates to around 3.7 \% of all oceans on Earth being protected.  However, not all of these MPAs are currently implemented, well-managed or enforced, which is often because of capacity shortfalls associated with inadequate staff and finances \citep{Gill2017}.

MPAs vary greatly in size and in the stringency of protection, ranging from large no-take zones (NTZs) where typically no activity is permitted, to smaller localised designations that may prohibit a specific recreational activity or the fishing of a particular species.  Research has suggested that the conservation benefits from MPAs increase exponentially when MPAs: (i) are NTZs, (ii) are large ($>$100 km$^2$), (iii) are well regulated and enforced, (iv) have been establish for a long time ($>$100 years), and (v) are isolated by deep water or sand \citep{Edgar2004}.  However, although large MPAs make substantial contributions towards the CBD's Aichi Target 11 (protecting 10 \% of coastal and marine areas by 2020), their contribution to marine conservation has been debated \citep{Davies2017,OLeary2018}.  The negativity has mainly stemmed from a lack of understanding of their actual benefits for conserving biodiversity, and that these large-scale MPAs are (i) driven predominantly by political targets, and (ii) situated typically in remote areas where there are minimal threats and where establishment is easier \citep{Leenhardt2013,Devillers2015}.  On the other hand, many advocate that they are essential for protecting wide-ranging or circumtropical species such as seabirds and tuna \citep{Young2015a}, and that they are important for maintaining pristine areas \citep[e.g. the Chagos Archipelago,][]{Sheppard2012} or for capturing habitat shifts associated with climate change (Toonen et al. 2013).  Moreover, it seems logical to create legislation to protect these remote areas now before they are potentially targeted for exploitation or habitation in the future \citep{Toonen2013b}.  Irrespective of these contrasting viewpoints, it is inevitable that large-scale MPAs will indeed become a prominent feature in our oceans in the coming years as governments seek to meet global conservation targets.

Despite the recent furore surrounding large MPAs, small MPAs still remain critical for marine conservation \citep{Toonen2013b}, particularly in coastal areas within the jurisdiction of individual governments.  For example, in the northeast Atlantic, individual governments have designated a number of MPAs within their exclusive economic zone (EEZ) with the aim of establishing a MPA network to protect local features (i.e. habitats and species) and, of course, to satisfy their commitments to international agreements.  A network of MPAs is thought to collectively deliver more benefits to biodiversity than individual, unrelated MPAs \citep{Foster2017}.  Countries have full rights to manage the marine resources in their EEZ, which extends 200 nautical miles (370 km) from the coastline, allowing each government to designate MPAs anywhere within their territorial waters.  However, unless these MPAs have well-thought-out management and monitoring plans, with adequate funding and staff to carry out the plans \citep{Alvarez-Fernandez2016}, they are essentially just polygons on a map/chart.  The challenge for local and regional marine managers is, therefore, how to strategically allocate available resources in a way that maximises protection of marine biodiversity, while also considering the impact to stakeholders that are involved, interested and/or affected by the MPA \citep{Dehens2018}.  Simultaneously, at the government level, a further challenge is how to demonstrate that MPAs are not simply designated randomly but are distributed purposefully across territorial waters such that they are an `ecologically coherent' network \citep{Ardron2008}, a requirement for contracting parties of the OSPAR Convention and EU member states.

\subsubsection{1.3.3 Ecological coherency}
As part of the OSPAR Convention, contracting parties agreed to establish an ecologically coherent well-managed network of MPAs, initially by 2010 \citep{Ardron2008}.  Yet, at the time the meaning of ecological coherency or how to assess whether a network is ecologically coherent or not was not explicitly stated \citep{Ardron2008a}.  Since then, OSPAR reports have generally defined an ecologically coherent MPA network as a network that considers Adequacy, Viability, Representation, Replication and Connectivity (\citeauthor{OSPARCommission2007} \citeyear{OSPARCommission2007}; \citeyear{OSPARCommission2013}).  Together, these criteria influence and take into account the size of MPAs, the coverage of species and habitats, the spatial distribution of MPAs across biogeographical regions, the number of replicate sites for specific features of interest, and the links between sites at different spatial and temporal scales \citep{OSPARCommission2013}.  However, in 2013, the methods developed for assessing these criteria were still being refined and re-evaluated as the availability of data and knowledge of marine ecosystems increases \citep{OSPARCommission2013}.

The five principles of the recent 2016 status report of the OSPAR network of MPAs were phrased slightly differently to previous reports: Features, Representativity, Connectivity, Resilience and Management (Fig. \ref{coherence}), but which overall make it easier to interpret the OSPAR principles of ecological coherence \citep{OSPARCommission2017}.  Essentially, Adequacy and Viability have been merged into Features, Resilience encompasses elements of Replication, and Management has been officially added as a focal criterion of ecological coherence (which is a welcome addition due to the emphasis of a `well-managed' network outlined in the mission statement of the OSPAR Convention).  To assess these five principles in 2016, the `Madrid Criteria' was applied, which was designed to reflect the key network principles, while acknowledging the data limitations associated with target species and habitats, and OSPAR MPA performance \citep{OSPARCommission2017}.  For example, for connectivity, the Madrid Criteria stated that MPAs must be geographically well-distributed, with a maximum distance between MPAs of 250 km in coastal waters, 500 km in offshore waters, and 1000 km in areas outside national jurisdiction \citep{OSPARCommission2017}.  Other criteria for assessing Representativity, Features and Resilience were also outlined and in 2016 all principles were formally assessed using these criteria.  However, the report concluded that although significant progress has been made in developing the network, the OSPAR Maritime Area cannot yet be considered ecologically coherent, with OSPAR citing (once more) that further development of methods to assess ecological coherence is required going forward \citep{OSPARCommission2017}.

%% Figure: Ecological coherence criteria
\begin{figure}[h!]
\begin{small}
\noindent\fbox{\begin{minipage}{\dimexpr\textwidth-2\fboxsep-2\fboxrule\relax}
\flushleft
\noindent \textbf{Features} \\
\noindent \textit{MPAs should be designated in areas that best represent the range of habitats, species and ecological processes in the OSPAR Maritime Area. Proportions of features that should be protected by the MPA network may be higher for particularly threatened and/or declining features}. \\
\noindent \textbf{Representativity} \\
\noindent \textit{MPAs should protect examples of the same features across their known biogeographical extent to reflect known sub-types. EUNIS Level 3 habitats are stated as a potentially useful way of characterising the OSPAR Maritime Area for the purposes of including biogeographic variation in the network}. \\
\noindent \textbf{Connectivity} \\
\noindent \textit{In the absence of dispersal data, connectivity may be approximated by ensuring the MPA network is well distributed geographically. Where scientific understanding is further developed, the MPA network should reflect locations where a specific path between identified places is known (e.g. critical areas of a life cycle for a given species)}. \\
\noindent \textbf{Resilience} \\
\noindent \textit{Replication of features in separate MPAs in each biogeographic area is desirable where possible. The appropriate size of a site should be determined by the purpose of the site and be sufficiently large enough to maintain the integrity of the feature(s) for which it is selected}. \\
\noindent \textbf{Management} \\
\noindent \textit{OSPAR MPAs should be managed to ensure the protection of the features for which they were selected and to support the functioning of an ecologically coherent network}.
\end{minipage}}
\caption [OSPAR ecological coherence principles]
{OSPAR Convention five key principles for assessing ecological coherence of MPA networks. Definitions are taken directly from the OSPAR Commission 2017 report.}
\label{coherence}
\end{small}
\end{figure}
%% end of figure

\subsubsection{1.3.4 Britain's MPA network}
The UK is obliged to establish its own ecologically coherent network of well-managed MPAs because of commitments to the OSPAR Convention, the EU MFSD and the CBD.  This network will contribute to the OSPAR MPA network but will also satisfy national commitments announced under the Marine and Coastal Access Act 2009 (England and Wales), the Marine Act 2010 (Scotland) and the Marine Act 2013 (Northern Ireland).  As of March 2018, approximately 24 \% of marine/estuarine environments around Britain are within MPAs (www.jncc.defra.gov.uk/page-4549). At the time of writing, the network comprised 299 MPAs consisting of: Special Areas of Conservation (SACs; 105) and Special Protected Areas (SPAs; 107) with marine components, Nature Conservation Marine Protected Areas (NCMPAs; 30 in Scotland), Ramsar sites (Isle of Man), and Marine Conservation Zones (MCZs; 56 in England, Wales and Northern Ireland).

Recently, the UK government has focused on augmenting the network with MCZs in England and Wales (Fig. \ref{fig:allmczs}).  The MCZ project began in 2008 and was co-led by the Joint Nature Conservation Committee (JNCC) and Natural England (NE) with the aim of identifying and recommending candidate sites that fill in gaps in the MPA network, potentially addressing any deficits in Features, Representativity, Connectivity and Resilience.  Four regional projects covering southwest England (Finding Sanctuary), southeast England (Balanced Seas), the Irish Sea (Irish Sea Conservation Zones) and the North Sea (Net Gain) were commissioned and, with the support of an independent scientific advisory panel, they submitted their recommendations to JNCC and NE in September 2011. These recommendations were reviewed by JNCC and NE, who submitted 127 candidate MCZs to the Department for Environment, Food & Rural Affairs (DEFRA) in July 2012.  Subsequently, DEFRA designated 27 MCZs (tranche one, November 2013) and 23 MCZs (tranche two, January 2016) in English waters and one MCZ in Welsh waters (Skomer Island, 2014).  A third tranche of 41 MCZs and the addition of new features to 12 existing MCZs is currently under review at DEFRA following a public consultation in June 2018.

%% Figure: MCZs in England and Wales
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/General_intro/Map_UK_crop.png}
\caption [Marine Conservation Zones in England]
{Marine Conservation Zones (MCZs) designated (red outline) and proposed (blue outline) in English waters.}
\label{fig:allmczs}
\end{figure}
\clearpage
}
%% end of figure

The MCZ project has, in the view of some commentators, coincided with a shift from a bottom-up to a top-down approach, with stakeholder engagement now limited to bilateral consultations \citep{Lieberknecht2016}.  The MCZ project has also steered away from its initial focus on broad-scale networks and has instead concentrated efforts on single-feature conservation \citep{Lieberknecht2016}, such as protecting vulnerable species (e.g. pink sea fans) and key habitats (e.g. intertidal boulder communities).  This deviation from a holistic approach has meant that ascertaining whether the UK network satisfies the connectivity principle of the OSPAR Convention is not straightforward.  In England and Wales, assessing connectivity of the network has primarily focused on linking discrete habitats (e.g. littoral rock and hard substrata, sublittoral sediment, etc.), such that each habitat is represented by a MPA every 80 km or less (\citeauthor{Carr2014} \citeyear{Carr2014}; \citeyear{Carr2016}), the spacing recommended by \citet{Roberts2010} to maintain ecological connectivity. Connectivity for a discrete habitat is deemed sufficient when 40 km buffers drawn around two adjacent MPAs converge (\citeauthor{Carr2014} \citeyear{Carr2014}; \citeyear{Carr2016}).  For many benthic marine species, defining a network in this way may be sufficient to maintain connectivity between nearby populations. However, it is important to note that it may not suit all species because, as discussed in section 1.2, connectivity can be influenced by a number of biological and hydrological factors.  Moreover, deciding which species to include in assessments of connectivity is not trivial, and compromises are likely to be made in situations for which there are few available dispersal data or there are too few resources to generate novel data.


\subsection{1.4 Research aims and hypotheses}
The overall aim of this thesis was to investigate which taxa may be best suited for assessing genetic connectivity between MPAs, and to assess spatial genetic diversity and connectivity in the species chosen using population genetics and genomics.  To do this, a literature survey of population genetic and phylogeographic studies was firstly conducted to explore what has already been documented about the spatial genetic patterns of marine taxa across the northeast Atlantic. In addition to studying genetic patterns across a broad range of taxa, this meta-analysis provided further insights into the biological and methodological information needed to ascertain which taxa may be considered as good candidates for assessing genetic connectivity between MPAs across the British Isles and the wider northeast Atlantic. 

Secondly, the knowledge gained from the literature survey facilitated the design of a set of criteria that identified ideal traits of a candidate species for assessing genetic connectivity; subsequently, based on these criteria, this led to the selection of two species for further study.  Finally, microsatellite and SNP markers were employed to explore the population genetic structure of these species and these data were used to infer connectivity between MPAs designated within their respective ranges.  Specific research questions and hypotheses for each component are outlined below: \\

%% Chapter 2
\noindent \textbf{Comparative phylogeography meta-analysis} \\
\noindent 1. Are there common\deleted{alities in genealogical and} phylogeographic patterns across marine taxa in the northeast Atlantic?
\begin{itemize}
\setlength\itemsep{0.5pt}
  \item[] H\textsubscript{0}: Marine taxa have contrasting patterns of\deleted{genealogy and} phylogeography.
  \item[] H\textsubscript{1}: Marine taxa have common patterns of\deleted{genealogy and} phylogeography.
\end{itemize}
\vspace{1pt}
\noindent 2. Were historic population expansions linked to the Last Glacial Maximum?
\begin{itemize}
\setlength\itemsep{0.5pt}
  \item[] H\textsubscript{0}: Historic population expansions were not linked to the LGM.
  \item[] H\textsubscript{1}: Historic population expansions were linked to the LGM.
\end{itemize}

%% Chapter 3
\noindent \textbf{Selecting taxa to assess genetic connectivity between MPAs} \\
\noindent Can a set of criteria be designed to assist researchers and managers select appropriate taxa to use as surrogates for assessing connectivity between MPAs?
\begin{itemize}
  \item[] No hypotheses were tested for this component.
\end{itemize}

%% Chapter 4, 5 & 6
\noindent \textbf{Population genetic structure and connectivity in species selected} \\
\noindent 1. Is there evidence of population genetic structure across the sampled range?
\begin{itemize}
\setlength\itemsep{0.5pt}
  \item[] H\textsubscript{0}: No population structure - individuals at all sample sites analysed are in panmixia.
  \item[] H\textsubscript{1A}: Weak population structure - suggestive of high gene flow and/or \textit{N}\textsubscript{e} among sample sites.
  \item[] H\textsubscript{1B}: Regional population structure - suggestive of reduced gene flow among certain sample sites.
\end{itemize}
\vspace{1pt}
\noindent 2. Is the British MPA network sufficient to maintain connectivity in this species?
\begin{itemize}
\setlength\itemsep{0.5pt}
  \item[] H\textsubscript{0}: No - evidence suggests the network is insufficient to maintain connectivity.
  \item[] H\textsubscript{1A}: Yes - evidence suggests the network is sufficient to maintain connectivity.
  \item[] H\textsubscript{1B}: Yes - some evidence that the network could maintain connectivity between certain areas.
\end{itemize}



%--------------
%
% Chapter 2: Meta-analysis
% 
%--------------
\newpage
\section{Chapter 2: Meta-analysis of northeast Atlantic marine taxa shows contrasting phylogeographic patterns following post-LGM expansions}
\rule{\textwidth}{0.8pt}
\vspace{1pt}

\noindent This chapter is based on a paper published in the journal \textit{PeerJ}.  The reference is given below and the full paper is available in the Appendix. \\

\noindent Jenkins TL, Castilho R, Stevens JR (2018) Meta-analysis of northeast Atlantic marine taxa shows contrasting phylogeographic patterns following post-LGM expansions. \textit{PeerJ} \textbf{6}, e5684.

\newpage
\subsection{2.0 Abstract}
Comparative phylogeography enables the study of historical and evolutionary processes that have contributed to shaping patterns of contemporary genetic diversity across co-distributed species. In this study, we explored genetic structure and historical demography in a range of coastal marine species across the northeast Atlantic to assess whether there are commonalities in phylogeographic patterns across taxa and to evaluate whether the timings of population expansions were linked to the Last Glacial Maximum (LGM). A literature search was conducted using Web of Science. Search terms were chosen to maximise the inclusion of articles reporting on population structure and phylogeography from the northeast Atlantic; titles and abstracts were screened to identify suitable articles within the scope of this study. Given the proven utility of mtDNA in comparative phylogeography and the availability of these data in the public domain, a meta-analysis was conducted using published mtDNA gene sequences. A standardised methodology was implemented to ensure that the genealogy and demographic history of all mtDNA datasets were reanalysed in a consistent and directly comparable manner. Mitochondrial DNA datasets were built for 21 species. The meta-analysis revealed significant population differentiation in 16 species and four main types of haplotype network were found, with haplotypes in some species unique to specific geographical locations. A signal of rapid expansion was detected in 16 species, whereas five species showed evidence of a stable population size. Corrected mutation rates indicated that the majority of expansions were estimated to have occurred after the earliest estimate for the LGM ($\sim$26.5 Kyr), while few expansions were estimated to have pre-dated the LGM. This study suggests that post-LGM expansion appeared to be common in a range of marine taxa, supporting the concept of rapid expansions after the LGM as the ice sheets started to retreat. However, despite the commonality of expansion patterns in many of these taxa, phylogeographic patterns appear to differ in the species included in this study. This suggests that species-specific evolutionary processes, as well as historical events, have likely influenced the distribution of genetic diversity of marine taxa in the northeast Atlantic.

\subsection{2.1 Introduction}
Comparative phylogeographic studies present opportunities to explore how historical events may have helped shape patterns of genetic structure amongst co-distributed species \citep{Avise1987,Avise2009,Hickerson2010}. Patterns of concordant phylogeographical structure across multiple taxa are particularly informative because, while some patterns of spatial genetic structure may be caused by species-specific evolutionary processes, patterns common across multiple taxa may suggest similar evolutionary histories, such as common barriers to gene flow \citep{Avise2009,Hickerson2010}. These findings can be important for conservation because of the potential to modify management actions in the light of the differing phylogeography of multiple species across the same geographical area \citep{Pelc2009,Toonen2011,Heyden2014,Liggins2016}. In marine biology, such comparative studies have made important contributions to our understanding of how historical events, such as the Pleistocene glaciations, have helped shape the spatial patterns of contemporary genetic diversity of marine taxa \citep{Patarnello2007,Maggs2008,Marko2010,Ni2014}.

The Pleistocene epoch was characterised by recurrent glaciations and intensive fluctuations in climate that periodically influenced the spatial distributions of plants and animals \citep{Hewitt1999,Hofreiter2009}. The most recent glacial period began approximately 115 Ka and nearly all ice sheets were at their maximum (Last Glacial Maximum, LGM) between 26.5-19 Ka \citep{Clark2009}. The advances of the Northern Hemisphere ice sheets led to significant changes in temperature and sea levels \citep{Lambeck2001}. This must have had profound implications for habitat availability and the population persistence of coastal species-large parts of species' ranges would have been reduced, while other species may have survived in glacial refugia \citep{Maggs2008,Provan2008}. As the ice retreated and the sea level rose, a number of individuals from refugial populations may have dispersed and recolonised areas unavailable during the glaciation \citep{Hewitt2000}. Changes in latitudinal ranges and population sizes can have distinct effects on the genetic architecture of a species due to the competing processes of mutation, drift and selection; moreover, the deep molecular divergence reported in taxa associated with several known European refugia suggests repeated expansion and contraction of conspecific populations were common throughout the Pleistocene \citep{Hewitt2004}.

In the northeast Atlantic, the ice sheets extended as far south as Britain and Ireland, leaving an ice-free zone in mid-southern England, with possibly a small area in southwest Ireland free of ice \citep{Chiverrell2010}. However, the predicted extent of ice coverage across southern Ireland and the Celtic Sea differs among studies \citep[e.g.][]{Taberlet1998,Hughes2016}. The advance of the ice sheets led to a drastic drop in sea levels in the English Channel, resulting in the complete emersion of the channel between England and France, except for a palaeo-river that extended across the continental margin \citep{Menot2006}. This suggests that extant coastal communities inhabiting these areas are likely recolonisers originating from glacial refugia. It has been suggested that Hurd Deep, a trench in the English Channel (Fig. \ref{Atlantictopo}), might have persisted as a marine lake during the LGM, thereby acting as a potential glacial refugium \citep{Provan2005,Hoarau2007}. Other areas further south, including Brittany \citep{Coyer2003} and the Iberian Peninsula \citep{Hoarau2007,Neiva2012} (Fig. \ref{Atlantictopo}), have also been postulated to act as refugia during the LGM. This was supported by high levels of genetic diversity found at these areas in the species studied, a key signature indicative of glacial refugia \citep{Provan2008}.

%% Figure: Northeast Atlantic map with LGM and refugia
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter2/Figure_1_Map.pdf}
\caption [Topographical map of the northeast Atlantic Ocean]
{Topographical map of the northeast Atlantic Ocean. The white dotted lines represent the maximum extent of ice cover during the Last Glacial Maximum (LGM) (redrawn from Hughes et al. 2016).  Orange lines indicate putative refugia: Hurd Deep, Brittany and Iberia.}
\label{Atlantictopo}
\end{figure}
%% end of figure

Studies of single-species phylogeography across the northeast Atlantic are common; yet, because of the differences in molecular methodologies and analytical approaches, it can be difficult to compare results reliably. By applying a consistent methodology across all studies, this standardises the analysis \citep{Harrison2011}, enabling patterns of phylogeography to be explored and compared within and across taxa. Two comparative meta-analyses in the Atlantic Ocean have been published to-date: the first explored the feasibility of distinguishing genetic signatures of periglacial refugia from southern refugia in eight benthic marine species \citep{Maggs2008}, and the second looked for concordance among phylogeographical breaks around the southeast coast of the United States of America \citep{Pelc2009}. Systematic meta-analyses across diverse taxa in other seas and oceans have proved useful for exploring broad patterns of phylogeography \citep[e.g.][]{Patarnello2007,Kelly2010,Marko2010,Ni2014}; for example, one study of rocky-shore taxa from the northeastern Pacific found that 36 \% of species showed evidence of population expansions associated with the LGM, while 50 \% exhibited demographic patterns consistent with stable effective population sizes \citep{Marko2010}. However, such a study for marine taxa across the northeast Atlantic has yet to be undertaken.

In this study, we reanalyse available mitochondrial (mt)DNA data to compare the phylogeography of coastal benthic and demersal organisms across the northeast Atlantic (Fig. \ref{Atlantictopo}), an area characterised by complex oceanography and historical biogeographical events, such as the Pleistocene glaciations. Specifically, our aims were: (i) to identify commonalties (or otherwise) in contemporary genetic structure; (ii) to re-examine historical demography to test for signatures of population expansions; and (iii) to estimate the timings of any expansions detected. We discuss our findings in the context of the Pleistocene glaciations, asking in particular whether the LGM affected the phylogeography of marine taxa concordantly or discordantly.

\subsection{2.2 Materials and methods}
\subsubsection{2.2.1 Literature search}
To compare the phylogeography of benthic and demersal organisms across the northeast Atlantic, we undertook a meta-analysis of molecular phylogeographic studies. A literature search was conducted using Web of Science (Thomson Reuters) in February 2015. Search terms were chosen to maximise the inclusion of articles reporting on population structure and phylogeography from the northeast Atlantic. The following sets of Boolean search terms were submitted to the Advanced Search Tool: (1) gene flow OR population structure OR genetic diversity OR phylogeograph*; (2) marine OR intertidal OR subtidal OR estuar*; and (3) Atlantic. Titles and abstracts were screened to identify suitable articles within the scope of this study and only articles that matched the following criteria were retained: (a) organisms were fully marine or estuarine throughout their life history (diadromous species were excluded); (b) studies of temporal changes, hybridisation or introgression from closely related species were omitted; (c) the study included at least three sampling sites from within the northeast Atlantic (Fig. \replaced{\ref{Atlantictopo} --}{1-} sites outside of this area were not considered); (d) datasets contained a minimum of five individuals per site and a total sample size of at least 50; and (e) the study included latitude and longitude of the sampling sites or a detailed description or map which provided sufficient detail to determine the geographical location of sample origins. Given the proven utility of mtDNA in comparative phylogeography (e.g. Patarnello et al. 2007; Ni et al. 2014) and the availability of these data in the public domain, a meta-analysis was conducted using published mtDNA gene sequences.

\subsubsection{2.2.2 Data reanalysis}
A standardised methodology was implemented to ensure that all mtDNA datasets were reanalysed in a consistent and directly comparable manner. Data analyses in the original studies were far from consistent, particularly with respect to the analysis of haplotype networks and historical demography. The majority of studies reported information about population structure; however, in several instances the studies included additional samples outside of the northeast Atlantic in their analysis. Therefore, standardised tests of population structure were undertaken \replaced{\textit{de novo}}{de novo} for each species. Sites that were genetically homogeneous (as described by the original authors) and which were spatially close or situated in the same geographical region were combined in some datasets. This ensured that phylogeography within and across seas was examined in this meta-analysis. Population differentiation was examined using global values of Jost's \textit{D} \citep{Jost2008} and \textit{F}\textsubscript{st} \citep{Weir1984} using the \texttt{fastDivPart} function from the R package diveRsity \citep{Keenan2013,RDevelopmentCoreTeam2008} and significance was assessed using 10,000 permutation replicates.

To examine the genealogical relationships within species, haplotype networks were constructed using the \texttt{haploNet} function from the R package pegas \citep{Paradis2010}. Tajima's \textit{D} \citep{Tajima1989}, Fu's \textit{F}\textsubscript{s} \citep{Fu1997} and Ramos-Onsins' \textit{R}\textsubscript{2} \citep{Ramos-Onsins2002} neutrality tests were performed in DnaSP v5.10 \citep{Librado2009} to determine whether each species carried a signal that deviated from neutrality (significance was assessed using 10,000 bootstrap replicates). Mismatch analyses (frequency of pairwise nucleotide-site differences between sequences) were carried out using the population growth-decline model in DnaSP to further examine the demographic history, and Harpending's raggedness index (\textit{r}) \citep{Harpending1994} was used to evaluate the fit of the observed distribution to the growth-decline model (10,000 bootstrap replicates). A non-significant index suggests that the observed data have a relatively good fit to the growth-decline model. In contrast, a significant index is indicative of a stable population which is typically thought to show a `ragged', multi-modal mismatch \citep{Harpending1994}.

The equation \textit{t} = $\tau/(2$\mu$\textit{k}) was used to estimate the timing of a population expansion (\textit{t}), where $\tau$ is the date of the expansion measured in units of mutational time (Tau -- estimated using DnaSP), $\mu$ is the mutation rate per site per year and \textit{k} is the sequence length. In addition, Bayesian Skyline Plots (BSPs) were run using BEAST2 v2.5.0 \citep{Drummond2005,Bouckaert2014}. BEAST2 uses a Markov chain Monte Carlo (MCMC) sampling procedure to estimate \textit{N}\textsubscript{e} through time based on the temporal distribution of coalescences in gene genealogies. For each dataset, the substitution model was selected using bModelTest \citep{Barido-Sottani2018}, which uses reversible jump MCMC that allows the Markov chain to jump between states representing different possible substitution models. A strict clock and a coalescent Bayesian Skyline prior was implemented. Each run consisted of 100 million steps with a burn-in of one million and parameters were sampled every 10,000 steps. Chain convergence and BSPs were analysed with Tracer v1.7.1 \citep{Rambaut2018}.

Recent studies have shown that the use of mutation rates derived from ancient calibration dates or from phylogenetic analyses may not be appropriate for studies at the population level (\citeauthor{Ho2008} \citeyear{Ho2008}, \citeyear{Ho2011}). In this study, therefore, mutation rates were chosen based on the most recent calibration date available for the closest taxonomic relative (Appendix A1). In published studies where a mutation rate was not specified, the genetic distance provided by the study was divided by the date of the calibration event (in Myr) to obtain a \% mutation rate per Myr. For cases where only calibration dates older than 5 Myr were available for the species and gene of interest, a three-fold correction in mutation rate was applied to the original rate to control for the potential time-dependency of molecular rates. This adjustment was implemented because rates have been found to vary by three to six-fold for several marine species when calibration dates younger than 5 Myr vs. older dates have been tested \citep{Crandall2012,Laakkonen2015}. A range of mutation rates based on the rates reported by previous studies were used to calculate a minimum, maximum and average time estimate since a population expansion.

%%% Table: mutation rates
%% end of table


\subsection{2.3 Results}
\subsubsection{2.3.1 Literature search}
The initial search using Boolean terms identified 1,120 articles, which was reduced to 56 articles after the titles and abstracts were examined and the search criteria were applied. The final database for the meta-analysis consisted of mtDNA gene sequence data from 21 studies (Table \ref{tab:metastudyinfo}); some studies from the previous step were not included due to the use of RFLPs in mtDNA or because some mtDNA datasets were not publicly available. The final database spanned several taxonomic groups, with fishes, molluscs and crustaceans accounting for the majority of species (81\%). The most common mtDNA gene across all studies was cytochrome oxidase I (COI), followed by cytochrome b (Cyt \textit{b}), the control region (CR) and the intergenic spacer region (IGS). COI was the most commonly used gene for invertebrate studies, IGS for macroalgae, and studies of fish used either the CR or the Cyt \textit{b} gene.

%%% Table: Summary of studies used in meta-analysis
\afterpage{
\newgeometry{left=30mm, right=25mm, top=17.5mm, bottom=17.5mm}
\begin{landscape}
\begin{table}[h!]
% \label{tab:metastudyinfo}
\centering
\begin{small}
\begin{threeparttable}
\caption[Meta-analysis study information]{List of the papers used in the meta-analysis and a summary of the information extracted from each study. \label{tab:metastudyinfo}}
\begin{tabular}[l]{lp{1.1cm}p{1.8cm}llp{1.3cm}l}
\hline \\[-1.0em]
\textbf{Taxon} & \textbf{mtDNA} & \textbf{No. sites;} & \textbf{Sampling site} & \textbf{Larval development} & \textbf{No. of} & \textbf{Reference}\\
\textbf{   Species} & \textbf{gene} & \textbf{\textit{N}} & \textbf{distribution} & & \textbf{lineages} & \\
\hline \\[-1.0em]
\textbf{Crustacean} & & & & & & \\
\textit{   Carincus maenas} & COI & 13; 200 & SW Spain to Norway & PLD, long & 1 & \citet{Roman2004} \\
\textit{   Maja brachydactyla} & COI & 13; 291 & SW Spain to W Ireland & PLD, 2-3 wk & 1 & \citet{Sotelo2008} \\
\textit{   Neomysis integer} & COI & 9; 379 & SW Spain to E Scotland & No PLD, brooder & 1 & \citet{Remerie2009} \\
\textit{   Palinurus elephas} & COI & 6; 119 & SW Spain to W Scotland & PLD, up to 1 yr & 1 & \citet{Palero2008} \\[3pt]

\textbf{Fish} & & & & & & \\
\textit{   Conger conger} & CR & 4; 232 & Azores to Ireland & Leptocephalus, up to 2 yr & 1 & \citet{Correia2012} \\
\textit{   Dicentrarchus labrax} & CR & 9; 93 & Bay of Biscay to Norway & PLD, 8-12 wk & 1 & \citet{Coscia2011} \\
\textit{   Labrus bergylta} & CR & 7; 279 & W Ireland to Norway & PLD, 37-49 d & 1 & \citet{DArcy2013} \\
\textit{   Pomatoschistus microps} & Cyt \textit{b} & 10; 232 & Bay of Biscay to Norway & PLD, 6-9 wk & 1 & \citet{Gysels2004} \\
\textit{   Pomatoschistus minutus} & Cyt \textit{b} & 8; 165 & S Portgual to Norway & PLD, unknown & 1 & \citet{Larmuseau2009} \\
\textit{   Raja clavata} & Cyt \textit{b} & 9; 315 & Azores to North Sea & No PLD, oviparous & 1 & \citet{Chevolot2006} \\
\textit{   Solea solea} & Cyt \textit{b} & 10; 645 & Bay of Biscay to Skagerrak & PLD, up to 3 wk & 1 & \citet{Cuveliers2012} \\
\textit{   Symphodus melops} & CR & 10; 263 & S Portugal to Skagerrak & PLD, 14-25 d & 1 & \citet{Robalo2012} \\[3pt]

\textbf{Macroalgae} & & & & & & \\
\textit{   Pelvetia canaliculata} & IGS & 15; 429 & Portugal to Norway & \deleted{No PLD, }\replaced{E}{e}xternal fertilisation & 1 & \citet{Neiva2014} \\[3pt]

\textbf{Mollusc} & & & & & & \\
\textit{   Cerastoderma edule} & COI & 12; 300 & Portugal to Norway & PLD, up to 4 wk & 1 & \citet{Krakau2012} \\
\textit{   Macoma balthica} & COI & 15; 339 & Bay of Biscay to North Sea & PLD, 2-5 wk & 2 & \citet{Becquet2012} \\
\textit{   Modiolus modiolus} & COI & 4; 73 & Irish Sea to Norway & PLD, up to 24 wk & 2 & \citet{Halanych2013} \\
\textit{   Nassarius nitidus} & COI & 3; 62 & NW Spain to Sweden & PLD, 4-8 wk & 1 & \citet{Couceiro2012} \\
\textit{   Nassarius reticulatus} & COI & 6; 156 & S Portugal to UK & PLD, 4-8 wk & 1 & \citet{Couceiro2007} \\[3pt]

\textbf{Polychaete} & & & & & & \\
\textit{   Owenia fusiformis} & COI & 11; 283 & Portugal to North Sea & PLD, up to 28 d & 3 & \citet{Jolly2006} \\
\textit{   Pectinaria koreni} & COI & 10; 289 & Portugal to North Sea & PLD, up to 15 d & 2 & \citet{Jolly2006} \\[3pt]

\textbf{Bryozoan} & & & & & & \\
\textit{   Celleporella hyalina} & COI & 9; 63 & NW Spain to Iceland & PLD, 1-4 h & 1 & \citet{Gomez2007} \\[3pt]
\hline
\end{tabular}
  \begin{tablenotes}
  \setlength\labelsep{0pt} % align all notes with table
    \scriptsize
    \item MtDNA, mitochondrial DNA; No. of sites, number of sampling sites; \textit{N}, number of individuals; PLD, pelagic larval duration.
  \end{tablenotes}
\end{threeparttable}
\end{small}
\end{table}
\end{landscape}
\restoregeometry
\clearpage
}
%% end of table

%%% Table: Summary statistics in meta-analysis
\afterpage{
\restoregeometry
\clearpage
% \begin{landscape}
% \newgeometry{left=15mm, right=15mm, top=20mm, bottom=20mm}
\begin{table}[h!]
\begin{scriptsize}
\caption[Meta-analysis summary statistics]{Summary statistics for each species. Population differentiation and demographic statistics are shown. In all statistical tests, significance was assessed using 10,000 permutations or bootstraps replicates.}
\label{tab:metastudyresults}
\begin{threeparttable}
\begin{tabular}[l]{lllp{0cm}lllll}
\hline \\[-1.0em]
\textbf{Species} & \multicolumn{2}{c}{\textbf{Differentiation}} & & \multicolumn{4}{c}{\textbf{Demography}} & & & & & \\
\cline{2-3} \cline{5-8} \\
& \textbf{Jost's \textit{D}} & \textbf{\textit{F}\textsubscript{st}} & & \textbf{Tajima's \textit{D}} & \textbf{\textit{F}\textsubscript{s}} & \textbf{\textit{R}\textsubscript{2}} & \textbf{\textit{r}} & \textbf{Expansion} \\
\hline \\[-1.0em]

\textbf{Crustacean} & & & & & & & & \\
\textit{   Carcinus maenas} & 0.584*** & 0.157*** & & -1.73* & -40.36*** & 0.034* & 0.018 & Yes \\
\textit{   Maja brachydactyla} & 0.298*** & 0.045*** & & -1.86** & -33.72*** & 0.028* & 0.030 & Yes \\
\textit{   Neomysis integer} & 0.956*** & 0.554*** & & 0.14 & -0.954 & 0.024 & 0.086 & No \\
\textit{   Palinurus elephas} & 0.023 & 0.000 & & -2.31*** & -30.19*** & 0.019* & 0.094 & Yes \\[3pt]

\textbf{Fish} & & & & & & & & \\
\textit{   Conger conger} & 0.124 & 0.000 & & -2.58*** & -211.1*** & 0.012*** & 0.031 & Yes  \\
\textit{   Dicentrarchus labrax} & 0.540* & 0.031* & & -1.88** & -21.52*** & 0.047* & 0.011 & Yes \\
\textit{   Labrus bergylta} & 0.672*** & 0.135*** & & -0.53 & -49.35*** & 0.074 & 0.024 & Yes \\
\textit{   Pomatoschistus microps} & 0.391*** & 0.385*** & & -1.39 & -17.90*** & 0.044 & 0.215 & Yes \\
\textit{   Pomatoschistus minutus} & 0.652*** & 0.100*** & & -1.96** & -90.56*** & 0.034* & 0.015 & Yes \\
\textit{   Raja clavata} & 0.375*** & 0.330*** & & -0.09 & -2.340 & 0.076 & 0.309 & No \\
\textit{   Solea solea} & 0.049 & 0.002 & & -2.02*** & -131.9*** & 0.021** & 0.221 & Yes \\
\textit{   Symphodus melops} & 0.578*** & 0.349*** & & -1.70* & -50.52*** & 0.032* & 0.086 & Yes \\[3pt]

\textbf{Macroalgae} & & & & & & & & \\
\textit{   Pelvetia canaliculata} & 0.689*** & 0.482*** & & -1.53* & -19.02*** & 0.036 & 0.043 & Yes \\[3pt]

\textbf{Mollusc} & & & & & & & & \\
\textit{   Cerastoderma edule} & 0.662*** & 0.304*** & & -2.24*** & -34.47*** & 0.019** & 0.033 & Yes \\
\textit{   Macoma balthica} & 0.702*** & 0.470*** & & -- & -- & -- & -- & -- \\
{   lineage 1} & \added{0.551***} & \added{0.434***} & & -0.80 & -3.773 & 0.053 & 0.241 & No \\
{   lineage 2} & \added{0.007} & \added{0.000} & & -0.99 & -1.110 & 0.089 & 0.173 & No \\
\textit{   Modiolus modiolus\textsuperscript{a}} & 0.083 & $<$0.001 & & -1.79* & -11.91*** & 0.045* & 0.156 & Yes \\
\textit{   Nassarius nitidus} & 0.222*** & 0.302*** & & -1.49* & 0.028 & 0.049* & 0.446 & No \\
\textit{   Nassarius reticulatus} & 0.047 & 0.000 & & -2.51*** & -48.33*** & 0.016** & 0.080 & Yes \\[3pt]

\textbf{Polychaete} & & & & & & & & \\
\textit{   Owenia fusiformis} & 0.788*** & 0.055*** & & -- & -- & -- & -- & -- \\
{   lineage 1} & \added{0.636} & \added{0.001} & & -2.34*** & -114.8*** & 0.024** & 0.020 & Yes \\
{   lineage 2} &\added{0.734} & \added{0.012} & & -2.06** & -55.00*** & 0.030** & 0.008** & Yes \\
{   lineage 3} & \added{0.050} & \added{0.000} & & -1.26 & -3.934** & 0.084 & 0.080 & Yes \\
\textit{   Pectinaria koreni} & 0.596*** & 0.112*** & & -- & -- & -- & -- & -- \\
{   lineage 1} & \added{0.638*} & \added{0.024**} & & -1.99** & -76.48*** & 0.027** & 0.021 & Yes \\
{   lineage 2} & \added{0.390} & \added{0.050} & & -2.63*** & -54.02*** & 0.018*** & 0.029* & Yes \\[3pt]

\textbf{Bryozoan} & & & & & & & & \\
\textit{   Celleporella hyalina} & 0.513*** & 0.488*** & & -1.35 & -0.554 & 0.063 & 0.061 & No \\[3pt]
\hline
\end{tabular}
  \begin{tablenotes}
  \setlength\labelsep{0pt} % align all notes with table
    \tiny
    \item *$<$0.05, **$<$0.01, ***$<$0.001.
    \item \textit{F}\textsubscript{s}, Fu's \textit{F}\textsubscript{s}; \textit{R}\textsubscript{2}, Ramos-Onsins' \textit{R}\textsubscript{2}; \textit{r}, Harpending's raggedness index.
    \item \textsuperscript{a}Only statistics for lineage 1 are shown.
  \end{tablenotes}
\end{threeparttable}
\end{scriptsize}
\end{table}
% \restoregeometry
% \end{landscape}
\clearpage
}
%% end of table


\subsubsection{2.3.2 Genetic structure}
Sixteen species showed significant global Jost's \textit{D} and \textit{F}\textsubscript{st} values, indicative of population differentiation (Table \ref{tab:metastudyresults}), while the remaining five species showed little evidence of population differentiation. Across the 21 datasets, four different types of haplotype network (Fig. \ref{hapnetworks}) were putatively identified based on the structure of the networks:

(i) A `Star' network (Fig. \ref{hapnetworks}a), in which a single, widespread haplotype is typically positioned at the centre of the network and is thought to be the ancestral haplotype. Additional haplotypes are linked to this dominant haplotype by a single (or a few) mutational step(s), suggesting these haplotypes are the product of recent mutation events. Eight species showed this type of relationship (\textit{Celleporella hyalina}, \textit{Conger conger}, \textit{Nassarius nitidus}, \textit{Nassarius reticulatus}, \textit{Palinurus elephas}, \textit{Pelvetia canaliculata}, \textit{Pomatoschistus microps} and \textit{Raja clavata}). In one case, the dominant haplotype had far fewer connections than a low-frequency haplotype in the network, making it difficult to distinguish the centre of the network with confidence (\textit{Pomatoschistus microps});

(ii) A `Complex star' network (Fig. \ref{hapnetworks}b), in which there are multiple high-frequency haplotypes and connections. Six species showed this type of relationship (\textit{Carcinus maenus}, \textit{Cerastoderma edule}, \textit{Maja brachydactyla}, \textit{Pomatoschistus minutus}, \textit{Solea solea}, \textit{Symphodus melops});

(iii) A `Reciprocally monophyletic' network (Fig. \ref{hapnetworks}c), in which more than one lineage is apparent and each lineage is linked by a long branch associated with numerous mutations. Four species showed this type of relationship (\textit{Macoma balthica}, \textit{Modiolus modiolus}, \textit{Owenia fusiformis} and \textit{Pectinaria koreni});

(iv) A `Complex mutational' network (Fig. \ref{hapnetworks}d), in which some branches were separated by a very large number of mutations, while other branches had contrarily one or two mutations. Three species showed this type of relationship (\textit{Dicentrarchus labrax}, \textit{Labrus bergylta} and \textit{Neomysis integer}). In most cases, a dominant haplotype was present and was presumed to be the ancestral form. However, \textit{Neomysis integer} presented an unusual network in which a distinct ancestral haplotype was not apparent and the centre of the haplotype network was not readily distinguishable. Haplotype networks for all species are available in the Appendix (A2).

%% Figure: Haplotype networks
\begin{figure}[h!]
\centering
\includegraphics[width=0.95\textwidth]{Chapter_figures/chapter2/Figure_2_Haplotype_Networks_Vect.pdf}
\caption [Hapotype networks showing four different network structures]
{Haplotype networks showing four different network structures: (a) `star' (\textit{Palinurus elephas}), (b) `complex star' (\textit{Carcinus maenas}), (c) `reciprocally monophyletic' (\textit{Macoma balthica}) and (d) `complex mutational' (\textit{Dicentrarchus labrax}). Each circle represents a unique haplotype and the sizes of the circles are proportional to the haplotype frequencies for each network but are not comparable across studies. Each line represents one mutation step and two or more steps are indicated by bars or numbers. Colours inside the circles correspond to sites which have individuals represented in that particular haplotype.}
\label{hapnetworks}
\end{figure}
%% end of figure

\subsubsection{2.3.3 Historical demography}
Historical demography was inferred for each species based on the observed mismatch distribution, neutrality tests and the raggedness index (Table \ref{tab:metastudyresults}). Four main types of mismatch distributions were observed: unimodal, skewed unimodal, multimodal and bimodal (Fig. \ref{mismatch}). Unimodal is associated with a sudden population expansion (e.g. \textit{Maja brachydactyla}; Fig. \ref{mismatch}a), and skewed unimodal is generally associated with a recent expansion or bottleneck (e.g. \textit{Nassarius reticulatus}; Fig. \ref{mismatch}b). Multimodal (e.g. \textit{Labrus bergylta}; Fig. \ref{mismatch}c) and bimodal (e.g. \textit{Macoma balthica}; Fig. \ref{mismatch}d) are usually associated with constant population size. However, previous research has suggested that bimodal peaks may indicate the presence of two distinct lineages (e.g. Alvarado-Bremer et al. 2005), which would potentially violate the assumptions of coalescent theory if analysed as one `genetic' population. In this case, the first peak would represent intra-clade pairwise differences, whereas the second peak would likely represent more ancient inter-clade pairwise differences (Fig. \ref{mismatch}D). For each instance of bimodality, the haplotype network was inspected for evidence of two or more lineages. The networks indicated that more than one distinct lineage was evident for all bimodal mismatches (\textit{Macoma balthica}, \textit{Modiolus modiolus}, \textit{Owenia fusiformis} and \textit{Pectinaria koreni}) and, therefore, mismatch analysis and neutrality tests were carried out on each lineage separately. These analyses were not conducted for lineage 2 of \textit{Modiolus modiolus} due to the small number of individuals (\textit{N} = 3) comprising this lineage. Mismatch distributions for all species are available in the Appendix (A3).

%% Figure: Mismatch distributions
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter2/Figure_3_Mismatch_graphs_Vect.pdf}
\caption [Mismatch analyses showing four different distributions]
{Mismatch distributions showing four different distributions: (a) unimodal (\textit{Maja brachydactyla}), (b) skewed unimodal (\textit{Nassarius reticulatus}), (c) multimodal (\textit{Labrus bergylta}) and (d) bimodal (\textit{Macoma balthica}). Unimodal and skewed unimodal distributions are generally associated with a sudden expansion and a recent sudden expansion, respectively. Multimodal and bimodal are thought to be associated with a constant population size (but see text). Bars represent the frequency of pairwise nucleotide differences between individuals. Curves correspond to the expected distribution fitted to the data under a model of constant population size (solid line) or demographic expansion (dotted line).}
\label{mismatch}
\end{figure}
%% end of figure

Neutrality statistics for testing the drift-mutation equilibrium (Tajima's \textit{D}, \textit{F}\textsubscript{s} and \textit{R}\textsubscript{2}) were found to be contrasting between species (Table \ref{tab:metastudyresults}). These tests tended to be significant for species that showed a star-shaped network and for which the mismatch graph was unimodal or skewed unimodal. This supported evidence that a signal of rapid population expansion was detected; however, a selective sweep can also produce the same genetic signal. Harpending's \textit{r} suggested that two datasets departed from a model of demographic expansion (Table \ref{tab:metastudyresults}), but inspection of the mismatch graphs and neutrality tests indicated there was strong evidence to support a rapid population expansion (or selective sweep) in both datasets. No signatures of rapid population expansion were detected in five species (\textit{Celleporella hyalina}, \textit{Macoma balthica}, \textit{Nassarius nitidus}, \textit{Neomysis integer} and \textit{Raja clavata}), suggesting a stable constant population size.

%% Figure: Expansion esimates
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter2/Figure_4_Expansions_Ink.png}
\caption [Estimated dates of expansion for species or lineages]
{Estimated dates of expansion for species or lineages (L) in which the demographic expansion hypothesis was not rejected. A minimum and maximum time since expansion is plotted as horizontal bars for some datasets, estimated from a minimum and maximum mutation rate (Appendix A1). The beginning of the last glacial period (dotted line) and the estimated time-frame of the Last Glacial Maximum (grey shaded area) are displayed. Species are organised by taxa: crustaceans, \textit{Carcinus maenas} - \textit{Palinurus elephas}); fish, \textit{Conger conger} - \textit{Symphodus melops}; macroalgae, \textit{Pelvetia canaliculata}; molluscs, \textit{Cerastoderma edule} - \textit{Nassarius reticulatus}; polychaetes, \textit{Owenia fusiformis} - \textit{Pectinaria koreni}.}
\label{expansion}
\end{figure}
%% end of figure

For the remaining 19 datasets (16 species, 19 including lineages), a historic population expansion was assumed and the timing of the expansion was estimated (Fig. \ref{expansion}). All expansions were found to take place during the Pleistocene or the Holocene epoch. Estimated timings for 17 datasets were after or overlapped the earliest estimate for the LGM ($\sim$26.5 Ka). Expansion estimates for one fish (\textit{Labrus bergylta}) and one lineage of the polychaete \textit{Owenia fusiformis} pre-dated the LGM but were still positioned during the last glacial period. Bayesian Skyline Plots (Fig. \ref{bsp}) were generally consistent with the results from the mismatch analyses. Among the 17 datasets for which the mismatch analyses estimated the time of an expansion to have occurred after the LGM, a rise in \textit{N}\textsubscript{e} post-LGM was apparent in 15 of these datasets, but the strength of the increase varied across datasets. In comparison to the mismatch analysis, the BSP for \textit{L. bergylta} (Fig. \ref{bsp}f) and \textit{O. fusiformis} lineage 2 (Fig. \ref{bsp}p) indicated a population expansion after the earliest estimate for the LGM as opposed to pre-dating the LGM. In addition, although the mismatch analyses inferred a post-LGM expansion for \textit{M. modiolus} lineage 1 (Fig. \ref{bsp}m) and \textit{O. fusiformis} lineage 3 (Fig. \ref{bsp}q), BSPs generally suggested \textit{N}\textsubscript{e} was constant after the LGM.

%% Figure: Bayesian Skyline plots
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter2/Figure_5_BSPs.pdf}
\caption [Bayesian Skyline Plots for species or lineages]
{Bayesian Skyline Plots for species or lineages (L) in which the demographic expansion hypothesis was not rejected. Solid black lines show the median effective population size over time (\textit{N}\textsubscript{e} = effective population size and \textit{T} = generation time); dashed black lines represent the 95\% confidence intervals. The estimated time-frame of the Last Glacial Maximum is denoted by the area shaded dark grey. Species are organised by taxa: crustaceans, \textit{Carcinus maenas} (a), \textit{Maja brachydactyla} (b), \textit{Palinurus elephas} (c); fish, \textit{Conger conger} (d), \textit{Dicentrarchus labrax} (e), \textit{Labrus bergylta} (f), \textit{Pomatoschistus microps} (g), \textit{P. minutus} (h), \textit{Solea solea} (i), \textit{Symphodus melops} (j); macroalgae, \textit{Pelvetia canaliculata} (k); molluscs, \textit{Cerastoderma edule} (l), \textit{Modiolus modiolus} lineage 1 (m), \textit{Nassarius reticulatus} (n); polychaetes, \textit{Owenia fusiformis} lineage 1 (o), \textit{O. fusiformis} lineage 2 (p), \textit{O. fusiformis} lineage 3 (q), \textit{Pectinaria koreni} lineage 1 (r), \textit{P. koreni} lineage 2 (s).}
\label{bsp}
\end{figure}
%% end of figure

\subsection{2.4 Discussion}
The results of this study show a range of contemporary genetic patterns across the coastal marine taxa analysed in the northeast Atlantic. In general, genealogical patterns were not uniform within taxonomic groups, though common patterns were observed in both polychaete species, which implies that historical events may have affected these polychaete species similarly. Most species (76 \%) showed evidence of population structuring, suggestive of restricted contemporary or historical gene flow between the sites studied. Of the species that exhibited no population differentiation, all five species have a pelagic larval phase, with a pelagic larval duration (PLD) ranging from up to three weeks (\textit{S. solea}) to a year or more (\textit{P. elephas} and \textit{C. conger}) (Table \ref{tab:metastudyinfo}). However, most of the species that demonstrated significant population differentiation also had a pelagic larval phase, ranging from a relatively short PLD of 1-4 h (\textit{C. hyalina}) to a relatively long PLD of 8-12 weeks (\textit{D. labrax}) (Table \ref{tab:metastudyinfo}). Although speculative, taken altogether, this may suggest that larval development and PLD could be important factors in maintaining gene flow in some, but not all, of these species; however, more evidence is needed to confirm this. Indeed, whether a general correlation exists between PLD and genetic differentiation measures remains unclear because some studies have reported poor correlations between the two \citep{Weersing2009,Kelly2010,Riginos2011}, while other studies have reported the opposite \citep{Siegel2003,Selkoe2011} suggesting that PLD and genetic metrics can indeed reflect scales of dispersal if the sampling design is robust \citep{Selkoe2011}. \added{Other factors could also explain a lack of correlation between PLD and genetic differentiation such as: (i) temporal and spatial fluctuations in PLD within a species, (ii) habitat availability and suitability, and (iii) delayed metamorphosis (\citeauthor{Pechenik1990} \citeyear{Pechenik1990}, \citeyear{Pechenik2006}).} As a result, speculative relationships between PLD and genetic differentiation should be interpreted with caution.

In some of the species studied, certain geographical areas were dominated by a particular haplotype that was rarely or not present in other areas across the sampled range. For example, the green crab \textit{Carcinus maenas} showed highly significant differentiation and distinctive haplotypes in the Faroe Islands and Iceland, a pattern detected by the original authors who subsequently concluded that a deep-water barrier to dispersal in green crabs was the driver of this pattern \citep{Roman2004}. A similar pattern was also observed for two species around western Ireland in the northeast Atlantic. In \textit{Celleporella hyalina} and \textit{Macoma balthica}, distinct haplotypes composed a population around western Ireland; however, unique haplotypes were not apparent in other species analysed in this study with similar sampling coverage (e.g. \textit{Labrus bergylta}, \textit{Palinurus elephas} and \textit{Pelvetia canaliculata}). A discrepancy in genetic structure between species at this spatial scale has also been observed between two temperate octocoral species (\textit{Eunicella verrucosa} and \textit{Alcyonium digitatum}) using microsatellite markers, whereby northwest Ireland samples were found to be genetically isolated from other northeast Atlantic samples in \textit{E. verrucosa}, but not in \textit{A. digitatum} \citep{Holland2017}. This suggests that historical or contemporary gene flow between areas in the northeast Atlantic and western Ireland is likely possible, but in some cases the spatial patterns of genetic structure could be influenced by other processes such as strong selection pressures, species-specific life history traits, demographic fluctuations, or range expansions occurring at different times in different species \citep{Hellberg2009}.

\subsubsection{2.4.1 Demographic history}
Demographic history was variable across species in the northeast Atlantic, as evidenced by both the diverse structuring of the haplotype networks and the observed mismatch distributions within species. The presence of one or more lineages and the complexity of mutational patterns in several networks suggested some species have undergone pronounced changes in their demography and genealogy. Connections with large mutation steps separating some haplotypes are indicative of deep phylogenetic splits in the genealogies and suggests the persistence of old populations in these species. Accumulating new mutations is a relatively slow process and, therefore, sufficient time since coalescence must have elapsed to facilitate these large sequence divergences \citep{Avise2009}.

In the northeast Atlantic, the LGM has often been viewed as a possible explanation for discrepancies in genealogies and for rapid population expansions via recolonisation as glaciers started to retreat from their maximum positions \citep{Hewitt2004}. In this study, we detected rapid expansions in many different taxa, of which the majority were estimated to occur after the LGM. This supports evidence for post-LGM expansions, possibly from periglacial refugia \citep{Maggs2008} or via recolonisation of areas previously affected by the Northern Hemisphere ice sheets. These results are in contrast to the northeast Pacific where regional persistence during the LGM appeared to be common in rocky-shore organisms \citep{Marko2010}. The conclusions of several previous studies reanalysed in this meta-analysis also detected rapid expansions \citep[e.g.][]{Jolly2006,Sotelo2008,Larmuseau2009}; however, the authors of these studies estimated the dates of these expansions to have occurred pre-LGM. This discrepancy could be due to the differences in mutation rates, whereby the original authors typically used rates derived from ancient calibrations, while in this study we attempted to use more recent calibration dates to correct for the potential time-dependency of molecular rates \citep{Ho2011}.

Of course, we acknowledge that the signal of deviation from neutrality we detected may, in some cases, be the result of a selective sweep and not a rapid expansion. This signal could be distinguished by incorporating multi-locus data; nevertheless, given that a variety of species in this study showed similar genealogical patterns consistent with demographic expansion, it seems likely that most of them did indeed experience demographic changes associated with the end of the LGM, rather than selective sweeps. Moreover, distinctive haplotypes were found in several species networks (\textit{Pelvetia canaliculata}, \textit{Pomatoschistus minutus}, \textit{Owenia fusiformis} and \textit{Pectinaria koreni}) to the south of where the Eurasian ice sheet is proposed to have extended during the LGM (Fig. \ref{Atlantictopo}). This finding suggests populations of these species may have survived in southern glacial refugia; though, as pointed out by some of the original authors, deep sequence divergences in some species (e.g. \textit{O. fusiformis} and \textit{P. koren\replaced{i}{a}}) and the lack of a species-specific molecular clock calibration makes inferences about refugia challenging \replaced{(\citeauthor{Jolly2005} \citeyear{Jolly2005}, \citeyear{Jolly2006})}{\citep{Jolly2005,Jolly2006}}.

It is difficult to suggest an explanation for the two expansions estimated to have pre-dated the LGM (using mismatch analysis), but which fall within the last glacial period. This pattern of pre-LGM expansion has also been reported in a number of previous studies for a variety of marine taxa \citep[e.g.][]{Hoarau2007,Marko2010,Ni2014,Almada2017}. One potential explanation for this pattern is that sea level during the last glacial cycle did not decrease uniformly towards the level observed at the LGM, but oscillated rapidly over a period of 60 Ka to 30 Ka \citep[see Fig. 3a in][]{Lambeck2002}. Therefore, it may be possible that we are detecting the signature of a population expansion during one of these sudden increases in sea level during the last glacial period. Alternatively, as the BSP analysis inferred a post-LGM expansion for these two datasets, this could be a limitation associated with the mismatch analysis approach, which does not consider genealogy, and may, therefore, produce a less precise estimation. In addition, the sample of genetic diversity for this species may not be representative \citep{Karl2012} or the genetic signal we detected may have been the result of a selective sweep and not a rapid expansion.

The use of single marker mtDNA genealogies and coalescence theory can introduce challenges associated with the interpretation of data and these limitations should be acknowledged \citep{Karl2012}. For example, the populations under study may have experienced multiple episodes of growth and decline; however, only the most recent expansion event can be detected using coalescence analysis and, in some cases, these events may not be sufficiently severe to be detected \citep{Karl2012}. In addition, coalescent histories can differ amongst loci because they can experience mutation and drift independently. Therefore, analysis of a single gene only gives insight into the coalescent history of that locus, which may not always be representative of population history. Analysis of multiple loci and genomics would help to alleviate these concerns, and would likely provide enhanced resolution for exploring the phylogeography of northeast Atlantic marine fauna.

Although population expansions were detected in a number of species in this study and also in the wider literature, populations of other marine species, including five from this study, have been found to remain stable throughout the LGM. As previously reported, not all coastal marine taxa appear prone to demographic changes during or after ice ages \citep{Janko2007,Marko2010,Olsen2010}. It is also important to acknowledge that earlier events in the Pleistocene and more ancient events that pre-date the Pleistocene may have helped shape the contemporary patterns of genealogical structure observed in this study.


\subsubsection{2.4.2 Implications for conservation}
\added{Conservation and management plans are crucial for mitigating the loss of biodiversity from the effects of climate change and anthropogenic stressors in the marine environment.  However, conservation plans have typically prioritised protecting contemporary patterns of biodiversity and seldom considered the evolutionary processes that generated them \citep{Wright2015}.  As a result, genetic data have rarely been used to inform management, despite their relevance in conservation for describing/inferring patterns of genetic diversity, dispersal ability and evolutionary history, and for delineating cryptic species and evolutionary significant units \citep{Beger2014}.}

\added{Identifying discrete populations with high genetic diversity and detecting phylogeographic breaks to gene flow is of high importance for managers of MPA networks and conservation \citep{Allendorf2010}.  Summarising broad patterns of genetic diversity and population structure across a broad range of marine species can facilitate the detection of major genetic breaks/barriers to gene flow and identify genetic diversity unique to specific areas.  Such genetic diversity was found in several species analysed in this study, particularly in Iberian and western Ireland sites, and implies that marine conservation in the northeast Atlantic should attempt to address these areas of unique genetic diversity which may be species-specific. This supports previous suggestions that managers should attempt to consider data from multiple species to position MPAs when trying to maximise ecological connectivity in their network \citep{Marti-Puig2013,Pascual2017}.}

\added{Although the results of this study are based on data from a single mtDNA gene and 21 species (albeit across a range of phyla), the patterns detected and discussed throughout this study may be informative as they represent the contemporary genealogical spatial structure of the matrilineal lineages and provide information about the evolutionary histories of these species and about potential cryptic species.  While challenges exist in the interpretation of these data, conducting a meta-analysis of comparative phylogeography using multiple species can identify common signals of phylogeography, which can give clues about how populations of benthic marine species may respond to future changes in the environment.}


\subsubsection{2.4.3 Conclusions}
The findings of this meta-analysis indicate that species in the northeast Atlantic do not show a uniform pattern of phylogeography, but rather a mixture of complex contemporary genealogical structure. Reanalysis of demographic histories indicated that a large proportion of the species included in this study have experienced post-LGM expansions, supporting the general expectation that rapid population expansions occurred after the LGM as the ice sheets started to retreat (\citeauthor{Hewitt2000} \citeyear{Hewitt2000}, \citeyear{Hewitt2004}). This suggests that regional extirpation during the LGM appears to be a common biogeographic history for many northeast Atlantic marine taxa. However, improvements in mutation rate estimates, as well as the incorporation of multi-locus markers and genomics, would likely provide greater accuracy and resolution for overcoming the challenges associated with single mtDNA genealogies, and for improving our understanding of phylogeography in the northeast Atlantic Ocean.


%--------------
%
% Chapter 3: Which taxa
% 
%--------------
\newpage
\section{Chapter 3: Selecting taxa to study genetic connectivity between Marine Protected Areas}
\rule{\textwidth}{0.8pt}
\vspace{1pt}

\noindent This chapter is based on a paper published in the journal \textit{Marine Policy}.  The reference is given below and the full paper is available in the Appendix. \\

\noindent Jenkins TL, Stevens JR (2018) Assessing connectivity between MPAs: selecting taxa and translating genetic data to inform policy. \textit{Marine Policy} \textbf{94}, 165-173.

\newpage
\subsection{3.0 Abstract}
\noindent Connectivity is frequently cited as a vital component of Marine Protected Area (MPA) networks and was formally identified as one of five key principles for marine network design in European waters. Yet, without the ability to demonstrate connectivity, it is impossible to be certain that sites designated within a MPA network do in fact constitute a network, when they may -irrespective of the diversity and rarity of the taxa within them- be in reality a set of unlinked habitats and associated species assemblages. However, the process of assessing connectivity between MPAs, and which taxa to include in assessments of connectivity, is often difficult and can be dependent on a variety of factors that may be beyond the control of managers, stakeholders and policymakers.  In this chapter, a set of biological and methodological factors are highlighted, consideration of which may help to inform the selection of species for assessments of genetic connectivity between MPAs in a network. After cogitating these factors, two benthic species with differing life histories were selected as candidates for assessing connectivity between UK MPAs: the pink sea fan (\textit{Eunicella verrucosa}) and the European lobster (\textit{Homarus gammarus}). Exploring the population structure and genetic connectivity in these species will provide an empirical assessment of connectivity between MPAs; in addition, due to the important ecological and economical status of these two species, the results have the potential to inform the relevant conservation and fisheries management bodies. 

\subsection{3.1 Introduction}
\noindent Connectivity is identified as a key component in the design of European Marine Protected Area (MPA) networks \citep{OSPARCommission2013}. However, changes to the definition of connectivity outlined in many different reports (\citeauthor{OSPARCommission2010} \citeyear{OSPARCommission2010}, \citeyear{OSPARCommission2013}; \citeauthor{Carr2014} \citeyear{Carr2014}) suggest there is potential confusion or conflict amongst stakeholders and scientists concerning the exact definition and function of connectivity in the context of MPA networks. The most simplistic definition is taken from \citet{Palumbi2003} whereby ``\textit{connectivity is the extent to which populations in different parts of a species range are linked by the movement of eggs, larvae or other propagules, juveniles or adults}'' \citep{OSPARCommission2013}.  In contrast, other reports have outlined a more detailed definition such that maintaining connectivity involves creating ``...\textit{ecologically connected and functional networks with `corridors' or `stepping stones' that facilitate the range shifts of populations and the movements of individuals and genes in response to ocean climate change}'' \citep{OSPARCommission2010}, or that ``...\textit{the MPA network is well distributed in space and takes into account the linkages between marine ecosystems}'' \citep{Carr2014}. 

Knowledge of connectivity is fundamental for optimising the location and size of MPAs to create a well-connected network (instead of individual unrelated MPAs) \citep{Jones2007,Almany2009}, and for evaluating the impacts of exploitation on the population dynamics of commercial marine species \citep{Bernatchez2017}. To understand connectivity, an ideal scenario might incorporate multiple sources of data informing on connectivity from many types of taxa within the boundaries of an MPA network; however, this is often impossible due to financial and logistical constraints. Instead, managers of MPAs have typically concentrated their efforts on species that are endangered or rare, and which may be on the brink of extirpation in parts of their range, or on so-called `umbrella', `keystone' or `flagship' species \citep{Simberloff1998,Kalinkat2017}. The concept of an umbrella species, a species whose protection indirectly protects many other species in an ecological community, is generally recognised as appealing for assessing connectivity. This is because the establishment of a network based on such data may extrapolate the benefits of preserving the connectivity of one focal species to other species in a community with similar life histories and dispersal traits. Hypothetically, a species associated with all three concepts (umbrella, keystone or flagship) would likely be the `holy grail' species for studying connectivity between MPAs, but identification of such species (if indeed they exist) has continued to elude those involved in marine conservation. Moreover, for a variety of reasons (Table \ref{tab:factors}), the study of species that come close to satisfying the criteria of a `holy grail' species may not be feasible and, therefore, compromises are needed to facilitate the collection of data that are informative about connectivity in a given system.

In this chapter, a number of biological and methodological factors are highlighted that should be considered before selecting taxa to assess genetic connectivity between MPAs.  Subsequently, based on the set of factors highlighted, and existing knowledge of northeast Atlantic coastal taxa, two species are selected to explore spatial genetic structure and connectivity between British MPAs and across the wider northeast Atlantic.

\subsection{3.2 Selecting taxa}
\noindent The selection of appropriate taxa to use as surrogates for assessments of genetic connectivity between MPAs has seldom been discussed in the literature \citep[but see][]{Marti-Puig2013}.  Coastal benthic marine invertebrates are often good candidates because they can be relatively abundant with large ranges, and dispersal is typically defined during a pelagic phase undertaken by an early life stage (e.g. eggs or larvae), while the adults remain relatively sedentary \citep{Cowen2009}.  This type of development means connectivity is mainly dependent on local hydrological conditions (as well as species-specific traits) and, therefore, better reflects natural patterns of connectivity, as opposed to studying connectivity driven by organismal behaviour in motile and migratory species.  Since patterns of genetic connectivity can vary between species over similar geographical areas \citep{Coleman2011,Holland2017}, it is also important to consider assessing connectivity in more than one species with differing biology/ecology. This allows species-specific genetic connectivity and patterns of connectivity common across taxa to be examined \citep{Cowen2007,Marti-Puig2013}.


\subsubsection{3.2.1 Biological factors}
\noindent Some biological features of candidate species can inevitably enhance the public appeal and societal impact of a study, while other features can limit the collection of samples and the interpretation of data generated by genetic markers (Table \ref{tab:factors}).  For the purpose of promoting marine conservation, charismatic megafauna such as marine mammals and sharks frequently dominate awareness campaigns (flagship species) because they can raise funds and change public opinions and behaviour. Although many of these species may not be the best candidates for assessing MPA connectivity, these enigmatic animals are typically well-known by the wider public and benefit from a greater awareness and potential impact than other marine fauna. As a result, if a candidate species is poorly known to the public community, highlighting its importance for the conservation of an associated enigmatic species may have an equivalent effect \citep[e.g. the interactions between kelp forests and sea otters,][]{Lubchenco2016}. 

%%% Table: Factors to consider choosing taxa
\afterpage{
\clearpage
\newgeometry{left=30mm, right=20mm, top=20mm, bottom=20mm}
\begin{landscape}
\centering
\begin{scriptsize}
\begin{longtable}[c]{p{4cm}LLR}
\caption[Factors to consider before selecting species to assess genetic connectivity between Marine Protected Areas]{Biological and methodological factors to consider before selecting a species to assess genetic connectivity between Marine Protected Areas.}
\label{tab:factors}
\hline \\[-1.0em]
\footnotesize{\textbf{Factor}} & \footnotesize{\textbf{Description}} & \footnotesize{\textbf{Example}} & \footnotesize{\textbf{Significance}}\\
\hline \\[-1.0em]
\textbf{\textit{Biological}} & & & \\
\rowcolor{Grey}
Ecological importance & Does the organism have a fundamental importance to a functioning ecosystem? & Ecosystem engineers (e.g. mussel beds). & These species may be protected under legislation. Higher potential impact.\\
Flagship species & Is the species charismatic, well known to the public, and a poster child of conservation campaigns? & Many large megafauna including cetaceans and sharks. Some threatened invertebrates. & Greater public awareness/interest. Higher potential impact.\\
\rowcolor{Grey}
Economic importance & Is the species commercially exploited? & Fish and invertebrate coastal fisheries. & Opportunities for collaboration during sample collection. \\
Taxonomy & Is the taxonomy not well resolved and the organism hard to identify? & Sister species with very similar morphology. Very small organisms. & Species difficult to ID morphologically demand more resources and time. In some cases, a taxonomist or DNA barcoding method may be required for validation.\\
\rowcolor{Grey}
\replaced{Biology and habitat}{Biological} knowledge & A sound knowledge of the biology and ecology of the organism. & Habitat / distribution, larval development, dispersal, etc. & Improve the testing of hypotheses. Improve interpretation of the results.\\
 & & & \\
\textbf{\textit{Methodological}} & & & \\
Sample collection & Collecting tissue samples from the organism for genetic analysis. & Feasibility / cost of collecting samples. & Protected species may require permits for tissue removal. Logistical barriers may limit sample collection in some areas (e.g. deep sea). \added{Non-destructive tissue sampling advantagous for endangered or rare species.} \\
\rowcolor{Grey}
Sample sites & The number of sampling sites and the spatial separation between sites. & Consider the number of sites needed in and around MPAs. & Adequate sampling sites in and out of MPAs could enhance hypothesis testing. \\
Sample sizes & The number of individuals per sampling site. & Consider the number of individuals needed to draw robust conclusions. & The number of individuals can be influenced by the choice of genetic marker. During data analysis, the power of the markers and sample sizes can be tested using various software. \\
\rowcolor{Grey}
DNA extraction & Extracting genomic DNA for analysis. & Consider tissue type and extraction protocol before sampling. & High quantity and quality DNA can be difficult to extract from some organisms and tissue types (e.g. crustacean exoskeleton) using standard kits. \\
Choice of genetic marker & Choosing a genetic marker that is polymorphic enough to investigate genetic patterns. & Microsatellites / SNPs. & The choice and number of markers will depend on the power and resolution required. \\
\rowcolor{Grey}
Availability of genetic markers & Are panels of markers already available for the organism? & Microsatellite / SNP panels. & This would avoid the need to develop markers \textit{de novo}. \\
\hline
\end{longtable}
\end{scriptsize}
\end{landscape}
\restoregeometry
\clearpage
}
%% end of table

%%% Table: Selecting seafans and lobsters Using framework criteria
\afterpage{
\clearpage
\begin{landscape}
\centering
\begin{scriptsize}
\begin{longtable}[c]{p{6cm}RR}
\caption[Selecting two species using an assessment framework]{Summary of the factors considered from the framework outlined in Table \ref{tab:factors} for the pink sea fan (\textit{Eunicella verrucosa}) and the European lobster (\textit{Homarus gammarus}).} \\
\hline \\[-1.0em]
\textbf{\footnotesize{Factor}} & \textbf{\footnotesize{\added{Pink sea fan}}} & \textbf{\footnotesize{\added{European lobster}}} \\
\hline \\[-1.0em]
\textbf{\textit{Biological}} && \\
\rowcolor{Grey}
Ecological importance & IUCN Red Listed species and is specifically protected in English and Welsh waters. Is considered a flagship species in UK conservation. & Is a charismatic crustacean in the UK. \\
Flagship species & \textquotedbl \space \textquotedbl & \textquotedbl \space \textquotedbl \\
\rowcolor{Grey}
Economic importance & Not commericially exploited. & Active fishery that supports many coastal communities. \\
Taxonomy & Well resolved. & Well resolved. \\
\rowcolor{Grey}
Biology and habitat knowledge & Distribution and habitat well known. & Distribution, habitat, larval development and reproduction well known. \\
\textbf{\textit{Methodological}} && \\
Sample collection & Samples in sufficient quantity have already been collected in a previous study. & Collaborations with fishermen and other stakeholders can facilitate the collection of a large number of samples. \\
\rowcolor{Grey}
Sample sites & Present in habitats within and outside of Marine Protected Area boundaries. & Present in habitats within and outside of Marine Protected Area boundaries. \\
Sample sizes & Samples in sufficient quantity have already been collected in a previous study. & Collaborations with fishermen and other stakeholders can facilitate the collection of a large number of samples. \\
\rowcolor{Grey}
DNA extraction & Protocol developed to extract high quality genomic DNA. & Protocol developed to extract high quality genomic DNA. \\
Choice of genetic marker & To explore fine-scale genetic patterns, more powerful markers may need to be developed anew. & Previous molecular work exists but limitations with resolution and sampling suggests markers will need to be developed anew. \\
\rowcolor{Grey}
Availability of genetics markers & Microsatellite markers have been developed and genotyped. & \textquotedbl \space \textquotedbl \\
\hline
\label{selectingtaxa}
\end{longtable}
\end{scriptsize}
\end{landscape}
\clearpage
}
%%% End of table

Benthic marine invertebrates are generally not flagship species (but there are exceptions, e.g. pink sea fans).  However, it is recognised that many benthic invertebrates have a crucial ecological role (e.g. mussel beds as ecosystem engineers / habitat builders) or are commercially exploited (e.g. scallops and lobsters), meaning they are either fundamentally important to the ecosystem or the local/regional economy, or both.  This may encourage relevant management bodies and/or stakeholders to collaborate, to contribute funding and/or to share equipment (depending on the organisation's interests and capacity), all of which can serve to advance a particular project.  For example, lobster fishermen have access to a potential myriad of individuals from which tissue samples can be obtained. Forming these types of collaborations can facilitate access to a virtually unlimited number of samples depending on the fishery status, thereby avoiding the need to arrange dedicated sampling trips, and the associated costs and researcher time typically required for collection. Moreover, maintaining dialogue with such a stakeholder(s) may promote more effective communication of the potential benefits of the research and, ultimately, dissemination of the results \citep{Shafer2015,Britt2018}. 

Other factors to consider include whether the biology and ecology of the candidate species is well known. This process starts, perhaps obviously, by accurate identification of the candidate species and avoiding the erroneous inclusion of closely related or cryptic species, which can drastically influence the results of population genetic structure analyses \citep{Pante2015a}.  The difficulty of accurate taxonomic identification can be further exacerbated when the organism is very small; in some cases, a second opinion from an experienced taxonomist or molecular verification (e.g. DNA barcoding) may be required.  In addition, a thorough understanding of the dispersal, life history and habitat of the candidate species will usually help to explain some of the genetic patterns observed, thereby improving interpretation of the genetic data.

\subsubsection{3.2.2 Methodological factors}
The sampling design of a study should be carefully considered prior to sample collection to ensure that the resulting genetic data are robust and applicable for assessments of genetic connectivity. This typically includes assessing whether the desired sampling strategy is feasible and that sufficient tissue samples from a broad enough range of sites can be taken for meaningful genetic analysis. For example, as suggested previously, if an organism is commercially fished, it may be possible to have tissue samples collected \replaced{\textit{in situ}}{in situ} by fishery personnel. Moreover, ensuring that samples of a species of interest are collected from both within the boundaries of a MPA network and from sites outside ensures that hypotheses about connectivity beyond MPA boundaries can be tested. This approach has provided useful data in several previous studies \citep{Huserbraten2013,Puckett2014,Holland2017}, allowing the performance of a MPA network to be evaluated for the species being studied. 

Other factors to consider include the type(s) of tissue to sample and which genetic markers to use in assessments of population genetic structure. This is of critical importance because the type of tissue can profoundly influence the quantity and quality of DNA obtained post-extraction. For example, crustacean exoskeletal tissues, such as pleopods, are advantageous because they are easily obtained and constitute a non-destructive tissue sample; however, extracting sufficient amounts of pure (contaminant-free) DNA from these tissue types can be extremely difficult using both conventional and kit-based protocols \citep{Li2011}. Moreover, obtaining high molecular weight, non-degraded DNA can be important for methods that utilise next-generation sequencing technology, for example, whole-genome sequencing and SNP discovery from RADseq \citep{Graham2015}. In these cases, optimising the preservation and extraction of DNA will need to be considered prior to sampling and DNA extraction. Choosing appropriate genetic markers and the method of isolation for studies of population genetic structure is also a non-trivial task.  However, a number of comprehensive review papers have been published to address this question \citep{Hellberg2002,Schlotterer2004,Allendorf2010,Cuellar-Pinzon2016,Allendorf2017}.  In addition, tools and papers exist that can help practitioners choose the appropriate number of samples and genetic markers for their particular question \citep[e.g.][]{Hoban2013a}.  Prior to commencing development work, the literature should be screened thoroughly to determine whether genetic markers of a suitable resolution are already available for a candidate species -- this can avoid the costs and time typically required for the development of novel markers.  For example, SNP panels are now available for a wide range of marine species (e.g. salmonids, \citeauthor{Meek2016} \citeyear{Meek2016}; crustaceans, \citeauthor{Jenkins2018} \citeyear{Jenkins2018}; and molluscs, \citeauthor{Jiao2014} \citeyear{Jiao2014}), and are likely to be useful for the analysis of genetic structure, population assignment and connectivity.

\subsection{3.3 Discussion}
\noindent As this thesis focuses on assessing connectivity between British MPAs, species selected for further study had to occupy habitats that are located within the boundaries of MPAs designated in UK waters.  Candidate species whose distribution also spans adjacent seas were desirable because this enabled spatial genetic structure and connectivity to be explored at a much broader scale (e.g. across neighbouring European seas). In effect, this meant that selected species had to have a northeast Atlantic distribution and be relatively abundant across part or most of the British Isles. 

After consideration of the biological and methodological factors outlined previously (Table \ref{tab:factors}), two species were identified as suitable candidates: the pink sea fan (\textit{Eunicella verrucosa}) and the European lobster (\textit{Homarus gammarus}) (Fig. \ref{twospecies}). \added{However, it is important to be aware that the factors outlined in this framework differ in their ease of being assessed. For example, some factors may be prone to subjectivity (e.g. ecological importance) and are potentially more difficult to assess, while other factors are somewhat objective (e.g. the availability of genetic markers) and are thus easier to assess. In addition, some factors in this framework can be more important than other factors for assessing genetic connectivity between MPAs. For instance, as mentioned above, to assess connectivity between a network of MPAs, the distribution of the candidate species must overlap that of MPA boundaries. When considering the relative importance of each factor on a study-by-study basis, a weighting criteria could be applied, such that each factor is ranked according to its importance for the study; this would facilitate an objective process of selecting (or omitting) taxa.}   

%% Edit life history
In this thesis, selecting the pink sea fan and the European lobster meant that population structure and genetic connectivity could be explored in two very different taxa with different life histories. \added{The pink sea fan is a sessile gorgonian soft coral, is relatively long-lived with the potential for sexual and asexual reproduction, and is primarily found in habitats where there are hard substrates (i.e. rocky substrata) to attach to (section 4.1.1). In contrast, the European lobster is a sexually reproducing decapod crustacean with some capacity for movement as adults, and is found in habitats composed of hard and soft substrates where there is adequate shelter (section 6.1.1). Both species are broadcast spawners, but the development of eggs and larvae are markedly different, for example, pelagic larvae of pink sea fan are lecithotrophic whereas pelagic larvae of European lobster are planktotrophic.} Choosing these two species enabled a multi-species assessment of connectivity between British MPAs to be conducted, with the potential to inform both marine conservation and fisheries management. 

\added{Based on the framework from Table \ref{tab:factors},} there were several advantages for selecting the pink sea fan (Table \ref{selectingtaxa}). Firstly, its distribution and abundance are sufficient to explore connectivity patterns within and outside of MPA boundaries.  Secondly, it is generally considered a flagship species for UK conservation and is accordingly protected in England and Wales; it is also listed as `Vulnerable' on the IUCN Red List.  Thirdly, and perhaps most importantly for this candidate species, sampling in sufficient quantities had already taken place prior to this PhD.  This meant there was no need to arrange permits for tissue collection (required in the UK because of the protective status of \textit{E. verrucosa}), or organise sampling trips which would likely involve SCUBA diving and be very costly and time consuming.   

%% Species photos
\begin{figure}[h!]

\begin{minipage}{0.495\textwidth}
  \includegraphics[width=\textwidth, height=6cm]{Chapter_figures/chapter3/PinkSeaFan1.jpg}
\end{minipage}
\begin{minipage}{0.495\textwidth}
  \includegraphics[width=\textwidth, height=6cm]{Chapter_figures/chapter3/lobster1.jpg}
\end{minipage}

\caption[Species selected: the pink sea fan and the European lobster]{Species selected: the pink sea fan (\textit{Eunicella verrucosa}) (left) and the European lobster (\textit{Homarus gammarus}) (right).}
\label{twospecies}
\end{figure}
%% end of figure

\added{Similarly, for the European lobster (\textit{Homarus gammarus}), there were a number of advantages for selecting this species (Table \ref{selectingtaxa}). For example, although samples needed to be collected \textit{de novo}, an active fishery across the British Isles and neighbouring seas meant there was massive potential to collect a vast number of samples from a variety of locations across its range.}  The commercial interest of this species also meant that the biology of \textit{H. gammarus} is well known, which would likely help to explain some of the patterns observed from a novel genomics study.  Moreover, \textit{H. gammarus} is reasonably abundant across the British Isles and most of the northeast Atlantic, meaning connectivity patterns could be explored within and outside of MPA boundaries, but also across the northeast Atlantic and parts of the Mediterranean Sea. 

In conclusion, \added{this study has created a framework which highlights a number of biological and methodological factors to consider before selecting taxa to use in assessments of genetic connectivity between MPAs.} After consideration of these factors, two species with differing life histories were selected to use as surrogates for assessing genetic connectivity between MPAs across the UK.  In addition, because of the ecological and economical importance of these two species, the findings of these studies may also inform conservation or fisheries management bodies relevant to each species. Subsequently, the upcoming chapters explore the spatial genetic structure and connectivity of these two species using microsatellite and/or SNP markers.



%--------------
%
% Chapter 4: Pink sea fan popgen
% 
%--------------
\newpage
\section{Chapter 4: Population genetic structure and connectivity of pink sea fans (\textit{Eunicella verrucosa}) using microsatellite and SNP markers}
\rule{\textwidth}{0.8pt}

\vspace{18pt}
\noindent \textbf{Microsatellite study} \\
The microsatellite-based study of two octocoral species discussed in this chapter is based on a paper published in the journal \textit{Heredity}. Tom L. Jenkins analysed the data, generated the figures and tables, wrote the first draft of the manuscript, and addressed all reviewer comments and edits. This study also formed part of the PhD thesis of Lyndsey P. Holland and, therefore, the results of this study are discussed as part of the Introduction and the Discussion as per university guidelines.  The reference is given below and the full paper is available in the Appendix. \\

\noindent Holland LP*, Jenkins TL*, Stevens JR (2017) Contrasting patterns of population structure and gene flow facilitate exploration of connectivity in two widely distributed temperate octocorals. \textit{Heredity} \textbf{119}, 35-48. *\begin{footnotesize}Joint first authorship.\end{footnotesize} \\

\noindent \textbf{SNP study} \\
The preliminary SNP study was designed and carried out by Tom L. Jenkins and Jamie R. Stevens.  
SNPsaurus (Oregan) prepared the nextRAD libraries and identified SNPs using their custom bioinformatics pipeline.  

\newpage
\subsection{4.0 Abstract}
The pink sea fan (\textit{Eunicella verrucosa}) is a priority species for conservation in English and Welsh waters; yet, until recently very little was known about its genetic diversity, population structure, dispersal and connectivity.  In this chapter, the first population genetics study of \textit{E. verrucosa} using 13 microsatellite markers is briefly discussed; this research spanned two PhD programmes (Lyndsey P. Holland and Tom L. Jenkins).  Following this study, a novel preliminary study was carried out that took advantage of a relatively new reduced-representation sequencing (RRS) method, nextRAD, to isolate genome-wide single nucleotide polymorphisms (SNPs).  The main aims of this novel study were to (i) test whether a RRS approach was feasible using the somewhat degraded DNA obtained from our \textit{E. verrucosa} samples, and (ii) test if patterns of genetic diversity and population structure were comparable (or different) between microsatellite and SNP markers.  The results suggested that isolating SNPs from across the \textit{E. verrucosa} genome was feasible using nextRAD, although a handful of samples and one population had to be discarded due to the influence of missing data affecting downstream analyses.  Spatial genetic patterns using 3,743 SNPs supported the results from the microsatellite study; three main genetic clusters were identified and organised into samples from Britain-France, western Ireland and southern Portugal, with evidence of weak differentiation between samples from Britain and France.  Genetic diversity measures were relatively low and consistent across populations (agreeing with the microsatellite study); moreover, significant heterozygote deficiencies in all sampling sites were observed, possibly caused by inbreeding or a Wahlund effect.  Overall, while it appears that microsatellite and SNP markers show similar patterns of genetic diversity and population structure in \textit{E. verrucosa}, the inclusion of more individuals and intermediate sample sites for the SNP markers is necessary to fully validate these findings and to further explore the drivers of these patterns.


\subsection{4.1 Introduction}

\subsubsection{4.1.1 Pink sea fan biology}
The pink sea fan (\textit{Eunicella verrucosa}) is a colonial gorgonian belonging to the class Anthozoa and the subclass Octocorallia.  They are generally found on rocky substrates at depths of 10-150 m in areas of moderate to high water currents and are native across the northeast Atlantic and parts of the Mediterranean Sea where there is suitable habitat \citep{Hayward1995}.  In particular, colonies have been recorded in western Africa and the western Mediterranean (southern range), around the coasts of Portugal, northern Spain and northwest France, and up to southwest Britain and northwest Ireland (northern range).  Fully-grown colonies usually stand about 30 cm tall but they can reach up to 75 cm in some areas \citep{Wood2013}.  Colonies are commonly seen orientated towards the direction of ocean currents to allow the polyps to filter nutrients or prey using their tentacles.  Two distinct colour morphs (orange-pink and white) of \textit{E. verrucosa} have been found (Fig. \ref{colmorph}), though it is not known whether one phenotype carries any fitness advantage for colonies.

%% Figure: Pink sea fan colour morphs
\begin{figure}[h!]
\centering
\includegraphics[width=0.8\textwidth]{Chapter_figures/chapter4/Colmorph_CWood}
\caption [Pink sea fan colour morphs]
{Pink sea fan (\textit{Eunicella verrucosa}) orange-pink (left) and white (right) colour morphs (image provided by Chris Wood).}
\label{colmorph}
\end{figure}
%% end of figure

\textit{Eunicella verrucosa} is thought to be a gonochoristic (separate sexes) and sexually reproducing species, yet asexual reproduction may be possible by clonal fragmentation \citep{Munro2004}.  Colonies are broadcast spawners, releasing gametes into the water column towards the end of summer (August-September), which are externally fertilised \citep{Munro2004}.  Larvae are thought to be lecithotrophic, meaning they are provided with a yolk sac as a source of nutrition to use during their dispersal via ocean currents.  However, the pelagic larval duration (PLD) of \replaced{\textit{E. verrucosa}}{E. verrucosa} is unknown, which presents uncertainty over the dispersal capability of this species.  

%% Figure: Pink sea fan forests
\begin{figure}[h!]
\centering
\includegraphics[width=0.8\textwidth]{Chapter_figures/chapter4/Forest_CWood.jpg}
\caption [Pink sea fan `forest']
{A pink sea fan (\textit{Eunicella verrucosa}) `forest' (image provided by Chris Wood).}
\label{forests}
\end{figure}
%% end of figure


In many sublittorial ecosystems, \textit{E. verrucosa} colonies can indirectly or directly support other marine organisms, particularly when they are locally abundant and form large `forests' \citep{Wood2013,Pikesley2016}. These forests (Fig. \ref{forests}) provide structural complexity and habitat for a number of epifaunal animals and likely play an important sheltering role for small or juvenile organisms seeking to take refuge from predators.  In addition, some organisms are known to settle on (e.g. barnacles and bryozoans), attach eggs on (e.g. catsharks), or even exclusively live on \textit{E. verrucosa} and other sea fans (e.g. the sea fan anemone, \textit{Amphianthus dohrnii}) \citep{Wood2013}.  This suggests that \textit{E. verrucosa} plays an important role in the functional ecology of the benthic communities it resides in and could be considered an ecosystem engineer \citep{Hall-Spencer2007,Pikesley2016}. 


\subsubsection{4.1.2 Conservation status} 
Pink sea fans are extremely vulnerable to seabed disturbance from trawling and other gears and marine litter (Hinz et al. 2011; Sheehan et al. 2017), primarily because of their delicate structure and overall slow growth rates \citep[$\sim$3.33 cm year\textsuperscript{-1} when colony height $<$15 cm and $\sim$0.62 cm year\textsuperscript{-1} when colony height $>$40 cm,][]{Sartoretto2012}.  Accordingly, the pink sea fan has been classified as `Vulnerable' by the IUCN Red List since 1996, which is defined as a species facing a very high risk of extinction in the wild in the medium-term future.  In addition, \textit{E. verrucosa} is also listed as a priority species under the UK Biodiversity Action Plan and a species of principal importance in England under the NERC Act 2006.  In response, the UK government has established several MCZs around southwest England (e.g. Chesil Beach and Stennis Ledges, Isles of Scilly, Skerries Bank and Surrounds, The Manacles, and Whitsand and Looe Bay) and around Wales (e.g. Skomer Island), that specifically identify \textit{E. verrucosa} as a protected feature in their designation listing.  To my knowledge, pink sea fans are not explicitly protected outside of the UK.  Nevertheless, the Republic of Ireland have established SACs under the EU Habitats Directive (e.g. Kenmare River SAC, Galway Bay Complex SAC and Donegal Bay SAC) and France has designated OSPAR MPAs in northwest Brittany (e.g. the Glenan Islands and the Baie de Morlaix, OSPAR Commission 2016) where \textit{E. verrucosa} colonies are known to occur, which may indirectly help to protect colonies from disturbance in those areas.


\subsubsection{4.1.3 Previous genetic research}
\noindent Genetic research on pink sea fans has been scarce, particularly at the population level. Studies involving \textit{E. verrucosa} have tended to focus on comparative transcriptomics across diverse taxa \replaced{\citep{Romiguier2014}}{\cite{Romiguier2014}} or on phylogenetic relationships within the Anthozoa \citep{Pratlong2016} or within the \textit{Eunicella} genus \citep{Aurelle2017}.  As a result, not much is known about the genetic diversity and population structure of this species, the study of which may shed light on patterns of connectivity among populations and the dispersal capacity of larvae. 

The first population genetic study of \textit{E. verrucosa} was published in early 2017 \citep{Holland2017}.  In this study, the genetic diversity and population structure of \textit{E. verrucosa} and another octocoral, dead man's fingers (\textit{Alcyonium digitatum}), were explored across the northeast Atlantic using 13 (\textit{E. verrucosa}) and eight (\textit{A. digitatum}) microsatellite loci developed by \citeauthor{Holland2013} (\citeyear{Holland2013}, \citeyear{Holland2013a}).  Details of DNA extraction, microsatellite development and scoring, and analytical methods can be found in the original papers (\citeauthor{Holland2013} \citeyear{Holland2013}, \citeyear{Holland2013a}, \citeyear{Holland2017}).  Due to the focus of this chapter on \textit{E. verrucosa}, hereafter only the results for \textit{E. verrucosa} are discussed. 

%% Figure: Pink sea fan msat sampling map
\begin{figure}[h!]
\centering
\includegraphics[width=0.95\textwidth]{Chapter_figures/chapter4/Holland2017_map.png}
\caption [Pink sea fan sampling sites: microsatellite study]
{Map of the sampling sites used in the microsatellite study from Holland et al. (2017).}
\label{holland2017map}
\end{figure}
%% end of figure

Tissue samples of \textit{E. verrucosa} were collected in 2007-2012 from sites across its middle (southern Portugal) and northern range (southwest Britain/northwest Ireland) (Fig. \ref{holland2017map}). This study revealed several important findings about the population genetics of \textit{E. verrucosa}. Firstly, genetic diversity (expected heterozygosity and allelic richness) was generally uniform across the sampling range, but was lower compared to that reported in other temperate corals (Table 3 in Holland et al. 2017), which included the closely related species \textit{E. cavolini} \citep{Masmoudi2016} and \textit{E. singularis} \citep{Costantini2016}, \textit{A. digitatum} \citep{Holland2017}, and two Mediterranean octocorals, \textit{Corallium rubrum} \citep{Ledoux2010a} and \textit{Paramuricea clavata} \citep{Mokhtar-Jamai2011}. However, in comparison to these temperate octocorals, the number of sites with significant heterozygotes deficiencies was lower in both \textit{E. verrucosa} and \textit{A. digitatum} \citep{Holland2017}.  The authors concluded that the most likely explanation for this pattern was that, overall, the frequency of inbreeding is generally low in \textit{E. verrucosa} and \textit{A. digitatum}, though inbreeding may be apparent at some sites which is likely due to site-specific factors \citep{Holland2017}.

Secondly, across the sites sampled, \textit{E. verrucosa} populations were not panmictic, instead revealing three distinct genetic clusters (Britain-France, southern Portugal and northwest Ireland) (Fig. \ref{pdfmsats}), with evidence of weak genetic differentiation between sites from southwest Britain and northwest France (Fig. \ref{pdfmsats}). Further analysis suggested that isolation-by-distance (IBD) was a likely explanation for the differentiation observed between sites from Britain, France and Portugal.  Interestingly, this pattern of IBD appears to be common in temperate octocorals (Table 3 in Holland et al. 2017), which is possibly due to their sedentary life history and their lack of, or shorter, PLD compared to other benthic marine species.  However, IBD did not explain the genetically distinct profiles observed in colonies from northwest Ireland.  This suggested that the differentiation observed in northwest Ireland colonies, inhabiting the northern peripheral range of the species, may be driven by other factors, such as barriers to gene flow and/or selection \citep{Holland2017}. Two previous studies of marine invertebrates studied across this region have also reported genetic differentiation in western Ireland compared to other locations in the northeast Atlantic \citep{Remerie2009,Casu2011}.  These studies attributed this differentiation to recolonisation from glacial refugia or from persistance in ice-free coastal areas during the LGM. \added{Lower genetic diversity at range margins can be explained by founder effects and genetic drift (due to a low number of post-glacial recolonisers); \citet{Casu2011} concluded this was the likely explanation in their study of microturbellarians from southwest Ireland. In contrast, \citet{Remerie2009} found higher genetic diversity and heterogeneity for \textit{Neomysis integer} (an estuarine shrimp) in glaciated areas, suggestive of range persistence during the LGM.} The findings for \textit{E. verrucosa} colonies from northwest Ireland, which exhibited the lowest genetic diversity detected in the entire study \textit{E. verrucosa}, are in line with those of \citet{Casu2011}; this suggests that founder effects following post-glacial recolonisations are a potential explanation for the genetic distinctiveness of northeast Ireland \textit{E. verrucosa} colonies. Yet, insufficient sampling across the rest of Ireland and at the southern-most limits of the range of \textit{E. verrucosa} makes inferences about the origin of populations in northwest Ireland difficult. In addition, lower diversity at the range margins can be characteristic of populations under intense selection pressures \citep{Johannesson2006}, which \added{may} suggest that natural selection may be driving this genetic divergence; however, it is not known which selection pressures, if any, may be acting on these most northerly populations of pink sea fan.


%% Figure: Pink sea fan PCoA and STRUCTURE plot
\begin{figure}[h!]
\begin{minipage}{\textwidth}
  \centering
  \includegraphics[width=0.8\textwidth]{Chapter_figures/chapter4/Seafan_13msats_pcoa.png}
\end{minipage}
\begin{minipage}{\textwidth}
  \includegraphics[width=\textwidth]{Chapter_figures/chapter4/K3_13_microsats}
\end{minipage}
\caption[Pink sea fan population structure: microsatellite study]
{Population structure of pink sea fans using 13 microsatellite markers (Holland et al. 2017).  A principle coordinates analysis (top) and results from STRUCTURE analysis (bottom) are presented. For the principle coordinates analysis, each point represents a sampling site and the colours correspond to the country of origin.  For the STRUCTURE plot, each bar represents an individual and the colours represent the membership proportion to each of the three genetic clusters inferred.}
\label{pdfmsats}
\end{figure}
%% end of figure

Thirdly, there was strong genetic similarity within regions (i.e. within southwest Britain), suggestive of high genetic connectivity at these spatial scales (Fig. \ref{pdfmsats}). This was further supported by analyses of contemporary gene flow, which indicated that the majority of gene flow was exchanged between sites from the same region (Fig. \ref{fig:pdfgeneflow}). In addition, this analysis also provided evidence that colonies from southwest Britain have potentially been a source of genetic variants for French colonies over the last few generations. Together, these results suggest that \textit{E. verrucosa} larvae are able to disperse and exchange genetic material at distances of up to 500 km; however, whether gene flow at these scales are be achieved by a single migration event or by a stepping-stone model of connectivity is not yet clear.  Of course, this genetic similarity may also suggest that at a regional scale \textit{E. verrucosa} have high effective population sizes, which can hinder the formation of population structure by mitigating the influence of drift. 

%% Figure: Gene flow
\begin{figure}[h!]
\centering
\includegraphics[width=0.6\textwidth]{Chapter_figures/chapter4/Eunicella_BA.pdf}
\caption [Pink sea fan gene flow: microsatellite study]
{\textit{Eunicella verrucosa} gene flow using 13 microsatellite markers \citep{Holland2017}. Gene flow was calculated using BayesAss v3.0.4 \citep{Wilson2003}, which provides an estimate of gene flow over the last few generations. For this analysis, sampling sites in each country were combined (denoted by colours).  The direction of an arrow represents the direction of gene flow from one country to another; the width of the arrows denotes the relative amount of gene flow (i.e. the wider the arrow, the more gene flow). The `humps' represent gene flow originating from sample sites within countries.}
\label{fig:pdfgeneflow}
\end{figure}
%% end of figure

\subsubsection{4.1.4 Study aims}  
The microsatellite study summarised previously provided a much-needed investigation into the population genetics and connectivity of pink sea fans across their middle and northern range.  However, as a Red Listed and priority species for conservation, it is critical that genetic data presented to MPA managers are as robust and reliable as possible before decisions are made and enacted.  Therefore, a study that explores the genetic patterns of \textit{E. verrucosa} using an alternative marker system is needed, which would facilitate a direct comparison with the 13 microsatellite markers.

In light of this, the first aim was to conduct a preliminary study to investigate whether isolating genome-wide SNPs using a reduced representation sequencing (RRS) approach was feasible.  As the quality of DNA required for microsatellite genotyping does not necessarily need to be of high molecular weight and quality, it was essential to re-extract and optimise a DNA extraction protocol for \textit{E. verrucosa} samples.  This was because the DNA samples used in \citet{Holland2017} were relatively degraded, proving inadequate for high-throughput sequencing (K. Moore, Exeter Sequencing Service, \textit{pers. comm.}).  However, even after trying many different extraction methods, only suboptimal degraded DNA could be extracted for some \textit{E. verrucosa} samples.  The second aim of this study was to use these preliminary data to test whether thousands of SNP markers showed similar patterns of spatial genetic structure to the 13 microsatellite markers analysed in \citet{Holland2017}.  Finally, to conclude, future research objectives and directions for the population genomics study of \textit{E. verrucosa} are discussed.  


\subsection{4.2 Materials and methods}
\subsubsection{4.2.1 Sample collection and DNA extraction}
\noindent \textit{Eunicella verrucosa} tissue samples were collected and preserved in 95-100 \% ethanol as described in \citet{Holland2017}.  Eight sampling sites from \citet{Holland2017} were chosen to include in this study and an additional novel site from the Isles of Scilly (Hathor Wreck) was also included (Table \ref{tab:snpsampling}; Fig. \ref{fig:snpsamplesites}).  All samples used for this study were collected between 2009 and 2012.  As one of the aims of this study was to compare the resolution obtainable with SNPs to microsatellite markers, the sampling sites and number of individuals were chosen such that roughly equal representation from the three genetic groups found in \citet{Holland2017} were included.

Genomic DNA was extracted from 15-20 polyps using a modified salting-out protocol (Appendix A4), originally designed to extract DNA from crustacean exoskeletal tissue \citep{Li2011}.  Obtaining high molecular weight and pure DNA from many of the pink sea fan tissue samples was difficult, with evidence of degradation present in nearly every sample (Appendix A5).  Initially, the Blood and Tissue Kit (Qiagen) was used for extractions; however, these DNA samples failed to generate adequate RAD libraries during a trial run and comparisons suggested this method was inferior to the salting-out protocol (Appendix A5).  The concentration and purity of all extractions were quantified by spectrophotometry using a NanoDrop 1000.  In addition, all DNA samples were further evaluated by running the DNA on a 1 \% agarose gel and by quantifying their concentration with fluorometry using the Invitrogen Qubit Assay kit, which only measures the amount of double-stranded DNA in the sample.  These quality assessments were vital to allow the best quality DNA samples from each sampling site to be submitted for high-throughput sequencing. 

%% Table: pink sea fan sampling table
\begin{table}[h!]
\begin{small}
\caption[Pink sea fan sampling information: SNP study]{Pink sea fan (\textit{Eunicella verrucosa}) sampling information for the SNP study.}
\label{tab:snpsampling}
\begin{threeparttable}
\begin{tabular}{llccccxc}
\hline \\[-1.0em]
\textbf{Country} & \textbf{Site} & \textbf{Code} & \textbf{\textit{N}} & \textbf{Lat} & \textbf{Lon} & \textbf{Depth (m)} & \textbf{Year} \\
\hline \\[-1.0em]
Britain	& \textsuperscript{a}Isles of Scilly, Hathor Wreck &	Hat	& 9 &	49.88 &	-6.35 &	28 & 2010 \\
 & \textsuperscript{a}Isles of Scilly, Lion Rock &	Lio &	9 &	49.98 &	-6.31 &	24 & 2009 \\
 & \textsuperscript{a}Lundy Island &	Lun &	18 & 51.17 & -4.69 & 23 &	2009 \\
France & \textsuperscript{b}Glenan Islands, Laonegued &	Lao &	10 & 47.73 & -4.06 & 30 &	2011 \\
 & \textsuperscript{b}Glenan Islands, Men Goe & Men & 9 & 47.69 & -3.99 & 30 &	2011 \\
Ireland &	\textsuperscript{b}Donegal Bay, Black Rock &	Bla	& 10	& 54.58 &	-8.43 &	25 &	2012 \\
 &	\textsuperscript{b}Donegal Bay, Thumb Rock &	Thu &	10 &	54.47 &	-8.44 &	20 &	2012 \\
Portugal	& Faro &	Far &	10	& 36.98 &	-7.99 &	17 &	2010 \\
 &	Portimao2	& Por &	10 & 37.10 &	-8.56 &	18 &	2010 \\
\hline
\end{tabular}
  \begin{tablenotes}
    \scriptsize
    \item \textit{N}, number of individuals submitted for sequencing.
    \item \textsuperscript{a}Marine Conservation Zone.
    \item \textsuperscript{b}Special Area of Conservation.
  \end{tablenotes}
\end{threeparttable}
\end{small}
\end{table}
%% end of table

%% Figure: pink sea fan SNP map
\begin{figure}[h!]
\centering
\includegraphics[width=0.7\textwidth]{Chapter_figures/chapter4/pink_sea_fan_snp_map_crop.pdf}
\caption [Pink sea fan sampling sites: SNP study]
{\textit{Eunicella verrucosa} sampling sites for the SNP study. See Table \ref{tab:snpsampling} for sampling site code information.}
\label{fig:snpsamplesites}
\end{figure}
%% end of figure

\subsubsection{4.2.2 nextRAD sequencing}
A relatively new RRS method, nextRAD \citep{Russello2015}, was chosen for isolating SNPs from across the \textit{E. verrucosa} genome. The rationale behind this choice was that only 10 ng or less is required as input DNA and the method was suggested to perform better than traditional RADseq with degraded DNA (SNPsaurus, \textit{pers. comm.}).  Genomic DNA was converted into nextRAD genotyping-by-sequencing libraries (SNPsaurus, LLC) which uses a selective primer sequence (rather than restriction enzymes) to genotype loci consistently across samples \citep{Russello2015}.  Genomic DNA was first fragmented with Nextera reagent (Illumina, Inc.), which also ligates short adapter sequences to the ends of the fragments. The Nextera reaction was scaled for fragmenting 5 ng of genomic DNA, although 7.5 ng of genomic DNA was used for input to compensate for the amount of degraded DNA in the samples and to increase fragment sizes. Fragmented DNA was then amplified for 26 cycles at 73\textsuperscript{o}C, with one of the primers matching the adapter and extending nine nucleotides into the genomic DNA with the selective sequence GTGTAGAGG.  Thus, only fragments starting with a sequence that can be hybridized by the selective sequence of the primer will be efficiently amplified. The nextRAD libraries were sequenced on one Illumina HiSeq 4000 lane with single-end 150 bp reads (University of Oregon).  The quality of reads for each sample was assessed using FastQC software (Babraham Bioinformatics). 

\subsubsection{4.2.3 Bioinformatics}
The genotyping analysis used custom scripts that trimmed the reads using bbduk (BBMap tools). A \textit{de novo} reference was created by collecting 10 million reads, evenly from the samples, and excluding reads that had counts fewer than seven or more than 700.  The remaining loci were then aligned to each other to identify allelic loci and collapse allelic haplotypes to a single representative.  All reads were mapped to the reference with an alignment identity threshold of 93 \% using bbmap (BBMap tools).  Genotype calling was done using Samtools and bcftools.  The output vcf file was filtered to remove alleles with a population frequency of less than 3 \%.  Loci were removed that were heterozygous in all samples or had more than two alleles in a sample (suggesting collapsed paralogs).  The absence of artefacts was checked by counting SNPs at each nucleotide position and ensuring that the number of SNPs did not increase with reduced base quality at the end of the read.

\subsubsection{4.2.4 Quality control and filtering}
Filtering of the vcf file provided by SNPsaurus was implemented in Stacks v1.48 \citep{Catchen2013a} and radiator v0.0.5 \citep{Gosselin2017}.  The \texttt{populations} program in Stacks was run with the following parameters: a locus had to be present in at least 80 \% of individuals in a population (-\texttt{r} 0.8) and in at least seven (out of nine) populations (-\texttt{M} 7), the minor allele frequency threshold was set to 5 \% (\texttt{--min\char`_maf 0.05}), loci had to have a maximum observed heterozygosity of less than 0.5 (\texttt{--max\char`_obs\char`_het 0.5}), and only the first SNP in each locus was processed (\texttt{--write\char`_single\char`_snp}).  A new vcf file was created and then imported into R for further filtering using radiator; several functions were run to manipulate and interrogate the data.  Firstly, the \texttt{missing\char`_visualization} function was executed using default parameters to visualise and assess missing data.  Secondly, the \texttt{detect\char`_mixed\char`_genomes} function was run using default parameters which assesses observed heterozygosity in each individual for a diagnostic test of mixed samples or poor polymorphism discovery.  Thirdly, the \texttt{detect\char`_duplicate\char`_genomes} function was run with the genome argument set to true (\texttt{genome=TRUE}) to highlight potential duplicate individuals.  After data exploration, only loci that were genotyped in all populations (\texttt{common.markers=TRUE}) were retained.  The final filtered dataset was exported in multiple formats using the \texttt{genomic\char`_converter} function.    

\subsubsection{4.2.5 Genetic diversity}
Observed heterozygosity (\textit{H}\textsubscript{o}), expected heterozygosity (\textit{H}\textsubscript{e}) and the inbreeding coefficient (\textit{F}\textsubscript{is}) were calculated using the \texttt{divBasic} function from the R package diveRsity v1.9.90 \citep{Keenan2013}.  For each population, the mean value across all loci was computed and significance of \textit{F}\textsubscript{is} was assessed by calculating bias corrected 95 \% confidence intervals (1,000 bootstrap replicates) and testing whether values were significantly different from zero.  

\subsubsection{4.2.6 Population structure}
Genetic differentiation between sampling sites was analysed by calculating pairwise values of \textit{F}\textsubscript{st} \citep{Weir1984} and \textit{D} \citep{Jost2008} using the \texttt{diffCalc} function from diveRsity.  Heatmaps of each statistic were visualised in R and significance was assessed using the same approach previously described for \textit{F}\textsubscript{is}.  

Population structure among sampling sites was explored using two different approaches.  Firstly, a discriminant analysis of principal components (DAPC) was run using the \texttt{dapc} function from the R package adegenet v2.1.0 \citep{Jombart2011}.  DAPC attempts to summarise genetic differentiation between groups, sampling sites in this context, while overlooking variation within groups \citep{Jombart2010}.  It first transforms the data using principal components analysis (PCA) to ensure variables are uncorrelated and that the number of variables (alleles) is less than the number of observations (individuals) in the dataset.  These are necessary prerequisites for discriminant analysis (DA).  Then a DA is performed on the number of principal components (PCs) retained, which is selected by the user. Cross validation using the \texttt{xvalDapc} function from adegenet was used to choose the optimal number of PCs to retain.  

Secondly, STRUCTURE v2.3.4 \citep{Pritchard2000}, a Bayesian clustering algorithm, was run in parallel using the program StrAuto v1.0 \citep{Chhatre2017}.  STRUCTURE attempts to estimate the number of ancestral populations (\textit{K}) from multi-locus allele frequencies.  Given a value of \textit{K}, the program assigns individuals probabilistically to these \textit{K} clusters, with the assumption that loci are under Hardy-Weinberg equilibrium (HWE) and linkage equilibrium \citep{Gilbert2016}.  Post-hoc tests have been developed to help assess the \textit{K} that best fits the data (e.g. mean L(\textit{K}), Pritchard et al. 2000; and delta \textit{K}, Evanno et al. 2005); however, several recent papers have suggested that detecting a true \textit{K} statistically is extremely difficult and that in some cases no true \textit{K} may exist \citep{Meirmans2015,Gilbert2016,Janes2017}.  The authors encourage visualising each \textit{K} to explore population structure at each level to make informed choices.  STRUCTURE was executed using the admixture model, with 10\textsuperscript{5} MCMC repetitions and a burn-in of 10\textsuperscript{5}.  The locprior option was selected, which meant that sampling site origins were used as priors; all other parameters were set to default values. The maximum assumed (\textit{K}) was nine and ten independent replicates per \textit{K} (1-9) were computed.  The mean L(\textit{K}) and delta \textit{K} statistics were examined using the R package pophelper v2.2.5.1 \citep{Francis2017}.  Replicates runs were aligned and merged with CLUMPP \citep{Jakobsson2007} using a wrapper script in pophelper and R was used to visualise the results.
% 
% Isolation-by-distance (IBD) was tested by preparing geographic and genetic distance matrices and performing Mantel tests on the matrices in R. Geographic distances were calculated using the R package marmap v1.0 (Pante and Simon-Bouhet 2013) using the \texttt{getNOAA.bathy}, \texttt{trans.mat} and \texttt{lc.dist} functions.  Least cost paths (km) between all sampling sites were calculated using a minimum depth of 10 m (\texttt{min.depth=-10}), as recommended by the authors to limit any paths crossing land masses that may arise due to inaccurate bathymetric data.  These geographic distances were standardised by log transformation.  Matrices of genetic distances were prepared using linearised pairwise \textit{F}\textsubscript{st} and \textit{D}.  A Mantel test was performed on the matrices using the \texttt{mantel.rtest} function from adegenet (10,000 permutations).  

\subsubsection{4.2.7 Detecting outlier SNPs}
To detect outlier SNPs potentially under selection, four differentiation-based methods were implemented: Arlequin v3.5.2.2 \citep{Excoffier2010}, Bayescan v2.1 \citep{Foll2008}, OutFLANK v0.2 \citep{Whitlock2015} and PCadapt v4.0.3 \citep{Luu2017}. Arlequin integrates heterozygosity and simulates a distribution of \textit{F}\textsubscript{st} for neutrally distributed markers to detect outliers. The infinite island model was run using 100,000 simulations and 1,000 demes. Bayescan is a Bayesian method based on a logistic regression model that attempts to distinguish locus-specific (alpha) effects of selection from population-specific (beta) effects of demography; departure from neutrality at a given locus is assumed when the locus-specific component is required to explain the observed pattern of diversity \citep{Foll2008}. Bayescan was run using default parameters and a prior odds of 10,000, which sets the neutral model as being 10,000 times more likely than the model of selection to try and minimise the risk of false positives \citep{Whitlock2015}. OutFLANK calculates a likelihood on a trimmed distribution of \textit{F}\textsubscript{st} values to infer the distribution of \textit{F}\textsubscript{st} for neutral markers and was executed using default parameters. PCAdapt uses principle components analysis to detect loci under selection and assumes that markers excessively related to population structure are candidates for local adaptation; default parameters were selected and three principle components were retained based on the number of clusters inferred from the DAPC (see Results).  For all methods, a false discovery rate of 0.05 was used to identify outliers and only SNPs that were identified as outliers in two or more methods were classed as outlier SNPs.  The DAPC was re-run with putatively neutral SNPs and SNPs putatively under divergent selection to explore the contribution of neutral versus potential adaptive processes in driving the genetic patterns observed.  

\subsection{4.3 Results}
\subsubsection{4.3.1 Sequencing}
In total, 362 million reads were generated by the Illumina HiSeq 4000.  The Phred score for most base calls exceeded 32, indicating that the sequence quality for reads across all samples was very good ($>$ 99.9 \% accuracy).  The number of reads was vastly unequally distributed among individual samples but there was no evidence to suggest that this pattern was population-specific (Fig. \ref{fig:no_reads}).

%% Figure: Number of reads per individual
\begin{figure}[h!]
\centering
\includegraphics[width=0.8\textwidth]{Chapter_figures/chapter4/Total_reads_seafan_nextRADv2.png}
\caption [nextRAD: number of reads per individual]
{The number of reads per individual of \textit{Eunicella verrucosa}. Colours denote the sampling site of origin for each individual.}
\label{fig:no_reads}
\end{figure}
%% end of figure

\subsubsection{4.3.2 Quality control and filtering}
The vcf file provided by SNPsaurus contained 18,459 loci which was reduced to 6,386 loci after filtering in Stacks.  Analysis in radiator highlighted some potential problems with missing data and heterozygosity of loci.  Firstly, there was greater than 30 \% missing data in 11 samples, including Bla09, Lun11 and all samples from Men Goe (Fig. \ref{nextradmissingdata}). Initial genetic analysis using STRUCTURE distinguished only Men Goe from all other samples; this finding does not accord with any obvious underlying biological, geographical or physical patterns or parameters, so this population was immediately removed from the analysis due to the potential bias of missing data.  The remaining individuals generally showed less than 20 \% missing data; therefore, individuals that met this threshold were retained to ensure downstream analyses would not be biased by missing genotypes. Similarly, enforcing a 20 \% threshold for missing data for each locus ensured that filtering of loci was also stringent and that only the highest quality SNPs were retained.

%% Figure: nextRAD: missing data
\begin{figure}[h!]
\centering
\includegraphics[width=0.8\textwidth]{Chapter_figures/chapter4/Missing_data_seafan_nextRADv2.png}
\caption [nextRAD: missing data per individual]
{Proportion of missing data per individual of \textit{Eunicella verrucosa}. Colours denote the sampling site of origin for each individual.}
\label{nextradmissingdata}
\end{figure}
%% end of figure

Secondly, observed heterozygosity averaged across all loci for each individual showed pronounced differences relative to the mean of the population (Fig. \ref{nextradhet}).  This pattern can suggest that samples have potentially been mixed or that polymorphism discovery has been biased based on DNA quality or other artefacts related to library preparation and sequencing \citep{Gosselin2017}.  Although it may be possible that bias could have been introduced into the dataset because of the degraded DNA of some of these samples, downstream genetic analyses made biological sense when using those loci retained post-filtering; therefore, no individuals or loci were omitted based on heterozygosity. Finally, comparison of two specimens, Lun03 and Lun10, revealed that they have more than 83 \% of their genotypes in common (Fig. \ref{nextradman}); therefore, only one sample was kept for further analyses because they could be duplicates.  The final \textit{E. verrucosa} dataset comprised 77 individuals, 8 sites and 3,743 SNPs. 

%% Figure: nextRAD: heterozygosity
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter4/individual_heterozygosity_manhattan_plot.pdf}
\caption [nextRAD: observed heterozygosity per individual]
{Observed heterozygosity per individual averaged across all loci. Each point represents an individual and the size of points is proportional to the amount of missing data for that individual. Colours denote sampling sites and the black dotted line represents the average heterozygosity for each sampling site.}
\label{nextradhet}
\end{figure}
%% end of figure

%% Figure: nextRAD: manhatten
\begin{figure}[h!]
\centering
\includegraphics[width=0.95\textwidth]{Chapter_figures/chapter4/manhattan_plot_genome.pdf}
\caption [nextRAD: individual pairwise genome similarity]
{Individual pairwise genome similarity.  The proportion of shared genotypes is averaged across shared markers between each pairwise comparison. Each point represents an individual and the size of points is proportional to the number of shared markers.  Black points denote pairwise comparisons between different sampling sites and blue points denotes pairwise comparisons between individuals from the same sampling site.}
\label{nextradman}
\end{figure}
%% end of figure

\subsubsection{4.3.3 Genetic diversity}
Genetic diversity was relatively consistent across all populations.  \textit{H}\textsubscript{o} and \textit{H}\textsubscript{e} ranged from 0.17-0.20 and 0.24-0.26, respectively (Table \ref{tab:snpdiversity}).  Values of \textit{F}\textsubscript{is} for all populations were significantly positive (confidence intervals did not span zero), suggesting a profound deficiency in heterozygotes. Results were also comparable when only putatively neutral SNPs were used to calculate genetic diversity statistics (Table \ref{tab:snpdiversity}).

%% Table: pink sea fan SNP genetic diversity
\begin{table}[h!]
\begin{small}
\caption[Pink sea fan genetic diversity: SNP study]
{\textit{Eunicella verrucosa} summary information and genetic diversity statistics.}
\label{tab:snpdiversity}
\begin{threeparttable}
\begin{tabular}{llllllllll}
\hline \\[-1.0em]
\textbf{Country} & \textbf{Code} & \textbf{\textit{N}} & \textbf{\textit{N}}\textsubscript{g} & \textbf{\textit{H}\textsubscript{o}} & \textbf{\textit{H}\textsubscript{e}} & \textbf{\textit{F}\textsubscript{is}} & \textbf{\textit{H}\textsubscript{o}\textsuperscript{neutral}} & \textbf{\textit{H}\textsubscript{e}\textsuperscript{neutral}} & \textbf{\textit{F}\textsubscript{is}\textsuperscript{neutral}}  \\
\hline \\[-1.0em]
Britain & Hat	& 9 &	9 &	0.17 &	0.25	& \textbf{0.321} & 0.17 &	0.28 &	\textbf{0.303} \\
 &  Lio	& 9 &	7 &	0.18 &	0.24 &	\textbf{0.271}  &	0.18 &	0.27 &	\textbf{0.267} \\
 &	Lun	& 18 &	15 &	0.18 &	0.26 &	\textbf{0.310}  &	0.18 &	0.27 &	\textbf{0.280} \\
France &	Lao &	10 &	9 &	0.19 &	0.25 &	\textbf{0.322}  & 0.19 &	0.27 &	\textbf{0.228} \\
Ireland	& Bla &	10 &	9	& 0.18 &	0.25 &	\textbf{0.280} & 0.18 &	0.27 &	\textbf{0.264} \\
 &	Thu	& 10 &	8	& 0.20 &	0.25 &	\textbf{0.211} &	 0.20 &	0.27 &	\textbf{0.214} \\
Portugal &	Far &	10 &	10 &	0.17 &	0.25 &	\textbf{0.294} &	 0.17 &	0.27 &	\textbf{0.269} \\
 &	Por	& 10 &	10 &	0.17 &	0.25 &	\textbf{0.324} &	 0.17 &	0.27 &	\textbf{0.303} \\
\hline
\end{tabular}
  \begin{tablenotes}
  \setlength\labelsep{0pt} % align all notes with table
    \scriptsize
    \item \textit{N} number of individuals submitted for sequencing, \textit{N}\textsubscript{g} number of individuals successfully genotyped, \textit{H}\textsubscript{o} observed heterozygosity, \textit{H}\textsubscript{e} expected heterozygosity, \textit{F}\textsubscript{is} inbreeding coefficient, \textsuperscript{neutral} only putatively neutral SNPs were used for this calculation.
    \item Bold font represents values that are significantly different from zero. 
  \end{tablenotes}
\end{threeparttable}
\end{small}
\end{table}
%% end of table

\clearpage
\subsubsection{4.3.4 Population structure}
Global \textit{F}\textsubscript{st} and \textit{D} were 0.016 and 0.003, respectively, and both pairwise differentiation statistics showed comparable pairwise patterns between sampling sites (Fig. \ref{nextradheatmaps}).  Values of \textit{F}\textsubscript{st} ranged from zero (Bla-Thu) to 0.030 (Far-Bla) and from 0.006 (Bla-Thu) to 0.019 (Far-Bla) for \textit{D}.  The highest values for both statistics were between Portugal (Far and Por) and Ireland (Bla and Thu), whereas the lowest values tended to be between sites originating from the same country which are spatially situated closer together. No values of \textit{F}\textsubscript{st} or \textit{D} were significantly different from zero. 
\vspace{12pt}

%% Figure: nextRAD: genetic differentiation heatmaps
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter4/Fst_d_heatmaps.png}
\caption [nextRAD: pairwise genetic differentiation heatmap]
{Pairwise genetic differentiation of \textit{F}\textsubscript{st} (left) and Jost's \textit{D} (right) between sampling sites.}
\label{nextradheatmaps}
\end{figure}
%% end of figure

\clearpage
The DAPC using all SNPs provided strong support for three genetic groups: western Ireland (Bla and Thu), Britain-France (Hat, Lio, Lun and Lao) and southern Portugal (Far and Por) (Fig. \ref{nextrad_dapc_allsnps}\added{)}. In total, axis 1 and axis 2 explained 88.7 \% of the genetic variation. Laonegued appeared to marginally differentiate itself from the main Britain-France cluster, which suggests there is weak differentiation between the samples from Britain and this sample from France. 
\vspace{12pt}

%% Figure: nextRAD: dapc
\begin{figure}[h!]
\begin{minipage}{\textwidth}
\centering
  \includegraphics[width=0.7\textwidth]{Chapter_figures/chapter4/Seafan_dapc_allsnps.png}
\end{minipage}

\begin{minipage}{0.50\textwidth}
  \includegraphics[width=\textwidth]{Chapter_figures/chapter4/Seafan_dapc_neutralsnps.png}
\end{minipage}
\begin{minipage}{0.50\textwidth}
  \includegraphics[width=\textwidth]{Chapter_figures/chapter4/Seafan_dapc_outliersnps.png}
\end{minipage}
\caption [nextRAD: discriminant analysis of principle components]
{Discriminant analysis of principle components (DAPC) using all SNPs (top), putatively neutral SNPs (left) and outlier SNPs (right). For each DAPC, points represent individuals and colours denote the sampling region of origin (Britain, shades of red-pink; France, blue; Ireland, shades of green; Portugal, shades of orange-brown).}
\label{nextrad_dapc_allsnps}
\end{figure}
%% end of figure

\clearpage
STRUCTURE results were not as easy to interpret as the DAPC results. The mean L(\textit{K}) statistic seemed to suggest that \textit{K} becomes more informative up to the maximum number of populations assumed; in comparison, the delta \textit{K} statistic indicated that the optimum number of ancestral populations was \textit{K}=3, with some support for \textit{K}4 and \textit{K}5 (Fig. \ref{nextrad_evanno}). Therefore, to explore structure at different levels of \textit{K}, data for \textit{K}3-\textit{K}5 were plotted (Fig. \ref{nextrad_structure}). The results for \textit{K}3 showed that both populations from southern Portugal were dominated by one genetic cluster, infrequently found in other populations. This was also supported by \textit{K}4 and \textit{K}5.  At \textit{K}5 the STRUCTURE results supported the DAPC, in which one genetic cluster is predominantly only found in the populations from western Ireland.    \vspace{12pt}

%% Figure: nextRAD: Evanno
\begin{figure}[h!]
\centering
\includegraphics[width=0.90\textwidth]{Chapter_figures/chapter4/Seafan_Evanno_3473snps.png}
\caption [nextRAD: interpreting \textit{K}]
{Interpreting \textit{K} using the L(\textit{K}) and the delta \textit{K} methods. (A) mean L(\textit{K}), (B) L$'$(\textit{K}), (C) L$''$(\textit{K}), (D) delta \textit{K}.}
\label{nextrad_evanno}
\end{figure}
%% end of figure

%% Figure: nextRAD: Structure
\begin{figure}[h!]
\centering
\subfigure{\includegraphics[width=\textwidth]{Chapter_figures/chapter4/K3_3473_SNPs.png}} \\[-1.5ex]
\subfigure{\includegraphics[width=\textwidth]{Chapter_figures/chapter4/K4_3473_SNPs.png}} \\[-1.5ex]
\subfigure{\includegraphics[width=\textwidth]{Chapter_figures/chapter4/K5_3473_SNPs.png}} 
\caption [nextRAD: STRUCTURE plots]
{STRUCTURE barplots showing individual membership proportions to \textit{K} ancestral populations (clusters). Each bar represents an individual and colours denote membership proportions to each cluster. Individuals are organised by sampling site and the country of origin.}
\label{nextrad_structure}
\end{figure}
%% end of figure

\clearpage
\subsubsection{4.3.5 Outlier SNPs}
Across all four outlier tests, 131 loci were identified as outliers but only eight of these loci were identified in two or more methods (Fig. \ref{nextrad_ven}).  The consensus sequences (150 bp) containing these eight loci were submitted to blastx (NCBI) to check whether they matched any translated proteins.  One of these eight consensus sequences (locus 1813) had several hits to uncharacterised or hypothetical proteins from scleractinian corals and a sea cucumber (\textit{Apostichopus japonicus}).  
\vspace{12pt}

%% Figure: nextRAD: venn diagram
\begin{figure}[h!]
\centering
\includegraphics[width=0.7\textwidth]{Chapter_figures/chapter4/Seafan_venn.pdf}
\caption [nextRAD: outlier SNPs venn diagram]
{Venn diagram of the outlier SNPs detected by four differentiation-based methods: Arlequin, Bayescan, OutFLANK and PCAdapt.}
\label{nextrad_ven}
\end{figure}
%% end of figure


\subsection{4.4 Discussion}
The results of this study demonstrate that isolating and genotyping genome-wide SNPs in \textit{E. verrucosa} using a RRS approach was successful, despite the constraints in obtaining clean, high molecular weight DNA from many samples.  Quality control steps showed that some individuals had a lot of missing data, which were subsequently removed as they could have biased downstream analyses.  Population structure analyses indicated that SNP markers showed very similar broad patterns of spatial genetic structure to microsatellite markers; however, slight differences were observed in the inbreeding coefficient calculated between the two types of markers.  This SNP study provides the first insight into the population genomics of \textit{E. verrucosa}, a species of major conservation importance in the UK and in the wider northeast Atlantic Ocean.

\subsubsection{4.4.1 Feasibility of nextRAD sequencing}
In \replaced{\textit{E. verrucosa}}{E. verrucosa}, isolating SNPs from across the genome appears to be feasible using nextRAD sequencing, even with the low-quality DNA obtained from many of our samples. Indeed, a number of samples had to be removed due to missing data, which may have resulted from poor DNA quality.  Fragmented DNA (or mutations) can result in the loss of sites where the nextRAD selective primers are designed to bind to (or restriction enzyme cut-sites in RADseq), potentially leading to allelic dropout.  Allelic dropout from these approaches can introduce bias into the dataset because it may produce overestimates of genetic variation within and between populations \citep{Gautier2013}. However, in this study, although most \textit{E. verrucosa} DNA samples showed evidence of degradation (Appendix A5), the samples retained after quality control contained relatively low amounts of missing data (<20 \%, Fig. \ref{nextradmissingdata}).  In RADseq and other methods that utilise NGS technology, missing data have been found to be quite prevalent regardless of the species studied \citep{Flanagan2017}.  The way in which this missing data is dealt with can have consequences for downstream analyses because, if not enough missing data is discarded, then one can run the risk of detecting spurious (non-biological) signals \citep{Flanagan2017}.  On the other hand, discarding too much missing data may introduce its own biases; for example, \citet{Huang2016} suggested that as the tolerance for missing data becomes more stringent, the mutational spectrum represented in the sampled loci is reduced, leading to the exclusion of loci with the highest mutation rates.  Nonetheless, it has been shown that informative SNP datasets can still be acquired when DNA is degraded \citep{Graham2015} and where there is a high proportion of missing data \citep{Chattopadhyay2014,Hodel2017,Tripp2017}.  

\subsubsection{4.4.2 Microsatellites vs. SNPs}
The microsatellite and SNP markers showed almost identical patterns of spatial structure.   Both sets of markers showed three distinct genetic clusters organised into samples from Britain-France, southern Portugal and western Ireland, with some weak genetic differentiation between samples from southwest Britain and northwest France.  The latter result could be due to reduced genetic connectivity across the English Channel, which would imply that drift is the dominant process driving these weak genetic differences.  In contrast to \textit{E. verrucosa}, a similar octocoral species, dead man's fingers (\textit{Alcyonium digitatum}), exhibited high genetic connectivity across this spatial scale (Holland et al. 2017), which suggests that reduced connectivity across the English Channel in \textit{E. verrucosa} may be related to life history strategies rather than a hydrological barrier to dispersal.  Although the effects of mid-channel currents and near-shore eddies \citep{Dauvin2012} on cross-Channel larval dispersal remains to be explored, previous research has also identified a potential genetic break around Brittany in some taxa, including polychaetes (Jolly et al. 2005), nematodes \citep{Wielgoss2008} and bivalves (Becquet et al. 2012). Conversely, other taxa show panmixia or high connectivity across this area, such as shrimps (Luttikhuizen et al. 2008), lobsters \citep{Triantafyllidis2005}, sea stars \citep{Baus2005} and cuttlefish \citep{Wolfram2006}.  This supports the notion that migration across the English Channel is likely dependent on species-specific factors rather than a universal barrier to gene flow.

What is also apparent from both marker systems is that within each region (i.e. southwest Britain, northwest France, southern Portugal and northwest Ireland), sampling sites are connected by high gene flow and/or have large \textit{N}\textsubscript{e}.  For example, in the microsatellite study, little differentiation was observed between the two most distant sites within Britain (Sawtooth Ledges in Lyme Bay and Skomer Island), implying that gene flow can potentially occur up to distances of 470 km.  This could not be tested in the SNP study because these sites were not included in the nextRAD libraries, but based on the genetic similarity of Hat, Lio (both Isles of Scilly sites) and Lun (Lundy Island), this provides evidence for gene flow potentially up to spatial scales of 192 km (Hat and Lun).  Moreover, the single sampling site from France, Lao, showed some genetic similarity with all three sites from Britain, suggesting that some gene flow may occur at scales up to 447 km.  However, as mentioned previously, this similarity could also be explained by large Ne which can mitigate the influence of drift; therefore, further work is required to assess whether the English Channel does indeed constitute a partial barrier to gene flow in \textit{E. verrucosa}.
	
The three distinct genetic groups found using both marker systems is robust evidence that the genetic structure of \textit{E. verrucosa} across the sampled area has been adequately resolved.  Still, the precise drivers of some of these patterns are yet to be determined with confidence.  In the microsatellite study, IBD appeared to explain the differentiation observed between sampling sites from Britain, France and Portugal \citep{Holland2017}.  Although the SNP study contained many fewer sampling sites, almost identical genetic patterns were found using differentiation indices (i.e. \textit{F}\textsubscript{st} and \textit{D}) and DAPC, which supports this explanation.  However, it is still unclear whether the genetically distinct colonies in northwest Ireland are driven by neutral processes (i.e. barrier to gene flow) or adaptive processes (i.e. selection). By isolating genome-wide SNPs using NGS, this allowed the exploration of outlier SNPs that may be potentially under divergent selection.  The DAPC patterns with and without the eight outlier SNPs detected were generally comparable to the DAPC using all SNP loci (Fig. \ref{nextrad_dapc_allsnps}).  This suggests that both putatively neutral and putatively adaptive SNPs may be contributing to the genetic distinctiveness of western Ireland colonies.  Interestingly, one of these SNP loci matched multiple translated proteins from stony corals on the NCBI database, but the functions of these proteins are currently unknown.  Ultimately, to fully explore the potential role of selection in these colonies inhabiting the peripheral range of \textit{E. verrucosa}, more genomic resources and gene annotations are required for this species or a closely related species.

In comparison to the spatial patterns of genetic structure, measures of genetic diversity were slightly contrasting between microsatellite and SNP markers.  Where microsatellite markers found a mixture of significant and non-significant heterozygote deficiencies across sites (Table 1 in Holland et al. 2017), SNP markers found significant heterozygote deficiencies across all sites included in the nextRAD study (Table \ref{tab:snpdiversity}).  In the microsatellite study, both Lio and Lun showed non-significant deficiencies, whereas they showed significant deficiencies in the SNP study.  A similar pattern was found for Bla; however, all other sites from the SNP study (Thu, Lao, Far and Por) agreed with the microsatellite markers.   This discrepancy in some sites is difficult to explain and could be due to one or more factors that are known to cause heterozygotes deficiencies: (i) the locus is under selection; (ii) the presence of null alleles that may lead to an excess of homozygotes; (iii) inbreeding; and (iv) the presence of population substructure leading to the Wahlund effect.  The latter two are the most likely explanations for the sites showing heterozygote deficiencies in both marker systems, which may be the product of low dispersal capacity of larvae and high self-recruitment at these sites.  Selection acting on certain loci could explain the discrepancies between the two marker systems, particularly as some loci in both the microsatellite and SNP studies were identified as being potentially under divergent selection (or genetic hitchhiking). However, in the SNP study, genetic diversity statistics were run without the eight outlier loci (Table \ref{tab:snpdiversity}) and significant homozygote excesses were still present, which indicates that selection is unlikely to be driving this deficiency of heterozygotes.  Of course, it is possible that there were false-negatives in the outlier tests of both studies, meaning some loci were considered neutral when they are actually under selection.  Null alleles were controlled for in both microsatellite and SNP studies; however, it is also possible that false-positives were present here which may have caused the discrepancy between the two marker systems.

Compared to other octocoral species, \textit{E. verrucosa} appears to have lower genetic diversity when both microsatellites and SNPs are considered (Table \ref{tab:octocoral_compare}).  For example, the closely related species \textit{Eunicella cavolini} \citep{Masmoudi2016} and \textit{E. singularis} \citep{Costantini2016}, \textit{A. digitatum} \citep{Holland2017}, as well as two other Mediterranean octocorals, \textit{Corallium rubrum} \citep{Ledoux2010a} and \textit{Paramuricea clavata} \citep{Mokhtar-Jamai2011} all have higher mean \textit{H}\textsubscript{e} and allelic richness at microsatellite markers, and the Pacific deep-sea octocoral \textit{Swiftia simplex} has higher mean \textit{H}\textsubscript{e} at SNP markers \citep{Everett2016}.  Although low genetic diversity observed from the 13 microsatellite markers may be explained by low polymorphism at some loci \citep{Holland2017}, the precise causes of this overall low genetic diversity remain to be determined but are perhaps the result of a combination of processes such as site-specific inbreeding, historic bottlenecks and the purging of alleles by strong drift or selection.

%% Table: pink sea fan SNP genetic diversity
\afterpage{
\begin{landscape}
\begin{table}[h!]
\centering
\small
\begin{threeparttable}
\caption[Octocoral genetic diversity comparison]
{Comparison of genetic diversity in temperate octocorals using microsatellite and SNP markers. \label{tab:octocoral_compare}}
\begin{tabular}{llllllll}
\hline \\[-1.0em]
\textbf{Family} & & & & & & &  \\
\textbf{  Species} & \textbf{Sea} & \textbf{Marker} & \textbf{Sites; \textit{N}} & \textbf{No. loci} & \textbf{mean \textit{H}\textsubscript{e}} & \textbf{mean \textit{A}\textsubscript{r}} & \textbf{Reference}  \\
\hline \\[-6pt]
\textbf{Alcyoniidae} & & & & & & & \\
\textit{  Alcyonium digitatum} & Atl & Msat & 20; 648 	& 8 &	0.63 & 4.18 & Holland et al. 2017 \\[6pt]
\textbf{Coralliidae} & & & & & & & \\
\textit{  Corallium rubrum} & Med & Msat & 40; 1,222 	& 10 &	0.74 &	7.30 & Ledoux et al. 2010  \\[6pt]
\textbf{Gorgoniidae} & & & & & & & \\
\textit{  Eunicella cavolini} &	Med & Msat & 19; 584 	& 7	& 0.56 &	4.24 & Masmoudi et al. 2016 \\
\textit{  Eunicella singularis} &	Med & Msat & 13; 301 	& 6	& 0.53 &	3.58 & Costantini et al. 2016 \\
\textit{  Eunicella verrucosa} &	Atl &  Msat & 27; 905 	& 13	& 0.42	& 2.58 & Holland et al. 2017 \\
 & Atl	& SNP	& 8; 77 &  3,743 & 0.25	& n/a & This study \\[6pt]
\textbf{Plexauridae} & & & & & & & \\
\textit{  Paramuricea clavata} & Med &	Msat & 39; 1114 	& 6	& 0.74 &	6.48 & Mokhtar-Jamai et al. 2011 \\
\textit{  Swiftia simplex} &	Pac & SNP & 4; 23 	& 786 &	0.26 &	n/a & Everett et al. 2016 \\[6pt]
\hline
\end{tabular}
  \begin{tablenotes}
  \setlength\labelsep{0pt} % align all notes with table
    \scriptsize
    \item Atl, Atlantic; Med, Mediterranean; Pac, Pacific.
    \item Msat, microsatellite; SNP, single nucleotide polymorphism.
    \item \textit{N}, number of individuals genotyped; \textit{H}\textsubscript{e}, expected heterozygosity; \textit{A}\textsubscript{r}, allelic richness.
    \item n/a, statistic not relevant because biallelic SNPs were used.
  \end{tablenotes}
\end{threeparttable}
\end{table}
\end{landscape}
}
%% end of table


\subsubsection{4.4.3 Implications for conservation and MPAs}
\replaced{\textit{Eunicella verrucosa}}{Eunicella verrucosa} is a conservation priority in England and Wales and is internationally recognised as a species facing a very high risk of extinction in the medium-term future.  Akin to tranche one, several of the MCZs recently designated in tranche two (January 2016) around southwest Britain (e.g. Bideford to Foreland Point, Hartland Point to Tintagel, and Runnel Stone) specifically identify \textit{E. verrucosa} as a protected feature in their designation listing.  In addition to protection from MCZs, it has been found that 60\% of \textit{E. verrucosa} colonies recorded by diver surveys in southwest Britain fall within areas protected by various other pieces of EU legislation \citep{Pikesley2016}.  However, not all of these MPAs prohibit bottom trawling (e.g. The Manacles, Whitsand and Looe Bay, and Chesil Beach and Stennis Ledges MCZs), the absence of which has been shown to positively affect the ability of pink sea fan populations to recover, albeit over a long time period of up to 20 years \citep{Kaiser2018}.  This suggests that a large proportion of \textit{E. verrucosa} in Britain remain vulnerable to anthropogenic disturbance and that the current level of protection afforded by MCZs is in some areas insufficient \citep{Lieberknecht2016,Pikesley2016}.

The genetic data for \textit{E. verrucosa} presented here (using both microsatellites and SNPs) highlight interesting findings relevant to the conservation of this ecologically important sessile species at local and regional scales; this has implications for both single-site feature designations and network connectivity.  For instance, the microsatellite and SNP results \replaced{suggest there is high genetic connectivity among populations of \textit{E. verrucosa} around coastal areas of southwest Britain. This may suggest high genetic connectivity, but large \textit{N}\textsubscript{e} can also produce similar patterns.  \textit{N}\textsubscript{e} is difficult is estimate in marine species \citep{Hare2011}, although methods to assess \textit{N}\textsubscript{e} based on linkage disequilibrium and coalescence analysis are being refined and developed that also incorporate genomic data \citep{Nunziata2018,Marandel2018}, which may improve estimates of \textit{N}\textsubscript{e} for marine species.}{indicate high genetic similarity} Moreover, inferences of gene flow suggested that populations of pink sea fan in southwest Britain act as a source for adjacent populations across the English Channel, highlighting the value in protecting these populations.  In the UK, the current recommendation for maintaining ecological connectivity between discrete habitats is the positioning of a MPA every 80 km or less \citep{Roberts2010}. Considering our findings from this population genetics study, it appears that these distances between MPAs would generally be sufficient to maintain genetic connectivity in \textit{E. verrucosa} across southwest England and Wales.  Of course, this assumes that contemporary local oceanic currents are able to facilitate the transport of enough larvae between sites, whether by a continuing stepping-stone process or a single dispersal event.

The genetic distinctiveness of \textit{E. verrucosa} populations from Donegal Bay in northwest Ireland, detected with both marker systems in this study, reinforces an argument for protecting these local sites.  Marginal populations often contain rare alleles (the highest extent of private alleles were found at these sites in the microsatellite study), but may recruit more slowly and be demographically isolated, implying reduced resilience to disturbance and therefore an increased need for protection.  Consequently, although \textit{E. verrucosa} are not specifically protected in the Republic of Ireland, the development and effective management of the Donegal Bay SAC could be crucial to the persistence of the distinct genetic variants of \textit{E. verrucosa} found in this area.

A similar pattern was found for pink sea fans in southern Portugal, which are also genetically distinct from other regions, but this is likely driven by the large geographical distance that separates these populations from those north of the Bay of Biscay.  This suggests that populations of \textit{E. verrucosa} would benefit from protection across the species range as a connected meta-population.  However, the implementation of such a measure for a single species is unlikely given the dwindling resources available to conservation managers. Nevertheless, highlighting these species-specific areas of unique diversity for consideration may prove valuable if national governments move towards an ecosystem-based management approach, whereby whole ecosystems are protected from disturbance.   

\subsubsection{4.4.4 Knowledge gaps and future research}
The two marker systems (microsatellites and SNPs) discussed in this chapter showed very similar results across the sites studied, which means we have a relatively robust understanding of the genetic patterns across the range sampled in \textit{E. verrucosa}.  However, increasing the number of genotyped individuals and sites for the SNP study would undoubtedly help to resolve whether there is any fine-scale genetic structure within regions, particularly for sites that were included in the microsatellite but not the SNP study.  Moreover, both marker systems only had samples from locations across the middle and northern range of \textit{E. verrucosa} -- no areas of the southern range were included in these studies (i.e. the Mediterranean).  In addition, there were several locations in the middle and northern range that could not be sampled due to logistical and time constraints (e.g. western and southern Ireland, and northern Spain).  Pink sea fans are apparently found in sufficient numbers for meaningful population genetic analysis in Tarragona (southeast Spain, Mediterranean), Marseille (southern France, Mediterranean), northern Spain (Aurelle et al. 2017), and Galway Bay (western Ireland, Wood 2013).  Inclusion of samples from these locations would enable three main hypotheses to be tested in future research:

Firstly, the role of IBD in driving population structure between \textit{E. verrucosa} colonies from southwest Britain, the Bay of Biscay and southern Portugal could be further explored by sampling colonies from northern Spain (Fig. \ref{fig:min_max_temp}). The null hypothesis would state that IBD is the main driver of population structure across these locations, which would agree with the results from this chapter and Holland et al. (2017).  The alternative hypothesis would suggest that colonies from northern Spain are genetically distinct from other locations, perhaps due to a barrier to gene flow or local adaptation, but that IBD still explains much of the differentiation observed between colonies from southwest Britain, northwest France and southern Portugal.


%% Figure: minimum and maximum SST
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter4/Heatmap_pink.pdf}
\caption [Minimum and maximum SST across the northeast Atlantic and the Mediterranean]
{Map of mean minimum (March) and mean maximum (August) sea surface temperature (SST) from 1870-2017 across the northeast Atlantic and the Mediterranean. White dots represent sampling sites from Holland et al. 2017 and the SNP study presented in this chapter.  Yellow triangles represent sites in which obtaining samples of \textit{E. verrucosa} would be advantageous for further exploring the drivers of population genetic structure in future research. SST data was extracted from the Met Office Hadley Centre Sea Ice and Sea Surface Temperature dataset.}
\label{fig:min_max_temp}
\end{figure}
%% end of figure

Secondly, additional samples from western or southern Ireland would allow further exploration of the relative contribution of gene flow, drift and selection to driving the distinct genetic profiles observed in Donegal Bay (northwest Ireland).  The null hypothesis would suggest that IBD is the primary driver of this differentiation, which would support a stepping-stone model of dispersal across southwest Britain and Ireland.  The first alternative hypothesis would suggest that IBD is not a primary driver, but that other neutral processes have mainly contributed to this differentiation (e.g. barrier to gene flow, past bottlenecks or periglacial refugia).  In contrast, the second alternative hypothesis would propose that natural selection is driving this differentiation, whereby these \textit{E. verrucosa} colonies have become adapted to specific environmental conditions in Donegal Bay.  In fact, this area represents the coldest local sea temperatures across the entire range of \textit{E. verrucosa} (Fig. \ref{fig:min_max_temp}), so it may be possible that these colonies have developed a tolerance to these sea temperatures, and that one or more of the loci studied here are involved in this adaptive tolerance.  Indeed, \citet{Pivotto2015} demonstrated that thermo-tolerance varied greatly along a depth gradient in a closely related species, \textit{E. cavolini}, which implies that some octocoral species have the capacity to adapt to different thermal regimes.
	
Lastly, samples from the Mediterranean would enable a direct comparison of the genetic diversity and population structure from sites in the southern range with sites in the middle and northern range of \textit{E. verrucosa}.  A number of questions could be put forward here, for example: (i) are spatial patterns of genetic diversity and population structure comparable between the two basins?; (ii) how connected are populations from the northeast Atlantic and the Mediterranean?; and (iii) are colonies inhabiting the periphery of the southern range locally adapted to the warmer temperatures?  Exploring this question of local adaptation is of particular interest because of the vast differences in sea temperatures present at the range limits of \textit{E. verrucosa}; for example, from 1870-2017, the average lowest temperature at Donegal Bay was 8.2\textsuperscript{o}C (in March), whereas the average highest temperature at Tarragona was 28.6\textsuperscript{o}C (in August) (Fig. \ref{fig:min_max_temp}).  Furthermore, investigating spatial patterns of population genetic structure across environmental gradients has enabled previous studies to reveal how different temperature regimes contribute to shaping the genetic structure of many marine organisms \citep{Benestan2016b,Diopere2017,VanWyngaarden2018,Lehnert2018}.

In conclusion, this chapter has confirmed that nextRAD sequencing is an appropriate RRS method for genome-wide SNP discovery in \textit{E. verrucosa}.  However, this was only possible after modifying a salting-out protocol which optimised the amount of high molecular weight DNA extracted.  Both markers systems, 13 microsatellites and 3,743 SNPs, showed almost identical patterns of spatial genetic structure, but slightly differing patterns were observed in the inbreeding coefficient at some sites.  Results from both marker systems suggest that the current network of MPAs in southwest England and Wales is sufficient for maintaining genetic connectivity in this species.  Inbreeding was found to be site-specific, but whether this has an impact on fitness and the long-term resilience of the sites in question is currently unknown.  Although we have a robust understanding of the spatial genetic patterns at the sites sampled, collecting samples from areas that were not represented previously will be advantageous going forward as it will enable a more thorough and complete assessment of these patterns across the entire range of \textit{E. verrucosa}.  Considering the ecological importance of this species, acquiring this information will be vital for the international conservation of this vulnerable species and for monitoring how climate change may impact populations at the range edges.



%--------------
%
% Chapter 5: European lobster SNP panel
% 
%--------------
\newpage
\section{Chapter 5: SNP discovery in European lobster (\textit{Homarus gammarus}) using RAD sequencing}
\rule{\textwidth}{0.8pt}
\vspace{1pt}

\noindent This chapter is based on a paper published in the journal \textit{Conservation Genetics Resources}.  The reference is given below and the full paper is available in the Appendix. \\

\noindent Jenkins TL, Ellis CD, Stevens JR (2018) SNP discovery in European lobster (\textit{Homarus gammarus}) using RAD sequencing. \textit{Conservation Genetics Resources} https://doi.org/10.1007/s12686-018-1001-8. \\

\noindent K. Moore (Exeter Sequencing Service) prepared the RADseq libraries. \\

\newpage
\subsection{5.0 Abstract}
The European lobster (\textit{Homarus gammarus}) is a decapod crustacean with a high market value and therefore their fisheries are of major importance to the economies they support. However, over-exploitation has led to profound stock declines in some regions such as Scandinavia and the Mediterranean. To manage this resource sustainably, knowledge of population structure and dispersal is crucial to inform management about stock structure, connectivity and food traceability. In this chapter, restriction-site associated DNA sequencing was used to develop novel SNP markers from 55 individuals originating from 27 geographically separate sampling locations which encompassed much of the species' range; SNPs were quality filtered, ranked using differentiation statistics and the top 96 SNPs adequate for primer design were retained. SNP markers were developed with the aim of maximising the power to detect genetic differentiation between: (i) Atlantic and Mediterranean lobsters and (ii) Atlantic lobsters. This SNP panel provides a useful resource for future studies of population genetic structure and assignment in \textit{H. gammarus}.

\subsection{5.1 Introduction}
The European lobster, \textit{Homarus gammarus} (Fig. \ref{fig:H_gam}), is a large decapod crustacean belonging to the family Nephropidae.  They are found in most coastal seas of the northeast Atlantic, historically ranging from northern Morocco to northern Norway, including the British Isles and Skagerrak, and the Mediterranean and western parts of the Black Sea. Though, they are extremely rare across their southern range (i.e. Morocco, parts of the Mediterranean and the Black Sea).  The species' high market value makes it a highly-prized seafood product, so its fisheries are of great importance to the local and regional economies they support.  However, current and historical over-exploitation has led to stock declines, some of which have been quite profound in several regions (e.g. Scandinavia, Mediterranean) and from which recovery has been slow or stagnant \citep{Kleiven2012}.  This has led to the rearing of \textit{H. gammarus} larvae in lobster hatcheries to produce juveniles which are released into the wild to supplement productive stocks where the risk of over-exploitation is high \citep{Ellis2015b}.  To manage this resource sustainably, developing genetic resources that enable the assessment of population structure and connectivity, as well as the ability to assign lobsters to their site of origin, is crucial to inform management about dispersal, stock boundaries and connectivity, and food traceability.

%% Figure: Homarus gammarus photo
\begin{figure}[h!]
\centering
\includegraphics[width=0.8\textwidth]{Chapter_figures/General_intro/Homarus.jpg}
\caption [\textit{Homarus gammarus}]
{European lobster (\textit{Homarus gammarus}).}
\label{fig:H_gam}
\end{figure}
%% end of figure

Over the last decade, genetic diversity and population structure has been investigated in \textit{H. gammarus} using traditional molecular markers including random amplification of polymorphic DNA (RAPDs) \citep{Ulrich2001}, allozymes \citep{Jorstad2005}, mtDNA restriction fragment length polymorphisms (RFLPs) \citep{Triantafyllidis2005} and microsatellites \citep{Huserbraten2013,Watson2016,Ellis2017}.  However, concerns over low sample sizes and geographical coverage, and limitations associated with the molecular markers used in these previous studies, question the power of these studies to adequately resolve the underlying population genetic structure in this species.  The isolation of thousands of genome-wide SNPs has commonly been attributed to maximising the power to resolve spatial patterns of genetic variation, showing particular promise for detecting subtle population structure in highly dispersive marine species that exhibit typically weak genetic differentiation (e.g. American lobsters, Benestan et al. 2015; great scallops, Vendrami et al. 2017). 

The aim of this study was to isolate thousands of genome-wide SNPs using RADseq and develop a small panel containing the most informative SNP markers that captures weak genetic differentiation between our sampling sites.  SNPs were chosen with the aim of maximising the power to detect genetic differentiation (i) between Atlantic and Mediterranean lobsters and (ii) between Atlantic lobsters.  Previous studies have suggested that \textit{H. gammarus} from the Atlantic and the Mediterranean are two distinct groups (e.g. \citeauthor{Jorstad2005} 2005; Triantafyllidis et al. 2005); thus, a SNP panel that can accurately assign unknown individuals to the Atlantic Ocean or the Mediterranean Sea may be a useful tool for food traceability and the monitoring of resources. 


\subsection{5.2 Materials and methods}
\subsubsection{5.2.1 Sample collection}
Tissue samples of adult European lobsters were mostly obtained by establishing national and international collaborations with a diverse number of personnal across western Europe.  The most common method used to collect samples comprised sending a sampling kit containing sterile gloves, 2 ml tubes filled with 95-100 \% ethanol and a signed return package to a sampler who had agreed to collect tissue samples.  The sampler then took a tissue sample from each lobster and placed the sample into one of the tubes provided.  When sampling was complete, the sampler sent the samples back to the lab, placing a note inside specifying the geographical origin of sampled lobsters.  Samplers consisted of fishermen, shellfish merchants / suppliers, restaurant owners, hatchery technicians, marine institutions, governmental bodies (e.g. IFCAs, BIM, etc.) and scientific researchers.  Some tissue samples were also taken by myself or archived tissue samples were provided by collaborators.  The type of tissue collected depended on the sampler and whether the lobster was caught to be sold or for scientific monitoring research.  For the majority of samples, a 1-2 cm section from one or two pleopods (swimmerets) (Fig. \ref{fig:lobster_tissue}) was removed from each lobster -- all lobsters destined to be sold were sampled in this way to maintain optimal condition and to avoid reducing sell-on prices.  The remaining tissue samples were taken from either the uropod (v-notches), the pereiopods (walking leg) or the antennae (Fig. \ref{fig:lobster_tissue}).  All tissue samples were placed in 95-100 \% ethanol and stored in a 4\textsuperscript{o}C cold room for long-term preservation.  

%% Figure: Lobster cross section and sampling tubes
\begin{figure}[h!]
\centering
\includegraphics[width=0.85\textwidth]{Chapter_figures/chapter5/Figure_Tissue_Sampling.pdf}
\caption [European lobster: diagram and tissue collection]
{European lobster side-view (top) and tubes containing tissue samples (bottom). Four different tissue types were taken for genetic analysis which was dependent on the individual conducting the sampling: v-notches, pleopods, pereiopods and antennae.}
\label{fig:lobster_tissue}
\end{figure}
%% end of figure

\subsubsection{5.2.2 DNA extraction}
Genomic DNA was extracted from all tissue types using a modified salting-out protocol designed to extract DNA from crayfish exoskeleton \citep{Li2011}.  Extracting DNA from pleopod material is challenging because their tough chitin exoskeleton resists grinding and lysing and they contain copious amounts of astaxanthin and other impurities which are not trivial to precipitate and can affect DNA purity \citep{Li2011}.  These constraints were addressed by firstly pulverising the tissue using a microbead and a TissueLyser (Qiagen) to homogenise the sample.  Secondly, sodium dodecyl sulphate (SDS) was added to the lysis buffer during the digestion to help reduce protein disulphide bonds and denature proteins.  Finally, two rounds of ammonium acetate treatment were carried out to completely remove proteins and cellular debris.  DNA was precipitated using 100 \% cold isopropanol and washed with 70 \% ethanol, followed by rehydration with 100 $\mu$l of nuclease-free water.  See Appendix A4 for a detailed step-by-step protocol.  The concentration and purity of all DNA extractions were quantified by spectrophotometry using a NanoDrop 1000.  In addition, the quality of the DNA samples were further evaluated by running the DNA on a 1 \% agarose gel and by quantifying their concentration with fluorometry using the Invitrogen Qubit Assay kit, which measures the amount of double-stranded DNA in the sample.    

\subsubsection{5.2.3 RAD sequencing}
RAD libraries were prepared by the Exeter Sequencing Service using Illumina Nextera XT barcodes.  Genomic DNA (400 ng) from each sample was sheared to an average size of 1000 bp using a Covaris E220 sonicator, having previously optimised for a size range of 800-1000 bp.  Fragmentation of eight samples was checked using a DNA 1000 screentape (Agilent).  The NEBNext Ultra II DNA library preparation kit was used for end-repair, A-tailing and for ligating P2 adapters, and the reactions were purified using AMPure XP magnetic beads (Beckmann Coulter).  P2-adapted DNA was then digested using the restriction enzyme SbfI at 37\textsuperscript{o}C for 4 hours and purified using AMPure XP beads to avoid heat-denaturing the enzyme, which can lead to a bias in the libraries.  Phased P1 adapters were ligated to the digested fragments and unligated adapters were removed using AMPure XP beads.  The P1 adapter was biotinylated at the 5'end of the top strand to enable capture with streptavidin beads.  After washing away fragments not bound to the P1 adapter, the DNA was amplified by PCR to add Nextera XT multiplexing barcodes and flow cell attachment regions.  Library quality and quantity was assessed using DNA screentapes. Equimolar pooling of the libraries was undertaken before size selection of libraries averaging 660 bp (inserts $\sim$530 bp).  The size selected pool was quantified by qPCR and stored at -20\textsuperscript{o}C prior to sequencing.

A pilot study using seven samples of \textit{H. gammarus} was firstly carried out to ensure RAD sequencing using SbfI was appropriate for this non-model species.  The SbfI restriction enzyme was initially chosen for two reasons.  Firstly, SbfI has been used in a previous study (Benestan et al. 2015) on a closely related species, the American lobster (\textit{Homarus americanus}), and good quality data were obtained.  Secondly, a theoretical prediction of the number of loci (RAD-tags) was estimated using the RAD-tag counter from GenePool (Edinburgh, UK).  The GC content was set to 0.40 and the genome size was set to 4,250 mb (Animal Genome Size Database).  The RAD-tag counter indicated there were approximately 24,480 cut sites and 48,960 RAD-tags in the genome, which was deemed adequate for this study.  The RAD libraries of these seven samples were added to a lane of Illumina HiSeq, with single-end sequencing at 150 bp read lengths.  After quality control and filtering, preliminary analysis indicated that the number of reads, the depth of coverage, and the number of SNPs discovered was adequate for this study.

After the successful pilot study, a more comprehensive RAD library was designed.  As the end goal was to discover and develop a panel of SNPs, the RAD library composed lobsters representing the geographical spread of our sampling coverage (Fig. \ref{fig:RADseq_coverage}), which was then sequenced at high coverage.  This approach attempted to incorporate as much of the genetic variation potentially present in our geographic sampling at the time of library construction (additional sites were sampled after this RAD library was sequenced).  In addition, sequencing at high coverage meant that more reads were merged to form RAD-tags and call SNPs which would theoretically improve the reliability of calling `true' SNPs.  This RAD library was composed of 48 lobsters, with one-three individuals per sampling site (Table \ref{tab:RADseq_sites}; Fig. \ref{fig:RADseq_coverage}).  The library was then sequenced on two lanes of an Illumina HiSeq 100 bp paired-end rapid run platform.  Paired-end sequencing was chosen because the reverse reads can be aligned back to the forward reads to increase the length of a RAD-tag, which may allow SNPs that are positioned at the end of the RAD-tag to be considered for primer development.

%% Figure: Lobster sampling map RAD library
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter5/Lobster_RAD1_Coverage.png}
\caption [European lobster: RADseq sampling coverage]
{European lobster: RADseq sampling coverage}
\label{fig:RADseq_coverage}
\end{figure}
%% end of figure


%% Table: European lobster RADseq sampling information
\afterpage{
% \newgeometry{left=10mm,right=10mm,top=20mm,bottom=20mm}
\begin{landscape}
\begin{table}
\footnotesize
\centering
\begin{threeparttable}
\caption[European lobster RADseq library sampling information]
{European lobster sampling information. A total of 55 individuals from 27 sampling sites were used in the preparation of RAD libraries. \label{tab:RADseq_sites}}
\begin{tabular}{llllllll}
\hline \\[-1.0em]
\textbf{Country} & \textbf{Site} & \textbf{Code} & \textbf{\textit{N}} & \textbf{Latitude} & \textbf{Longitude} & \textbf{Tissue type} & \textbf{Year}  \\
\hline \\[-1.0em]
Britain & Boscastle & Bos & 2	& 50.70 &	-4.71 & Pleopods & 2013 \\
        & Cromer & Cro & 2	& 52.95 &	1.31 & Pleopods & 2016 \\
        & Isle of Man & Iom & 2	& 54.24 &	-4.55 & Pleopods & 2016 \\
        & Isles of Scilly & Ios & 2	& 49.92 &	-6.33 & Pleopods & 2016 \\
        & Looe Harbour & Loo13 & 1	& 50.34 &	-4.43 & Pleopods & 2013 \\
        &              & Loo16 & 1	& 50.34 &	-4.43 & Pleopods & 2016 \\
        & Northumberland & Nhm & 1	& 55.24 &	-1.52 & Pleopods & 2013 \\
        & Orkney & Ork15 & 1	& 58.94 &	-2.74 & Pleopods & 2015 \\
        &        & Ork16 & 1	& 58.94 &	-2.74 & Pleopods & 2016 \\
        & Pembrokeshire & Pem & 2	& 51.81 &	-5.29 & Pleopods & 2016 \\
        & Shetland & She14 & 2	& 50.81 &	-1.53 & V-notches & 2014 \\
        & Shoreham-By-Sea & Sbs & 2	& 50.70 &	-0.26 & Pleopods & 2016 \\
Channel Islands & Alderney & Ald & 2	& 49.63 &	-2.12 & Pleopods & 2016 \\
                & Jersey & Jer & 2	& 49.05 &	-1.27 & Pleopods & 2016 \\
France  & Île de Ré, La Rochelle & Lro & 1	& 46.09 &	-2.12 & V-notches & 2013 \\
        & Île de Ré, La Rochelle & Idr16 & 2	& 46.09 &	-2.12 & V-notches & 2016 \\
Ireland & Aran Island & Ara & 2	& 55.00 &	-8.59 & Pleopods & 2016 \\
        & Cork Harbour & Cor & 2	& 51.84 &	-8.26 & Pleopods & 2016 \\
        & Donegal Bay & Don & 2	& 54.53 &	-8.50 & V-notches & 2016 \\
        & Hook Peninsula & Hoo & 2	& 52.12 &	-6.92 & V-notches & 2016 \\
        & Mullet Peninsula & Mul & 2	& 54.20 &	-10.05 & V-notches & 2016 \\
        & Ventry & Ven & 2	& 52.08 &	-10.05 & V-notches & 2016 \\
Italy   & Lazio & Laz & 2	& 41.44 &	12.55 & Antennae & 2013 \\
        & Sardinia & Sar13 & 2	& 40.12 &	9.01 & Antennae & 2013 \\
Norway  & Flodevigen & Flo15 & 1	& 58.42 &	8.76 & Pleopods & 2015 \\  
        &            & Flo16 & 2	& 58.42 &	8.76 & Pleopods & 2016 \\
        & Tvedestrand & Tve & 2 & 58.55 & 9.04 & Pleopods & 2015 \\
Spain   & Vigo & Vig13 & 2	& 42.49 &	-8.99 & Pleopods & 2013 \\
Sweden  & Gullmarfjord & Gul & 2	& 58.30 &	11.53 & Pereiopods & 2009 \\
        & Kavra & Kav & 2	& 58.33 &	11.37 & Pereiopods & 2007 \\
        & Singlefjord & Sin & 2	& 59.08 &	11.12 & Pereiopods & 2009 \\
\hline
\end{tabular}
  \begin{tablenotes}
  \setlength\labelsep{0pt} % align all notes with table
    \scriptsize
    \item \textit{N}, number of individuals used in the RAD library.
  \end{tablenotes}
\end{threeparttable}
\end{table}
\end{landscape}
% \restoregeometry
}
%% end of table


\subsubsection{5.2.4 Quality control}
Quality of the raw reads was initially examined using the FastQC software (Babraham Bioinformatics).  The sequencing data provided by the Exeter Sequencing Service had already been demultiplexed, which meant that fastq files were received for each individual (two files per individual because of paired-end sequencing).  The use of phased adapters meant there were one of four different combinations at the start of each forward read: \textcolor{red}{TGCA}, A\textcolor{red}{TGCA}, CA\textcolor{red}{TGCA} or GCA\textcolor{red}{TGCA} (nucleotides marked in red represent part of the SbfI cut-site).  As a result, the raw data was trimmed at the start of each read, such that each read started with \textcolor{red}{TGCA}, the SbfI cut-site.

Before further bioinformatics, raw reads from both the pilot study (seven individuals) and the RAD library (48 individuals) were pooled, creating a final RAD dataset containing 55 individuals (Table \ref{tab:RADseq_sites}).  Raw reads were then cleaned and truncated to 97 bp using the \texttt{process\char`_radtags} program in the Stacks software v1.45 \citep{Catchen2013a}.  A read was discarded if the score fell below a 90 \% probability of being correct (a raw Phred score of 10).  Reads were truncated to 97 bp because this was the shortest read length in the dataset and Stacks requires all reads to be the same length.  No barcode argument was provided because the data had already been demultiplexed. 

\subsubsection{5.2.5 Building loci \textit{de novo}}
The formation of RAD loci was carried out by running the wrapper script \texttt{denovo\char`_map.pl} in Stacks.  There are three main components of Stacks to consider when executing the \texttt{denovo\char`_map.pl} wrapper script for population genomic analyses: \texttt{ustacks}, \texttt{cstacks} and \texttt{populations}.  The \texttt{ustacks} program aligns the cleaned reads generated by \texttt{process\char`_radtags} into exactly matching stacks (or putative alleles) and then builds loci and calls SNPs \textit{de novo} for each individual.  With loci built for each individual, the \texttt{cstacks} program attempts to match loci across samples to create a catalog of loci across all of the samples.  The \texttt{populations} program computes population genetics statistics, as well as exporting SNP genotypes in user-defined formats (e.g. Genepop format).  The default assumes each individual in the RAD library is a separate sample; however, the user can submit a population map that specifies the sample (i.e. population or sampling site) that each individual belongs to.

There are three main parameters that control locus formation and SNP calling in Stacks: the minimum number of reads required to form a stack (\textit{m}; \texttt{ustacks}), the number of mismatches allowed between stacks to merge them into a locus (\textit{M}; \texttt{ustacks}) and the number of mismatches allowed between stacks during the construction of the catalog (\textit{n}; \texttt{cstacks}).  Exploring these parameters is important for optimising the number of assembled loci (RAD-tags), polymorphic loci and SNPs discovered in RAD datasets \citep{Paris2017}.  In this study, following the advice of \citet{Paris2017}, a strategy was implemented to optimise these three parameters in our RAD dataset.  Each parameter was sequentially changed while keeping the other two constant and the highest number of r80 polymorphic loci was considered the optimum parameter set.  The r80 polymorphic loci are identified by adjusting the \textit{r} parameter in the \texttt{populations} program, whereby a locus must be present in at least 80 \% (\textit{r} = 0.80) of the individuals in a population for it to be processed.  For this optimisation process, no population map was specified which meant that a locus had to be present in at least 44 individuals for it to be processed.  All other parameters were set to their default values during the optimisation of \textit{m}, \textit{M} and \textit{n}.

Once the optimum values \textit{m}, \textit{M} and \textit{n} were obtained, \texttt{denovo\char`_map.pl} was re-run with the maximum number of stacks at a single locus set to two (\texttt{--max\char`_locus\char`_stacks} 2, \texttt{ustacks}), meaning only loci with two alleles (biallelic loci) were allowed.  This parameter was enforced because, for a diploid species, if more than two alleles are allowed during the \textit{de novo} assembly of loci, the locus assembly could contain merged paralogs or additional alleles introduced as a result of sequencing error \citep{Catchen2013a,Mastretta-Yanes2014}.  Individuals were dropped from the analysis if their coverage was lower than 15x.

\subsubsection{5.2.6 SNP discovery}
SNPs were discovered by running the \texttt{populations} program with two sets of parameters. The first run was executed using all 55 samples with no population map, a minimum allele frequency set to 5 \% (\texttt{--min\char`_maf} 0.05), a maximum observed heterozygosity set to 0.5 (\texttt{--max\char`_obs\char`_het} 0.5) and \textit{r} was set to 0.8.  The rationale for this run was to search for SNPs that can detect genetic differentiation across broad geographical areas (i.e. between the Atlantic and Mediterranean basins).  The second run of populations searched for SNPs that could potentially detect genetic differentiation (if any) between geographical locations within the northeast Atlantic.  In this run, the Mediterranean and Skagerrak (Norway and Sweden) samples were removed because early genetic analyses indicated differentiation between these samples and all other Atlantic samples (see Results); accordingly, a population map for only the Atlantic samples was submitted and organised by geography (Table \ref{tab:stackspopmap}).  Since most sampling sites were only composed of two or three individuals, the \textit{r} parameter was set to 1, meaning a locus had to be present in every individual in that population for it to be processed.  Similarly, the \textit{p} parameter was set to nine (the total number of populations in the population map), which meant that a locus had to be present in all populations in the population map to be eligible.  All other parameters remained the same as the first run.

%% Table: lobster popmap
\begin{table}[h!]
\begin{small}
\caption[Stacks population map composed of nine putative populations and 40 individuals. ]
{Stacks population map composed of nine putative populations and 40 individuals.}
\label{tab:stackspopmap}
\begin{threeparttable}
\begin{tabular}{lll}
\hline \\[-1.0em]
\textbf{Population} & \textbf{Individuals} & \textbf{\textit{N}} \\
\hline \\[-1.0em]
English Channel & Ald3, Ald4, Jer2, Jer3, Sbs2, Sbs3	& 6 \\
West Ireland &  Ara1, Ara2, Don2, Don3, Ven1, Ven3, Mul3, Mul4	& 8 \\
Southwest England &	Bos55, Bos67, Ios5, Ios6, Loo13\textunderscore 31, Loo16\textunderscore 18	& 6 \\
Southeast Ireland &	Cor1, Cor2, Hoo1, Hoo2 &	4 \\
North Sea	& Cro4, Cro5, Nhm8 &	3 \\
Irish Sea &	Iom1, Iom4, Pem12, Pem14	& 4	\\
Orkney \& Shetland & Ork15\_6, Ork16\_1, She14\_3, She14\_4	& 4	\\ 
France &	Lro\_4, Idr16\_11, Idr16\_13	& 3\\
Spain & Vig13\_1, Vig13\_3 & 2 \\
\hline
\end{tabular}
  \begin{tablenotes}
  \setlength\labelsep{0pt} % align all notes with table
    \scriptsize
    \item \textit{N}, number of individuals.
  \end{tablenotes}
\end{threeparttable}
\end{small}
\end{table}
%% end of table

\subsubsection{5.2.7 Developing a SNP panel}
To develop the SNP panel, a set of criteria were implemented to filter out uninformative SNPs, and SNPs that were inadequate for primer design, whilst retaining the most informative SNPs that were suitable for primer design and high-throughput genotyping on a Fluidigm EP1 system (Fig. \replaced{\ref{fig:panel_flowchart}}{flowchart}). Firstly, the SNP dataset generated from the \texttt{populations} program were ranked and sorted by highest \textit{G}\textquotedbl \textsubscript{st} (Meirmans and Hedrick 2011) using the \texttt{diff\char`_stats} function from the R package MMOD \citep{Winter2012}.  Other differentiation statistics were computed (\textit{G}\textsubscript{st}, \citeauthor{Nei1983} \citeyear{Nei1983}; \textit{D}, \citeauthor{Jost2008} \citeyear{Jost2008}), but all statistics generally produced very similar results.  Post-ranking, information for the top 300 SNPs was recorded in a spreadsheet, including the position of the target SNP in the RAD-tag locus, the locus ID (given by Stacks) and the ranking according to highest \textit{G}\textquotedbl \textsubscript{st}.

%% Figure: SNP panel flowchart
\begin{figure}[h!]
\centering
\includegraphics[width=0.85\textwidth]{Chapter_figures/chapter5/Flow_diagram.pdf}
\caption [SNP panel development flowchart]
{SNP panel development flowchart.}
\label{fig:panel_flowchart}
\end{figure}
%% end of figure

Secondly, the locus ID for each SNP was queried using the Stacks SQL user interface to find out which loci had additional non-target SNPs in the RAD-tag.  The SNP database was sorted by the fewest number of SNPs per locus, followed by the highest ranking.  The ideal RAD-tag locus contained only one SNP as this was advantageous for primer design; however, RAD-tag loci with two or more SNPs were considered, particularly if they were ranked highly, because high-throughput assay designs, such as the Fluidigm EP1 system, can often deal with non-target SNPs assuming they are at least 30 bp away from the target SNP.

Lastly, to extend the length of RAD-tag loci, mini-contigs were assembled by aligning paired-end reads to the original RAD-tag sequence.  First, paired-end sequences were collated for each locus by executing the \texttt{sort\char`_read\char`_pairs.pl} program in Stacks using a whitelist of locus IDs.  Then, paired-end reads for each locus were aligned using the \texttt{exec\char`_velvet.pl} program with the \textit{M} parameter set to 100, meaning paired-end consensus sequences must be at least 100 bp long.  To build mini-contigs, the paired-end consensus sequences were then aligned to the original RAD-tag sequence (97 bp) using the alignment tool in Geneious v.10.1.3 \citep{Kearse2012}. SNPs with flanking sequences $<$25 bp or with non-target SNPs within 30 bp of the target SNP were discarded.  Finally, the mini-contig sequences for the remaining SNPs were submitted to Fluidigm's online portal for \textit{in silico} assay design.  This performs a quality control analysis on each sequence submitted, checking that SNP assays will be compatible with each other (i.e. no primer-dimers) and that parameters set by Fluidigm are met (e.g. no regions of large repeats, suitable GC content, adequate flanking sequence, etc.).


\subsection{5.3 Results}
\subsubsection{5.3.1 Sequencing and quality control}
The mean Q scores for the raw sequencing data for all samples ranged from 37-38 with 95 \% of reads having $>$Q30 score.  The fastQC report indicated that error bars towards the end of the read fell slightly below the threshold of Q28; however, these scores were deemed acceptable and, in any case, the last three bases were trimmed in \texttt{process\char`_radtags} during quality control.  In total, over 276 million reads (single and paired-end) were generated for the main RAD library (Fig. \ref{fig:lobster_RADreads}) and a mean average of 97.9 \% of reads were retained across all samples after quality control.

%% Figure: Lobster RADseq number of reads
\begin{figure}[h!]
\centering
\includegraphics[width=0.95\textwidth]{Chapter_figures/chapter5/Total_reads_lobster.pdf}
\caption [European lobster RADseq: number of reads per individual]
{Total number of reads per individual.}
\label{fig:lobster_RADreads}
\end{figure}
%% end of figure

\subsubsection{5.3.2 Parameter optimisation}
Initial results of \texttt{denovo\char`_map.pl} using default values of \textit{m}, \textit{M} and \textit{n} indicated high coverage for each RAD locus (38x mean average). Therefore, the parameter \textit{m} was kept at 3 as improving coverage was not an issue and values lower than 3 are not recommended \citep{Paris2017}.  The \textit{n} parameter was tested from 2-4 and the highest number of r80 polymorphic loci corresponded to a value of 3 (Fig. \ref{fig:RADseq_opt}).  Similarly, the \textit{M} parameter was tested from 2-5 and the highest number of r80 polymorphic loci corresponded to a value of 3 (Fig. \ref{fig:RADseq_opt}).  Subsequently, the optimum set of parameters used to form loci and call SNPs was \textit{m}=3, \textit{M}=3 and \textit{n}=3.

%% Figure: RADseq parameter optimisation
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter5/opt.png}
\caption [RADseq: optimising parameters]
{Exploring the optimum parameters for \textit{de novo} assembly of RAD loci.  The r80 represents the number of assembled loci, polymorphic loci or SNPs that are present in at least 80 \% of the individuals.}
\label{fig:RADseq_opt}
\end{figure}
%% end of figure

\clearpage
\subsubsection{5.3.3 SNP datasets}
The SNP dataset exported from the \texttt{populations} program using all 55 samples consisted of 7,022 SNPs.  Initial analysis using DAPC clustered these samples into three main groups: the Mediterranean (Italy samples), Skagerrak (Norway and Sweden samples), and the remaining samples from the northeast Atlantic (Fig. \ref{fig:RADseq_dapc}). Global \textit{F}\textsubscript{st} \citep{Weir1984} \added{between these three groups was 0.018}, calculated using the \texttt{diffCalc} function from diveRsity \citep{Keenan2013}.This was first evidence in this SNP dataset for genetic differentiation between the Atlantic and the Mediterranean in \textit{H. gammarus}.

\added{The SNP dataset exported from \texttt{populations} using only Atlantic samples (excluding Mediterranean and Skagerrak samples) consisted of 4,377 SNPs and 40 individuals organised into nine groups} (Table \ref{tab:stackspopmap}); global \textit{F}\textsubscript{st} \citep{Weir1984} for this dataset was 0.002.


%% Figure: RADseq ranking DAPC
\begin{figure}[h!]
\centering
% \begin{minipage}{0.5\textwidth}
  \includegraphics[width=0.9\textwidth]{Chapter_figures/chapter5/lobster_dapc_55ind_7022snps.pdf}
% \end{minipage}
% \begin{minipage}{0.5\textwidth}
%   \includegraphics[width=\textwidth]{Chapter_figures/chapter5/lobster_dapc_55ind_7290snps.pdf}
% \end{minipage}
\caption [RADseq: discriminant analysis of principle components]
{Discriminant analysis of principle components (DAPC) using 7,022 SNPs and 55 individuals.}
% (left); DAPC using  7,290 unique SNPs and 55 individuals (right).}
\label{fig:RADseq_dapc}
\end{figure}
%% end of figure

\clearpage
\subsubsection{5.3.4 SNP panel}
The SNP panel \replaced{was designed to contain SNPs with the highest \textit{G}\textquotedbl \textsubscript{st} from both datasets}{contained informative SNPs} that were eligible for primer design and synthesis; this study was limited to 96 SNP markers due to assay development costs and the requirements of the Fluidigm EP1 system (i.e. dependency on 96-well plates).  \added{Using the dataset composed of all 55 individuals, and after filtering SNPs that were inadequate for primer design, 21 SNPs (out of 7,022 SNPs) were selected to capture differentiation between Atlantic, Mediterranean and Skagerrak lobsters} (Fig. \ref{fig:RADseq_gstperSNP}). 

\added{Using the dataset containing 40 Atlantic samples (excluding Mediterranean and Skagerrak samples), 75 SNPs (out of 4,377 SNPs) were selected with the aim of capturing differentiation between lobsters originating from geographically separate regions in the Atlantic.}

%% Figure: RADseq Gst per SNP Atl-Ska-Med
\begin{figure}[h!]
\centering
  \includegraphics[width=\textwidth]{Chapter_figures/chapter5/Gst_per_snp_Atl-Ska-Med.pdf}
  \includegraphics[width=\textwidth]{Chapter_figures/chapter5/Gst_per_snp-Atl.pdf}
\caption [RADseq: \textit{G}\textquotedbl \textsubscript{st} per SNP]
{\textit{G}\textquotedbl \textsubscript{st} for each single nucleotide polymorphism (SNP) from two datasets: (top) SNPs ranked by \textit{G}\textquotedbl \textsubscript{st} when all samples were grouped by Atlantic, Mediterranean or Skagerrak; (bottom) SNPs ranked by \textit{G}\textquotedbl \textsubscript{st} using only Atlantic samples grouped by geographic region (Table \ref{tab:stackspopmap}). Red and blue points denote SNPs that were selected to compose the SNP panel.}
\label{fig:RADseq_gstperSNP}
\end{figure}
%% end of figure



\subsection{5.4 Discussion}
This study used RADseq to isolate SNP markers which capture genetic differentiation at different spatial scales across the range of \textit{H. gammarus}. In particular, a panel of 96 SNP markers has been developed to capture differentiation between Atlantic and Mediterranean lobsters, and to capture hierarchical differentiation between Atlantic lobsters. 	The RADseq dataset in this study, although composed of extremely small sample sizes, provided a glimpse into the patterns of genetic structure in this species.  A DAPC showed three distinct clusters: the Mediterranean, Skagerrak and the remaining northeast Atlantic samples (Fig. \ref{fig:RADseq_dapc}).  \added{However, although informative SNPs were selected to compose the SNP panel, only $\sim$0.30\% of SNPs were retained from the original 7,022 SNPs}, which may compromise power to detect these groups when further samples are genotyped.  On the other hand, a very low number of individuals composed sample sites in the RADseq dataset, so an increase in the number of individuals per sampling site may increase the power to detect this differentiation.  In any case, increasing the sample size for each site will facilitate more accurate and reliable calculations of allele frequencies.
	
Isolating markers from a small number of individuals using NGS can introduce effects of ascertainment bias \citep{Helyar2011}.  Ascertainment bias results from the selection of loci from an unrepresentative sample of individuals that are then used to infer aspects of genetic variation and population structure across a broader part of the species' range that were not represented in the original SNP discovery step \citep{Seeb2011}.  Acknowledging this potential bias is relevant for this study because several sites to be genotyped were sampled after the SNP discovery step.  However, ascertainment bias was mitigated in this study by including samples in the SNP discovery step from across most of the current range of \textit{H. gammarus}.
	
In studies of molecular ecology and population genetics, SNP genotyping is set to continuously increase in popularity as more markers are developed for model and non-model organisms \citep{Seeb2011}.  Indeed, recently there has been an increase in the number of informative SNP panels developed, which are likely to have useful applications in forensic science \citep{Martinsohn2009,Jacobs2017} and for the management of wild populations \citep{Meek2016,Baetscher2017}.  When developing these SNP panels, considering the statistical power of the markers to detect genetic differentiation between spatially discrete populations is critical; this is mainly influenced by the number of SNPs composing the panel and the sample size of each population being studied, with an increase in the latter more likely to provide greater improvements in statistical power \citep{Morin2009}.  Simulations using hypothetical data have shown that there is high power (0.80) to detect a differentiation level of \textit{F}\textsubscript{st}=0.01 with 75 SNPs and with a sample size of 30 individuals per population \citep{Morin2009}.  This suggests that with an increase in sample size per site, the SNP panel developed in this study for \textit{H. gammarus} has high power to detect differentiation and resolve population structure. The next chapter uses this SNP panel to genotype additional lobsters from sites included in this RADseq study and from other sites sampled after the RADseq study. 



%--------------
%
% Chapter 6: European lobster popgen
% 
%--------------
\newpage
\section{Chapter 6: Exploring patterns of genetic structure, connectivity and assignment in the European lobster (\textit{Homarus gammarus}) using SNP markers}
\rule{\textwidth}{0.8pt}
\vspace{1pt}

\noindent Tom L. Jenkins and Jamie R. Stevens conceived and designed the study.  Tom L. Jenkins coordinated and conducted sample collection, performed the SNP genotyping and analysed the data. Charlie D. Ellis and Alexandros Triantafyllidis assisted with sample collection.  Tom L. Jenkins, Charlie D. Ellis and Mats B.O. Huserbraten designed the individual-based model (IBM) for European lobster larvae.  Mats B.O. Huserbraten coded the IBM, performed simulations of larval dispersal and created the connectivity matrices. 

\newpage
\subsection{6.0 Abstract}
Delineating genetic structure and inferring connectivity in benthic marine invertebrates have been challenging due to typically weak genetic differentiation and the limited resolution offered by traditional genetic markers.  In this study, the population genetic structure of the European lobster (\textit{Homarus gammarus}), an economically important crustacean in the northeast Atlantic, was investigated across most of its current geographical range using a panel of genome-wide SNPs isolated using RADseq in a previous study.  The main aims were to test: (i) whether fine-scale population structure exists across the \textit{H. gammarus} range; (ii) whether patterns of connectivity are consistent with a stepping-stone model of dispersal; and (iii) whether this SNP panel can provide accurate individual assignment at various spatial scales, including ocean basin, region and sampling location of origin.  After quality control and filtering, 1,223 lobsters from 36 sampling sites (38 including temporal samples) were genotyped at 86 biallelic SNP loci using a Fluidigm EP1 system.  The results revealed strong genetic structure using all SNPs (global \textit{F}\textsubscript{st} = 0.06), partitioned between sites originating from the northeast Atlantic, the middle Mediterranean and the eastern Mediterranean (Aegean Sea).  In the northeast Atlantic, there was a pronounced genetic cline starting from the most southerly sampling site (Vigo, northwest Spain) to the most northerly sampling site (Lysekil, western Sweden).  Analysis indicated that isolation-by-distance is a key driver of this pattern; however, secondary contact after a period of isolation may also be responsible for this pattern.  This was supported by the Bay of Biscay and northwest Spain harbouring the highest genetic diversity, suggesting that these two sites may have served as glacial refugia which preceded secondary introgression of northward dispersal after the Last Glacial Maximum.  Individual assignment was 100 \% accurate to basin of origin (i.e. Atlantic or Mediterranean); however, power was reduced for region of origin and significantly reduced for sampling location of origin. The findings of this study should be useful for lobster fisheries management, but also serve as another taxon to assess connectivity between MPAs in British waters.


\subsection{6.1 Introduction}

\subsubsection{6.1.1 European lobster biology}
The European lobster (\textit{Homarus gammarus}) is a large decapod crustacean belonging to the family Nephropidae (clawed lobsters).  Adults are usually territorial and found hiding in crevices within hard substrates composed of rock or compressed mud in coastal areas from the low-tide mark to 150 m, but typically at depths not exceeding 50 metres.  The current range of \textit{H. gammarus} extends over most of the northeast Atlantic, from northern Norway to northern Morocco, and parts of the Mediterranean and the western Black Sea where they are found considerably more sparsely \citep{Spanier2015}.  They are not found in the Baltic Sea, presumably because the larvae or adults cannot survive in the lower salinity and temperature conditions of the Baltic Sea \citep{Jorstad2005}, as is reported in some other marine invertebrates \citep[e.g.][]{Podbielski2016}.

The genus \textit{Homarus} is composed of \textit{H. gammarus} and the closely related species, the American lobster (\textit{Homarus americanus}), which occupies similar habitats in the western Atlantic, ranging from Cape Hatteras (North Carolina, USA) in the south to the Strait of Belle Isle (Labrador, Canada) in the north \citep{Benestan2015}.  The next closest relative of these two clawed lobsters is the Norway lobster, \textit{Nephrops norvegicus}, which was recently confirmed using data from complete mitogenomes \citep{Shen2015}.  Homarid lobsters have two large specialised claws; one is blunt and designed for crushing, while the other claw is serrated and designed for slicing prey.  The diet of adult lobsters, who mainly forage nocturnally, comprises of mostly benthic invertebrates such as crustaceans, molluscs, echinoderms and polychaetes, but can include algae, zooplankton and some fish.  As with most crustaceans, the internal soft tissues are protected by a rigid exoskeleton, the front part of which forms the carapace and the hind portion comprises the abdomen and tail, which can be contracted quickly for rapid movements to escape danger.
	
Female \textit{H. gammarus} reach sexual maturity in 5-7 years, at a carapace length (CL) of approximately 82-96 mm, but the size of sexual maturity can be geographically variable \citep{Ellis2015}.  Mating typically occurs in late summer and fertilised eggs are carried underneath the abdomen for 9-12 months while the embryos develop \citep{Schmalenbach2010}; at this stage the female is said to be `berried' (Fig. \ref{fig:berried_lobster}).  Hatching typically occurs in spring-summer from late May to August and stage 1 larvae are discharged into the water column at night to begin their pelagic larval phase \citep{Schmalenbach2010}.       

%% Figure: Berried lobster
\begin{figure}[h!]
\centering
\includegraphics[width=0.6\textwidth]{Chapter_figures/chapter6/Berried_Lobsters.jpg}
\caption [A berried European lobster]
{A berried European lobster.}
\label{fig:berried_lobster}
\end{figure}
%% end of figure

The larvae are planktotrophic, feeding opportunistically on phytoplankton and zooplankton in the water column, and newly hatched larvae are thought to be positively phototaxic, which decreases as larvae age and develop \citep{Schmalenbach2010a}. Laboratory experiments have also shown that larvae are positively rheotactic, meaning they have some ability to swim against (horizontal or vertical) water currents \citep{Schmalenbach2010a}. The PLD is estimated to last 14-28 days and is dependent on sea temperatures encountered during transience \citep{Schmalenbach2010}.  The PLD is inversely proportional to temperature, meaning colder temperatures increase the PLD and vice versa; optimal larval survival in lobster larvae from Helgoland (southern North Sea) was found to be between 16\textsuperscript{o}C and 22\textsuperscript{o}C \citep{Schmalenbach2010}.  During this pelagic phase, larvae moult and develop into two intermediate stages, before free-swimming stage 4 post-larvae seek suitable benthic habitat to settle.  

Contrary to \textit{H. americanus}, the juvenile stage of \textit{H. gammarus} is particularly cryptic and not well known.  Juveniles of \textit{H. americanus} have been found to favour cobble-boulder substrata \citep{Wahle1997}; however, using a similar sampling method, no \textit{H. gammarus} juveniles were found in this habitat across Norway, Ireland, Italy and the UK \citep{Linnane2001}.  Common garden experiments suggest that \textit{H. gammarus} juveniles may favour shelters that provide extensive tunnel systems (\citeauthor{Jorstad2001} \citeyear{Jorstad2001}; \citeyear{Jorstad2009}), but exact nursery habitats in the wild still remain relatively unknown.  Adult \textit{H. gammarus} are thought to be relatively sedentary, with limited movement away from their home range \citep{Skerritt2015}.  This is supported by several tagging studies, which demonstrate that adult movements are generally <3 km for periods up to one year \citep{Oresland2013,Skerritt2015}, though some individuals have been recorded to travel up to 21 km away from their tagged origin \citep{Huserbraten2013}.    


\subsubsection{6.1.2 Fisheries}
Global landings of the European lobster have been steadily increasing since the 1980s, with recent landings of 5,194 tonnes in 2014 (Fig. \ref{fig:global_landings}, \citeauthor{FAO2018} \citeyear{FAO2018}).  The American lobster fishery, in contrast, is significantly more productive, with recent global landings close to 160,000 tonnes in 2014.  However, compared to the typical market price of the American lobster ($\$$6.31-9.24 kg\textsuperscript{-1}), the European lobster has much higher value, fetching a price of \textsterling 10.10-13.24 kg\textsuperscript{-1} at the time of writing \citep{FIS2018}.  

%% Figure: Global landings of European lobster
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/Global_landings.pdf}
\caption [Global landings of European lobster]
{Global landings of European lobster (\textit{Homarus gammarus}) from 1950-2014. Data from the Fisheries and Aquaculture Department of the Food and Agriculture Organization of the United Nations.}
\label{fig:global_landings}
\end{figure}
%% end of figure

The high market value of \textit{H. gammarus} makes it a prized seafood product for fishermen, so its fisheries are of great importance to the local and regional economies they support.  However, current and historical over-exploitation has led to profound stock declines, with several regions (e.g. Scandinavia, Mediterranean, western Black Sea) experiencing stock collapses and from which recovery has been slow or stagnant \citep{Kleiven2012}.  For example, in Norway, dramatic declines in landings from 1960-1980 indicated a severe collapse in the fisheries, and stocks have since remained at $<$10 \% of their pre-1960 levels (Agnalt et al. 2007; 2009).  Historically, \textit{H. gammarus} was of particular interest to humans in parts of the Mediterranean and the western Black Sea; however, centuries worth of over-exploitation likely led to collapses and have possibly contributed to the current sporadic and scarce distribution and abundance of \textit{H. gammarus} across these areas (Spanier et al. 2015).   

The majority of landings of European lobster now originate from the coastal fisheries of the UK, Channel Islands and the Republic of Ireland.  In England, there are six Lobster Fishery Units (LFUs) (Fig. \ref{fig:LFU}), which are reported to be based on the geographic distribution of the fisheries and what is known about hydrology and larval dispersal \citep{CEFAS2017}.  Each LFU spans one or more local Inshore Fisheries and Conservation Authorities (IFCAs) (Fig. \ref{fig:LFU}), which manage coastal fisheries out to six nautical miles, whereas DEFRA and the Marine Management Organisation (MMO) are responsible for managing lobster fisheries beyond six nautical miles \citep{CEFAS2017}.  Local legislation can differ among the ten IFCAs; for example, in European waters there is a minimum landing size of 87 mm (CL), but several IFCAs (e.g. Devon & Severn, Cornwall and Isles of Scilly) have introduced a larger minimum landing size of 90 mm.  In Scotland, similar local regulations exist whereby, at the time of writing, most areas have a minimum landing size of 87mm, except for the Outer Hebrides, Orkney and Shetland where a minimum landing size of 90mm is enforced.  This minimum landing size is linked to the age of maturity of females, in which most females (80-92 \%, CEFAS 2017) this size are thought to be mature and have had the opportunity to spawn at least once. However, spatial and temporal variability and uncertainty in the size-maturity relationship in \textit{H. gammarus} \citep{Tully2001} has led to additional measures at some local and regional levels.  For instance, it is illegal to land (i) lobsters that have been v-notched (a mark that remains for two-three moults), (ii) lobsters that have mutilated tail fans or (iii) lobsters that have been tagged, and in English waters it is also illegal to land berried females.  The rationale behind this movement is to ensure a portion of the breeding female lobster population can be protected from fishing pressure.   Moreover, in Scotland and the Republic of Ireland, there is a maximum size limit of 145mm (155mm in Orkney and Shetland) and 127mm, respectively.  Female (and male) fecundity in \textit{H. gammarus} has been found to be size-specific, with considerable variation between smaller spawners (CL 74mm, $\sim$4,000 eggs) and larger spawners (CL 151mm, $\sim$40,000 eggs) \citep{Agnalt2008}; therefore, this legislation attempts to protect larger and more fecund females that contribute disproportionately to egg production.

%% Figure: LFUs
\begin{figure}[h!]
\centering
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/LFUs.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/IFCAs.png}
\caption [Lobster Fishery Units and IFCA boundaries]
{Lobster Fishery Units (left) and Inshore Fisheries and Conservation Authority boundaries (right) in England.}
\label{fig:LFU}
\end{figure}
%% end of figure


Despite the legislation established for European lobster fishing, inaccurate landing reports from both commercial and recreational fishers can subvert these measures and undermine effective management of stocks \citep{Kleiven2012}.  For example, in southeast Norway, \citet{Kleiven2012} found that only 24 \% of lobster landed commercially were sold through the legal market and documented.  The authors also found that recreational fishing, for which landings are unrecorded, accounts for 65 \% of the total catch in the study area; however, whether this proportion is representative of other fishing areas across the northeast Atlantic is not known.  In any case, this suggests that illegal, unreported and unregulated (IUU) fishing may seriously underestimate the actual landings of \textit{H. gammarus}, and that many local and regional fisheries may have higher risks of over-exploitation than previously thought. 

\subsubsection{6.1.3 Hatchery stocking and aquaculture}
Stock declines in several regions have led to the rearing of \textit{H. gammarus} larvae in lobster hatcheries to produce juveniles which are released into the wild to supplement wild stocks \citep{Bannister1998,Ellis2015b}.  Such stocking has been implemented to either restore depleted or extirpated populations (restocking) or augment natural recruitment to maintain / increase yields (stock enhancement) \citep{Bell2008}.  Initially, stock enhancement trial programs in the UK, France and Norway showed that some cultured juveniles can survive to adult sizes in the wild, which suggests that stocking may be a viable approach to augment natural stocks \citep{Ellis2015b}.  However, uncertainty still surrounds whether stocking is economically viable, both in terms of the cost and labour required to produce cultured \textit{H. gammarus} juveniles, and to monitor their performance in wild populations.

The most common method of producing juveniles has been to loan or purchase wild berried females from fishers or merchants and to hold the females in aquaria until the eggs hatch \citep{Ellis2015b}.  Larvae are then normally reared to at least developmental stage 5 (post-larval stages), and up to one year in some hatcheries, then released into natural habitats at an early benthic juvenile stage \citep{Ellis2015b}.  This approach has released over 1.4 million cultured juveniles into European waters from 1983-2013, of which roughly 90 \% of these releases were used for stock enhancement programs (UK, Ireland and France) and 10 \% were used for restocking depleted populations (Norway, Germany and Italy) \citep{Bannister1998,Schmalenbach2011,Ellis2015b}.

Recently, there have been technological advances in the aquaculture of European lobster which has seen juveniles grown in sea-based containers submerged in estuaries near to hatcheries \citep{Beal2002,PerezBenavente2010,Daniels2015}. This mariculture approach is hoped to promote traits and behaviours that would develop in the natural environment and serve as an acclimation step before the juveniles are released \citep{Ellis2015b}.  In addition, no supplemental feed is required as the juveniles consume natural prey in the environment.  Such a project exists in southwest England, termed Lobster Grower (National Lobster Hatchery), whereby juveniles are reared in sea-based containers deposited in the estuary of the River Camel near Padstow.  These approaches have massive potential to improve the ecological conditioning of cultured lobsters, which could increase their survival in the wild.  Moreover, a future possibility of rearing juvenile lobsters to a marketable size could substantially reduce the pressure on wild populations. However, this approach is still in its infancy, and further cost and feasibility assessments are necessary to determine its economic viability \citep{Daniels2015}.  


\subsubsection{6.1.4 Previous genetic research}
Genetic techniques offer the only approach for exploring the population structure and genetic diversity of \textit{H. gammarus} stocks and for potentially determining the fate of hatchery-reared juveniles via parentage or population assignment \citep{Gagnaire2015}. Genetic research of paternity in both \textit{H. gammarus} and \replaced{\textit{H. americanus}}{H. americanus} has yet to be applied to tracking juveniles bred from a hatchery; however, these studies have provided insights into mating patterns among males and females at different spatial scales \citep{Gosselin2005,Ellis2015a,Sordalen2018}.

Population genetic structure has been investigated in \textit{H. gammarus} using traditional molecular markers including random amplification of polymorphic DNA (RAPDs) \citep{Ulrich2001}, allozymes \citep{Jorstad2005}, mtDNA restriction fragment length polymorphisms (RFLPs) \citep{Triantafyllidis2005} and microsatellites \citep{Huserbraten2013,Watson2016,Ellis2017}.  At regional and basin-wide scales, the general consensus using allozymes and mtDNA RFLPs suggests that lobsters from the Mediterranean, northern Norway and Oosterschelde (Netherlands) are genetically differentiated from each other and all other samples included in the studies (global theta=0.016, Jorstad et al. 2005; global \textit{G}\textsubscript{st}=0.078, Triantafyllidis et al. 2005).  Ulrich et al. (2001) found similar patterns with fewer sampling sites and RAPD markers, and analysis with geographic distances suggested that this may be a product of IBD.  Using 14 microsatellite markers, Ellis et al. (2017) were able to distinguish Skagerrak from all other sites sampled; however, these two sites from Skagerrak were genotyped by a different laboratory which suggests that artefacts from cross-calibration cannot be ruled out as a causal factor of the differentiation detected.

In comparison, at finer scales and using 12 microsatellites, high gene flow was found between sites within the Skagerrak region (Huserbraten et al. 2013) and between sites within the Irish Sea / Bristol Channel area (Watson et al. 2016).  Moreover, Ellis et al. (2017) found high gene flow between all northeast Atlantic sites sampled except for Scandinavia. However, concerns over low sample sizes and geographical coverage, and limitations associated with the molecular markers used in these previous studies, question the power of these studies to adequately resolve the underlying population structure in this species.  These potential limitations can be addressed by sampling more comprehensively across the current geographical range of \textit{H. gammarus} and by investigating the genetic structure using informative SNP markers isolated from across the genome.


\subsubsection{6.1.5 Study aims}
The main aim of this study was to use the panel of SNPs developed by \citet{Jenkins2018} to explore population genetic structure, connectivity and assignment in \textit{H. gammarus} across our sampling sites.  Specifically, this study asked three questions: (i) is there evidence of fine-scale population structure across the sampled range of \textit{H. gammarus}?; (ii) are patterns of connectivity consistent with isolation-by-distance and a stepping-stone model of dispersal?; and (iii) can this panel of SNPs provide accurate assignment success at various spatial scales, including ocean basin of origin, region of origin and sampling location of origin.  To conclude, future research objectives and implications for fisheries management and hatchery stocking are discussed. 


\subsection{6.2 Materials and methods}

\subsubsection{6.2.1 Sampling}
Tissue samples of adult European lobsters were obtained in the same way as described in section 5.2.1.  In total, 36 sites were sampled (38 including temporal samples), mainly during 2016 and 2017 (Table \ref{tab:snp_sites}, Fig. \ref{fig:lobster_map}).  Because \textit{H. gammarus} are extremely rare and precious in the Mediterranean, tissue and DNA samples collected in previous studies (Triantafyllidis et al. 2005; Ellis et al. 2017) were also included in this study (Table \ref{tab:snp_sites}).  Moreover, several Scandinavian samples collected in 2007 and 2009 were provided by Carl Andre (University of Gothenburg), which presented an opportunity to explore temporal genetic patterns in the Skagerrak area.  This sampling strategy covered most of the current northeast Atlantic and Mediterranean distribution of \textit{H. gammarus}, with only samples from northern Norway (northern limit), Portugal and Morocco (southern Atlantic), the western Mediterranean, and the western Black Sea (southern limit) absent, for which sourcing samples was attempted but was extremely difficult.  All tissue samples were placed in 95-100 \% ethanol and stored in a 4\textsuperscript{o}C cold room for long-term preservation.  

\subsubsection{6.2.2 DNA extraction and SNP genotyping}
Genomic DNA was extracted from all tissue types using a salting-out protocol (Appendix A4). The concentration and purity of all DNA extractions were quantified by spectrophotometry using a NanoDrop 1000.  SNP genotyping was carried out on a Fluidigm EP1 system using the 96 SNPs isolated and developed by \citet{Jenkins2018}.  Specific Target Amplification was carried out (as advised by Fluidigm) because it increases the copy numbers of the desired sequence containing the SNP, which can improve genotyping call rates and accuracy, particularly for heterozygous samples \citep{Bhat2012}.  Assays and samples were run on a 96.96 Dynamic Array integrated fluidic circuit (IFC) and genotypes were called using the Fluidigm SNP Genotyping Analysis software (Fig. \ref{fig:fluidigm}).  A confidence threshold of 95 \% was enforced and data were normalised using at least two negative controls per IFC run.  The performance of the algorithm was checked after each analysis and obvious mistakes were amended; this included invalidating samples or assays that performed badly and correcting calls where the clustering algorithm was erroneous or ambiguous.


%%% Table: European lobster sampling information for all sites
% \afterpage{
% \newgeometry{left=10mm,right=10mm,top=20mm,bottom=20mm}
\begin{landscape}
\begin{table}
\scriptsize
\centering
\begin{threeparttable}
\caption[European lobster sampling information and genetic diversity statistics]
{European lobster sampling information and genetic diversity statistics. \textit{F}\textsubscript{is} values significantly different from zero are highlighted in bold.}
\label{tab:snp_sites}
\begin{tabular}{llllllllllll}
\hline \\[-1.0em]
\textbf{Country} & \textbf{Site} & \textbf{Code} & \textbf{\textit{N}\textsubscript{1}} & \textbf{\textit{N}\textsubscript{2}} & \textbf{Lat} & \textbf{Lon} & \textbf{Tissue type} & \textbf{Year} & \textbf{\textit{H}\textsubscript{o}} & \textbf{\textit{H}\textsubscript{e}} & \textbf{\textit{F}\textsubscript{is}} \\
\hline \\[-1.0em]
Britain & Bridington & Brd & 36 & 36 & 54.07 & -0.17 & Pleopods & 2017 & 0.36 & 0.35 & -0.018 \\
   & \textsuperscript{a}Cromer & Cro & 36 & 35 & 52.94 & 1.31 & Pleopods & 2016 & 0.36 & 0.35 & -0.027 \\
   & Eyemouth & Eye & 36 & 27 & 55.88 & -2.07 & Pleopods & 2017 & 0.37 & 0.36 & -0.034 \\
   & Outer Hebrides & Heb & 36 & 36 & 57.79 & -7.25 & Pleopods & 2017 & 0.39 & 0.37 & \textbf{-0.053} \\
   & \textsuperscript{b}Isle of Man & Iom & 36 & 35 & 54.12 & -4.50 & Pleopods & 2016 & 0.40 & 0.38 & \textbf{-0.044} \\
   & \textsuperscript{a}Isles of Scilly & Ios & 36 & 36 & 49.92 & -6.33 & Pleopods & 2016 & 0.39 & 0.38 & -0.015\\
   & \textsuperscript{a}Looe Harbour & Loo & 36 & 36 & 50.35 & -4.44 & Pleopods & 2016 & 0.39 & 0.37 & \textbf{-0.066}\\
   & \textsuperscript{c}Llyn Peninsula & Lyn & 36 & 34 & 52.93 & -4.62 & Pleopods & 2017 & 0.41 & 0.38 & \textbf{-0.068}\\
   & Orkney & Ork & 36 & 36 & 59.00 & -2.83 & Pleopods & 2017 & 0.36 & 0.36 & 0.006\\
   & Padstow & Pad & 36 & 36 & 50.56 & -4.98 & Pleopods & 2017 & 0.37 & 0.36 & -0.023\\
   & Pembrokeshire & Pem & 36 & 36 & 51.81 & -5.29 & Pleopods & 2016 & 0.38 & 0.37 & -0.017\\
   & Shetland & She & 36 & 36 & 60.17 & -1.40 & Pleopods & 2017 & 0.37 & 0.36 & -0.025\\
   & Shoreham-By-Sea & Sbs & 36 & 36 & 50.82 & -0.26 & Pleopods & 2016 & 0.37 & 0.36 & -0.030\\
   & Sula Sgeir & Sul & 36 & 36 & 59.09 & -6.16 & Pleopods & 2017 & 0.35 & 0.36 & 0.028\\
Channel Islands & Jersey & Jer & 36 & 36 & 49.16 & -2.12 & Pleopods & 2016 & 0.37 & 0.37 & -0.002\\
France & Île de Ré, La Rochelle & Idr16 & 32 & 32 & 46.13 & -1.25 & V-notches & 2016 & 0.38 & 0.38 & -0.006\\
 &  & Idr17 & 29 & 29 & 46.13 & -1.25 & V-notches & 2017 & 0.40 & 0.39 & -0.024\\
Germany & Helgoland & Hel & 36 & 35 & 54.18 & 7.90 & Pleopods & 2017 & 0.33 & 0.33 & -0.012\\
Greece & Alexandroupoli & Ale & 35 & 28 & 40.84 & 25.87 & DNA & 1999-2001 & 0.33 & 0.34 & 0.040\\
 & Skyros & Sky & 37 & 37 & 38.82 & 24.53 & DNA & 1999-2001 & 0.35 & 0.34 & -0.033\\
  & Thermaikos Bay & The & 37 & 36 & 40.36 & 22.88 & DNA & 1999-2001 & 0.35 & 0.34 & -0.035\\
   & Toronaios Bay & Tor & 37 & 37 & 40.17 & 23.54 & DNA & 1999-2001 & 0.33 & 0.33 & -0.001\\
Ireland & Cork & Cor & 32 & 32 & 51.84 & -8.26 & Pleopods & 2016 & 0.38 & 0.38 & 0.006\\
 & \textsuperscript{c}Hook Peninsula & Hoo & 36 & 36 & 52.12 & -6.92 & V-notches & 2016 & 0.39 & 0.37 & -0.033\\
  & \textsuperscript{c}Kilkieran Bay & Kil & 35 & 35 & 53.28 & -9.77 & Pleopods & 2016 & 0.38 & 0.37 & -0.031\\
   & Mullet Peninsula & Mul & 36 & 36 & 54.19 & -10.15 & V-notches & 2016 & 0.37 & 0.38 & 0.016\\
    & Ventry & Ven & 36 & 36 & 52.12 & -10.35 & V-notches & 2016 & 0.39 & 0.37 & \textbf{-0.046}\\
Italy & Lazio & Laz & 7 & 5 & 41.44 & 12.62 & Antennae & 2013 & 0.38 & 0.31 & \textbf{-0.234}\\
 & Tarquinia, Lazio & Tar & 7 & 5 & 42.23 & 11.68 & Antennae & 2013 & 0.42 & 0.32 & \textbf{-0.292}\\
  & Sardinia & Sar13 & 7 & 7 & 41.26 & 9.20 & Antennae & 2013 & 0.32 & 0.29 & -0.092\\
   &  & Sar17 & 15 & 15 & 41.26 & 9.20 & Pleopods & 2017 & 0.34 & 0.34 & -0.019\\
Netherlands & Oosterschelde & Oos & 40 & 40 & 51.61 & 3.70 & Pleopods & 2017 & 0.31 & 0.32 & 0.010\\
Norway & Flodevigen & Flo & 36 & 36 & 58.42 & 8.76 & Pleopods & 2016 & 0.34 & 0.33 & -0.027\\
 & Singlefjord & Sin & 36 & 36 & 59.08 & 11.12 & Pleopods & 2009 & 0.34 & 0.33 & \textbf{-0.041}\\
Spain & Vigo & Vig & 36 & 36 & 42.49 & -8.99 & Pleopods & 2017 & 0.40 & 0.39 & -0.017\\
Sweden & Gullmarfjord & Gul & 36 & 35 & 58.25 & 11.33 & Pereiopods & 2009 & 0.37 & 0.34 & \textbf{-0.072}\\
 & \textsuperscript{d}Kavra & Kav & 36 & 36 & 58.33 & 11.37 & Pereiopods & 2007 & 0.36 & 0.34 & \textbf{-0.056}\\
  & Lysekil & Lys & 36 & 36 & 58.26 & 11.37 & Pleopods & 2017 & 0.32 & 0.33 & 0.014\\
\hline
\end{tabular}
  \begin{tablenotes}
  \setlength\labelsep{0pt} % align all notes with table
    \scriptsize
    \item \textit{N}\textsubscript{1}, number of individuals genotyped; \textit{N}\textsubscript{2}, number of individuals genotyped with missing data and duplicates removed; \textit{H}\textsubscript{o}, observed heterozygosity; \textit{H}\textsubscript{e}, expected heterozygosity; \textit{F}\textsubscript{is}, inbreeding coefficient. 
    \item \textsuperscript{a}Marine Conservation Zone, \textsuperscript{b}Ramsar site, \textsuperscript{c}Special Area of Conservation, \textsuperscript{d}Marine reserve.
  \end{tablenotes}
\end{threeparttable}
\end{table}
\end{landscape}
% \restoregeometry
\clearpage
% }
%% end of table


%% Figure: Lobster sampling site map
\begin{figure}[h!]
\centering
\includegraphics[width=0.95\textwidth]{Chapter_figures/chapter6/Sampling_Coverage_Lobster.png}
\caption[European lobster sampling sites]
{European lobster sampling sites.}
\label{fig:lobster_map}
\end{figure}
%% end of figure

%% Figure: Fluidigm EP1
\begin{figure}[h!]
\centering
\includegraphics[width=0.65\textwidth]{Chapter_figures/chapter6/Fluidigm_photo.jpg}
\caption[Fluidigm EP1 system]
{Fluidigm EP1 system.}
\label{fig:fluidigm}
\end{figure}
%% end of figure

\subsubsection{6.2.3 Quality control and filtering}
Individuals and SNP loci with more than 20 \% missing genotypes were removed from the dataset using the \texttt{missingno} function from poppr v2.8.0 \citep{Kamvar2014}.  Due to concerns over double-sampling, duplicated genotypes were identified using the \texttt{mlg} and \texttt{mlg.id} functions from poppr and were removed using custom R code.  Deviations from Hardy-Weinberg equilibrium (HWE) were tested using the \texttt{hw.test} function from pegas v0.11 \citep{Paradis2010}. The exact test based on Monte Carlo permutations of alleles was performed using 1,000 replicates.  The false discovery rate (FDR), computed using the \texttt{p.adjust} function in R, was used to adjust for multiple comparisons; loci were considered to be out of HWE if they significantly (p $<$ 0.05) deviated in more than 50 \% of populations.

\subsubsection{6.2.4 Genetic diversity}
Observed heterozygosity (\textit{H}\textsubscript{o}), expected heterozygosity (\textit{H}\textsubscript{e}) and the inbreeding coefficient (\textit{F}\textsubscript{is}) were calculated using the \texttt{divBasic} function from diveRsity v1.9.90 \citep{Keenan2013}.  Significance of \textit{F}\textsubscript{is} was assessed by calculating bias corrected 95 \% confidence intervals (1,000 bootstrap replicates) and testing whether values were significantly different from zero.  

\subsubsection{6.2.5 Detecting outlier SNPs}
Outlier SNPs potentially under divergent selection were identified using three differentiation-based methods.  Firstly, BayeScan v2.1 \citep{Foll2008} was implemented using default parameters and a prior model (pr\_odds) of 10,000, which sets the neutral model as being 10,000 times more likely than the model of selection to minimise the risk of false positives (Lotterhos and Whitlock 2014).  Secondly, PCAdapt v4.0.3 \citep{Luu2017} was run using three principal components (\textit{K}=3).  PCAdapt uses PCA to detect loci under selection and assumes that markers excessively related to population structure are candidates for local adaptation.  Lastly, the infinite island model in Arlequin v3.5.2.2 \citep{Excoffier2010} was run using 100,000 simulations and 1,000 demes.  This method integrates heterozygosity and simulates a distribution of \textit{F}\textsubscript{st} for neutrally distributed markers.  For all methods, a false discovery rate of 0.05 was used to identify outliers.  Outlier tests were conducted on all filtered SNPs and, using the results of these tests, the SNPs were divided into three datasets: (i) all SNPs, (ii) putatively neutral SNPs, and (iii) outlier SNPs putatively under divergent selection (SNPs putatively under balancing selection were removed).  A SNP was considered as putatively under divergent selection if all three methods identified it as an outlier.  

\subsubsection{6.2.6 Population structure}
Genetic differentiation between sampling sites was analysed by calculating pairwise values of \textit{F}\textsubscript{st} \citep{Weir1984} and \textit{D} \citep{Jost2008} using the \texttt{diffCalc} function from diveRsity.  Heatmaps of each statistic were visualised in R and significance was assessed using the same method previously described for \textit{F}\textsubscript{is}.

Population genetic structure was explored using two different approaches.  Firstly, a DAPC was run using the \texttt{dapc} function from the R package adegenet v2.1.1 \citep{Jombart2011}.  Cross validation using the \texttt{xvalDapc} function from adegenet was used to choose the optimal number of PCs to retain.  Secondly, STRUCTURE v2.3.4 \citep{Pritchard2000}, a Bayesian clustering algorithm, was run in parallel using the program StrAuto v1.0 \citep{Chhatre2017}.  STRUCTURE was executed using the admixture model, with 10\textsuperscript{5} MCMC repetitions and a burn-in of 10\textsuperscript{5}.  The locprior option was selected, with sampling locations used as \textit{a priori} information; all other parameters were set to default values.  The maximum number of populations (\textit{K}) assumed was 20 and ten independent replicates per \textit{K} (1-20) were computed.  To statistically assess different values of \textit{K}, the mean value of L(\textit{K}) \citep{Pritchard2000} and the delta \textit{K} \citep{Evanno2005} statistics were examined in the R package pophelper v2.2.5.1 \citep{Francis2017}.  Replicates runs were aligned and merged with CLUMPP v1.1.2 \citep{Jakobsson2007} using a wrapper script in pophelper and R was used to visualise the results. 


\subsubsection{6.2.7 Isolation-by-distance}
As genetic connectivity between populations is driven by neutral processes, only putatively neutral SNPs were used for inferring dispersal and connectivity.  For these analyses, temporal samples from both the Île de Ré (Idr16 and Idr17) and Sardinia (Sar13 and Sar17) were combined because of their genetic similarity; in addition, Laz and Tar were combined into one Lazio sample due to their spatial proximity and genetic similarity (see Results).

Two approaches were implemented that explore whether spatial distribution explains any of the observed genetic variation between sampling locations: (i) traditional Mantel tests and (ii) redundancy analysis (RDA).  Mantel tests assume that the relationship between two dissimilarity matrices (D1:D2) is linear and that small D1 and large D1 values correspond to small D2 and large D2 values, respectively.  However, it has been suggested that these patterns rarely exist unless spatial correlation extends over the whole study area \citep{Legendre2015}.  RDA is a combination of multiple linear regression and PCA that examines how much of the variation in a matrix of independent variables (i.e. spatial distribution, temperature, etc.) explains the variation in a matrix of dependant variables (i.e. allele frequencies).  RDA assumes, like Mantel tests, that the expected relationship between the dependant and independent variables is linear.  The main advantage of RDA over Mantel tests is that it operates on the raw data (i.e. the allele frequencies), as opposed to genetic distance matrices; this enables one to directly test how the spatial distribution of genetic variation within a species is influenced by effective dispersal \citep{Meirmans2015}.  In addition, RDA has been shown to be provide more power for assessing the influence of spatial correlation than Mantel tests \citep{Legendre2015}. It is also worth noting that the \textit{r}\textsuperscript{2} statistics of the Mantel test and from RDA are not comparable; in the Mantel test, \textit{r}\textsuperscript{2} measures the proportion of the dissimilarity variances in D1 that are explained by geographic distances, whereas RDA \textit{r}\textsuperscript{2} measures how much of the variance in the dependant variable (allele frequencies) is explained by geography \citep{Legendre2015}.

To create geographic distance matrices for the Mantel tests, least-cost distances (km) between sampling sites were calculated using the \texttt{lc.dist} function from the R package marmap v1.0 \citep{Pante2013}.  Matrices of pairwise \textit{F}\textsubscript{st} and \textit{D} were calculated using the R package mmod v1.3.3 \citep{Winter2012} and negative values were converted to zeros.  Mantel tests were performed using the \texttt{mantel.rtest} function from the R package ade4 v1.7.11 \citep{Dray2007} and significance was assessed using 10,000 permutations.  For RDA, geographic coordinates (lat and lon in decimal degrees) of sampling sites were transformed into Cartesian coordinates using the \texttt{geoXY} function from the R package SoDA v1.0.6 \citep{Chambers2013}.  Then, Euclidean distances were calculated from the Cartesian coordinates using the \texttt{dist} function and distance-based Moran's eigenvector maps (dbMEMs) were computed using the \texttt{dbmem} function from the R package adespatial v0.2.0 \citep{Dray2018}.  The dbMEMs are a series of variables that summarise the spatial structure among the sampling sites, thereby representing a spectral decomposition of the spatial relationships between study sites \citep{Borcard2002}.  RDA was performed on the population allele frequencies (dependant variables) and the dbMEMs (independent variables) using the \texttt{rda} function from the R package vegan v2.5.2 \citep{Oksanen2018}. Significance was assessed by analysis of variance (ANOVA) using the \texttt{anova.cca} function (10,000 permutations) and only significant (\textit{p}$<$0.05) dbMEMs were included in subsequent analyses.


\subsubsection{6.2.8 Simulations of larval dispersal}
Oceanographic drift modelling was implemented to estimate larval dispersal and predict connectivity between study sites for a comparison with the observed patterns of genetic connectivity.  The hydrodynamic model used is described in \citet{Lien2014} and the particle-tracking algorithms applied are detailed in \citet{Vikebo2010}.  The ocean current model had a horizontal resolution of 4 km and simulations were performed from 2007-2016, except for 2012-2014 for which data were unavailable; however, patterns of connectivity were consistent across the years, so the results were deemed representative (Mats Huserbraten, \textit{pers. comm.}).  Due to the limited spatial extent of the ocean model, only study sites from the English Channel to the North Sea and Skagerrak were included in the simulations; as it represents a partially closed system, Oosterschelde was also outside the scope of the ocean model so the closest feasible point in the North Sea was used instead.  Furthermore, Gullmarfjord, Kavra and Lysekil (western Sweden) were located in the same grid-cell and were therefore merged in the simulations.

An individual-based model (IBM) was designed for European lobster larvae that incorporated aspects of their known life history.  Particle (larvae) release followed a normal distribution with peak release in mid-June; approximately 6,000 particles were released from each site across all years, which translated to a total of 1 million particles released over 200 days.  Particles drifted at a fixed depth between 1-20 m for 12-28 days depending on the median temperature encountered during the drift trajectory \citep{Schmalenbach2010}.  The PLD decreased with increasing temperature and increased with decreasing temperature; however, there is a critical value in which temperature affects survival in hatchery-reared lobsters, resulting in mortality (Schmalenbach and Franke 2010).  Therefore, when larvae encountered median temperatures of $<$14\textsuperscript{o}C and $>$22\textsuperscript{o}C (i.e. larvae experienced temperatures less than $<$14\textsuperscript{o}C or $>$22\textsuperscript{o}C for more than half of their drift time), larvae were considered dead because the temperature was assumed to be outside of the thermal niche required for development.

Dispersal trajectories for all sites and years were plotted with and without the use of the IBM to compare dispersal patterns including and excluding biological parameters (i.e. temperature-dependency and mortality).  Connectivity between sites was assessed by creating a connectivity matrix, whereby one unit in the matrix represented one day spent by a source particle within a 40 km radius from a sink. 


\subsubsection{6.2.9 Individual assignment}
The accuracy of assigning individuals back to their basin of origin (i.e. Atlantic Ocean or Mediterranean Sea) and to their sampling location/region of origin was assessed using the R package assignPOP v1.1.4 \citep{Chen2018}.  assignPOP uses a cross validation procedure followed by PCA to evaluate assignment accuracy and membership probabilities.  Firstly, the dataset is partitioned into training (baseline) and test (holdout) datasets using a resampling cross validation procedure, with the user specifying the number or proportion of individuals from each source `population' (i.e. Atlantic or Mediterranean in the basin analysis) to be used in the training dataset.  This approach of creating randomly selected, independent training and test datasets avoids introducing high-grading bias \citep{Anderson2010}.  Secondly, the features of the training datasets (i.e. the genotypes) are reduced in dimensionality using PCA, the output of which are used to build predictive models from user-chosen classification machine-learning functions \citep{Chen2018}.  Finally, these models are then used to estimate membership probabilities of test individuals and assign individuals to a source population, while also evaluating the baseline data and conducting assignment tests on individuals for which the origin is unknown \citep{Chen2018}.

All filtered SNPs were used in the assignment tests because the inclusion of both neutral and outlier loci can increase the power of assignment tests \citep{Gagnaire2015}.  For assigning individuals to their basin of origin, before dividing the dataset into baseline and test datasets, two individuals per sampling location (76 individuals in total) were randomly selected in R to compose a file representing `unknown' individuals, whereby the basin \replaced{of}{was} origin was considered to be unknown.  Due to the potential bias of unequal sample size in assignment studies \citep{Wang2016}, 250 individuals from the Atlantic basin were randomly selected to compose this source population (with 154 individuals composing the Mediterranean basin). 

A Monte-Carlo cross validation procedure was used to group individuals into baseline and test datasets using the function \texttt{assign.MC} from assignPOP.  Resampling was repeated 30 times for each combination of training individuals and loci.  The proportion of individuals from each source population randomly allocated to the baseline dataset was set to 0.5, 0.7 and 0.9.  Lastly, the support vector machine (svm) and the linear discriminant analysis (lda) classification functions were used to build predictive models.  After building predictive models based on the baseline dataset, the origin of the unknown individuals were assessed, further testing the accuracy of the assignment.

  
\subsection{6.3 Results}

\subsubsection{6.3.1 SNP genotyping and quality control}
Five SNP assays (H\_gam\_25580, H\_gam\_32362, H\_gam\_41521, H\_gam\_53889, H\_gam\_65376) did not work consistently on the Fluidigm EP1 system, possibly due to inadequate assay design, poor STA amplification, or ascertainment bias.  One locus (H\_gam\_22365) contained 28.3 \% missing data and was therefore removed from the dataset.  In addition, eight individuals (Ale04, Ale06, Ale08, Ale13, Ale15, Ale16, Ale19 and The24) were removed because of missing data ranging from 42.9-54.9 \%, which was likely due to very poor DNA quality, evidenced by gel electrophoresis, because repeats also produced similar levels of missing data. 

In total, 1,223 unique multi-locus genotypes from 1,242 individuals were obtained (Table \ref{tab:snp_sites}). Although duplicates were apparent between some Laz and Tar samples (western Italy), most duplicates were mainly individuals from the same sampling site (Table \ref{tab:dups}).  These results could have arisen from double-sampling during sample collection, contamination of DNA samples, or the inclusion of closely related siblings.  In any case, because the exact cause could not be determined, only one individual from each duplicate was retained.  One locus (H\_gam\_21197) significantly deviated from HWE and was removed from the dataset.  In addition, three loci (H\_gam\_8953, H\_gam\_21880, H\_gam\_22323) exhibited an unexpectedly high proportion of observed heterozygosity (0.72, 0.61 and 0.67, respectively); these loci were removed because they could be paralogous loci as true variants are often considered to have a frequency of 0.50 heterozygous genotypes \citep{Dufresne2014}.  The final filtered dataset contained 1,223 individual lobsters from 36 sampling sites (38 including temporal samples) and 86 biallelic SNP loci. 

\added{Out of the 96 SNPs selected in Chapter 5, all ten SNP loci that were omitted after quality control in this study originated from the 75 SNPs selected from 4,377 SNP dataset (Fig. \ref{fig:RADseq_gstperSNP}).  This meant that only 65 of the 75 SNPs were used in this study, while all 21 SNPs selected from the 7,022 dataset (Fig. \ref{fig:RADseq_gstperSNP}) were retained as they met the quality control thresholds.}


%%% Table duplicates
\begin{table}[h!]
\begin{small}
\caption[European lobster individual IDs with identical genotypes]
{European lobster individual IDs with identical genotypes.}
\label{tab:dups}
\begin{tabular}{llll}
\hline \\[-1.0em]
\textbf{Sampling site} & \textbf{Individual 1} & \textbf{Individual 2} & \textbf{Individual 3}  \\
\hline \\[-1.0em]
Cromer & Cro08 & Cro15 & -- \\
Eyemouth & Eye01 & Eye17 & -- \\
         & Eye02 & Eye04 & -- \\
         & Eye05 & Eye06 & Eye23 \\
         & Eye07 & Eye24 & -- \\
         & Eye14 & Eye31 & -- \\
         & Eye15 & Eye16 & -- \\
         & Eye20 & Eye36 & -- \\
         & Eye25 & Eye29 & -- \\
Isle of Man & Iom02 & Iom22 & -- \\
Gullmarfjord & Gul86 & Gul101 & -- \\
Helgoland & Hel07 & Hel09 & -- \\
Lazio / Tarquinia & Laz01 & Tar01 & --  \\
  & Laz02 & Tar02 & --  \\
  & Laz03 & Tar03 & -- \\
  & Laz04 & Tar04 & --  \\
Llyn Peninsula & Lyn04 & Lyn15 & Lyn34 \\
\hline
\end{tabular}
\end{small}
\end{table}
%% end of table


\subsubsection{6.3.2 Genetic diversity}
The \textit{H}\textsubscript{o} and \textit{H}\textsubscript{e} ranged from 0.31-0.42 (Oos-Tar) and 0.29-0.39 (Sar13-Idr17 and Vig), respectively (Table \ref{tab:snp_sites}).  Overall, lower diversity measures (\textit{H}\textsubscript{e}) were found in Oosterschelde, Helgoland, Scandinavia and the Mediterranean samples, while the highest diversity measures were found in northwest Spain (Vig), the Bay of Biscay (Idr), and a few western and southerly sites from Britain and Ireland.  The inbreeding coefficient (\textit{F}\textsubscript{is}) ranged from -0.292 (Tar) to 0.040 (Ale), and ten \textit{F}\textsubscript{is} measures were significantly different from zero. Laz, Sar13 and Tar had the lowest significant \textit{F}\textsubscript{is} measures (-0.234, -0.292 and -0.092, respectively), which were also the sites with very low sample sizes (Table \ref{tab:snp_sites}).  The lowest significant measures excluding these sites were -0.072 (Gul) and -0.068 (Lyn).  

\subsubsection{6.3.3 Outlier SNP detection}
Bayescan and Arlequin detected 17 SNPs putatively under divergent selection, while PCAdapt detected 22 SNPs putatively under divergent selection.  All three methods identified the same 15 SNPs as outliers potentially under divergent selection; the remaining SNPs were considered neutral.  In addition, outlier tests were also performed on the original RADseq dataset composed of 7,022 SNPs (Chapter 5); these analyses identified 59-124 outlier SNPs depending on the outlier test used, of which 13 out of the 15 outliers identified in the SNP dataset from this study were also identified in at one least or more outlier tests in the RADseq dataset.

\subsubsection{6.3.4 Population structure}
Global values of \textit{F}\textsubscript{st} and \textit{D} were 0.060 and 0.014, respectively, and both pairwise differentiation statistics showed comparable pairwise patterns between sampling sites (Fig. \ref{fig:lobster_heatmaps}).  Values of \textit{F}\textsubscript{st} ranged from zero to 0.314 (Oos-Sar13) and from zero to 0.041 (Oos-Sar13) for \textit{D}.  The highest values for both statistics were between the Mediterranean sites and the Atlantic sites, of which many of these values were significantly different from zero.  The lowest values tended to be between sites originating from Britain, Ireland, France and the Channel Islands. However, sites situated spatially close together in other regions also had low pairwise comparisons (i.e. Laz, Sar and Tar in the mid-Mediterranean; Ale, Sky, The and Tor in the eastern Mediterranean; and Flo, Gul, Kav, Lys and Sin in Skagerrak).

%% Figure: Lobster Fst and D heatmaps
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/heatmap_fst.pdf}
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/heatmap_d.pdf}
\caption[European lobster \textit{F}\textsubscript{st} and \textit{D} heatmaps]
{European lobster heatmaps of \textit{F}\textsubscript{st} (top) and \textit{D} (bottom). Asterisks represent values significantly different from zero.}
\label{fig:lobster_heatmaps}
\end{figure}
\clearpage
}
%% end of figure

The DAPC using all 86 SNPs showed distinct separation of lobsters from the Atlantic and the Mediterranean (Fig. \ref{fig:lobster_DAPC}).  There was also evidence for structure within the Mediterranean, partitioned between the mid-Mediterranean (Sar, Laz and Tar) and the eastern Mediterranean (Aegean Sea -- Ale, Sky, The and Tor).  Within the Atlantic cluster, there was a pronounced genetic cline starting from the most southerly site in the northeast Atlantic (Vig) to the most northerly sites in Skagerrak (Kav, Flo, Gul, Lys and Sin).  In total, the first and second axes explained 69.8 \% of the variation in the dataset. The 15 outlier SNPs showed very similar patterns to those described using all SNPs (Fig. \ref{fig:lobster_DAPC}); however, the first and second axes explained considerably more of the variation in the dataset (91.3 \%).  In addition, compared to the DAPC using all SNPs, differentiation between the middle and eastern Mediterranean was weaker.

%% DAPC: all SNPs and outlier SNPs
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=0.9\textwidth]{Chapter_figures/chapter6/lobster_dapc_1223ind_86snps_axis1vs2.pdf}
\includegraphics[width=0.9\textwidth]{Chapter_figures/chapter6/lobster_dapc_1223ind_15snps_axis1vs2.pdf}
\caption[DAPC using all 86 SNPs and 15 outlier SNPs]
{Discriminant analysis of principal components using all 86 SNPs (top) and 15 outlier SNPs (bottom). For each DAPC, points represent individuals and colours denote the sampling region of origin.}
\label{fig:lobster_DAPC}
\end{figure}
\clearpage
}
%% end of figure

%% DAPC: neutral SNPs
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=0.9\textwidth]{Chapter_figures/chapter6/lobster_dapc_1223ind_71snps_axis1vs2.pdf}
\includegraphics[width=0.9\textwidth]{Chapter_figures/chapter6/lobster_dapc_1223ind_71snps_axis2vs3.pdf}
\caption[DAPC using 71 neutral SNPs]
{Discriminant analysis of principal components using 71 neutral SNPs: axis 1 and 2 (top) and axis 2 and 3 (bottom) are shown. For each DAPC, points represent individuals and colours denote the sampling region of origin.}
\label{fig:lobster_DAPC_neutral}
\end{figure}
\clearpage
}
%% end of figure

The 71 neutral SNPs, in contrast, showed the same Atlantic-Mediterranean divide but the separation was weaker, particularly between mid-Mediterranean sites and Atlantic sites (Fig. \ref{fig:lobster_DAPC_neutral}). There was also stronger separation between the middle and eastern Mediterranean; however, overall the first three axes explained a lower amount of variation in the dataset.  Furthermore, in comparison to the other SNP datasets, the Atlantic cluster showed no obvious clinal pattern using the neutral SNPs, but weak separation of Oosterschelde lobsters from the main Atlantic cluster was apparent after exploring and visualising the first three axes (Fig. \ref{fig:lobster_DAPC_neutral}).

\clearpage
STRUCTURE analysis generally supported the DAPC results for all SNP datasets (all SNPs, neutral SNPs and outlier SNPs); however, a genetic cline was evident from the eastern Mediterranean to Skagerrak in all SNP datasets (Fig. \ref{fig:lobster_str_maps}).  After exploring different levels of \textit{K} and examining the statistics (Appendix A7), the most likely \textit{K} for all SNP datasets was \textit{K}=3, which was generally consistent with the clusters found from the DAPCs.  The membership proportions were estimated for each individual (Fig. \ref{fig:lobster_str_ind_out}, \ref{fig:lobster_str_ind_neu}) and the mean was calculated for each sampling site to generate an average membership proportion to each \textit{K} cluster which was visualised as pie charts on a map (Fig. \ref{fig:lobster_str_maps}).  In the neutral SNP dataset, Oosterschelde was not apparent in \textit{K}3; however, analysis of \textit{K}5 showed that one cluster was predominantly found in Oosterschelde, which was also consistent with the results from the neutral DAPC. 

%% STRUCTURE maps
\begin{figure}[h!]
\centering
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/lobster_allSNPs_strk3_map.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/lobster_15outlier_strk3_map.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/lobster_71neutral_strk3_map.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/lobster_71neutral_strk5_map.png}
\caption[STRUCTURE results averaged per site]
{STRUCTURE results per individual using all 86 SNPs (top-left, \textit{K}3), 15 outlier SNPs (top-right, \textit{K}3), 71 neutral SNPs (bottom-left, \textit{K}3; bottom-right, \textit{K}5). The mean was calculated for each sampling site to generate an average membership proportion to each \textit{K} cluster which was visualised as pie charts on a map.}
\label{fig:lobster_str_maps}
\end{figure}
%% end of figure

%% STRUCTURE individuals: ALL AND OUTLIER
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/lobster_allSNPs_strk3.pdf}
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/lobster_15outlier_strk3.pdf}
\caption[STRUCTURE results per individual: all SNPs and outlier SNPs]
{STRUCTURE results per individual using all 86 SNPs (top, \textit{K}3) and 15 outlier SNPs (bottom, \textit{K}3). Each bar represents an individual and colours denote membership proportions to each cluster.}
\label{fig:lobster_str_ind_out}
\end{figure}
\clearpage
}
%% end of figure

%% STRUCTURE individuals: NEUTRAL K3 AND K5
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/lobster_71neutral_strk3.pdf}
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/lobster_71neutral_strk5.pdf}
\caption[STRUCTURE results per individual: neutral SNPs]
{STRUCTURE results per individual using 71 neutral SNPs (top, \textit{K}3; bottom, \textit{K}5). Each bar represents an individual and colours denote membership proportions to each cluster.}
\label{fig:lobster_str_ind_neu}
\end{figure}
\clearpage
}
%% end of figure


\clearpage
\added{To further visualise differentiation between the three groups found by DAPC and STRUCTURE, the population allele frequency for one allele was visualised for each SNP identified as an outlier in this study (Fig. \ref{fig:lobster_allele_freq}). Most of these SNPs showed large differences in frequency between the Atlantic, the middle Mediterranean and/or the eastern Mediterranean. For instance, SNPs \textit{22740}, \textit{33066}, \textit{51507}, \textit{53052}, \textit{53263}, \textit{65064} and \textit{65576} showed noticeably different frequencies between sites from the middle Mediterranean compared to sites from both the Atlantic and the eastern Mediterranean. In addition, SNPs \textit{42395} and \textit{53935} were completely fixed for the T allele in the eastern Mediterranean, while both SNPs were fixed in most, but not all, of the sampling sites in the middle Mediterranean. In the Atlantic, SNP \textit{58053} was fixed for the A allele in several sites (typically sites from Scandinavia), while sites from both the middle and the eastern Mediterranean had comparably very low frequencies of the A allele.}     

%% Figure: Lobster allele freq. outlier SNPs
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/allele_freq.pdf}
\caption[Population allele frequency of one allele for each outlier SNP]
{Population allele frequency of one allele for each outlier SNP identified in this study. For each SNP, the sampling sites (x-axis) are arranged in the same order as the STRUCTURE results (Fig. \ref{fig:lobster_str_ind_out}, \ref{fig:lobster_str_ind_neu}). Colours denote the sampling site region of origin: the Atlantic (blue), the middle Mediterranean (orange) and the eastern Mediterranean (red).}
\label{fig:lobster_allele_freq}
\end{figure}
%% end of figure


\clearpage
\subsubsection{6.3.5 Isolation-by-distance}
Initial analysis with Mantel tests showed concordant patterns using \textit{F}\textsubscript{st} and \textit{D}; thus, only results for \textit{F}\textsubscript{st} are described.  Using all sites, there was a strong positive correlation between geographic distance and \textit{F}\textsubscript{st} (Fig. \ref{fig:lobster_IBD}a; \textit{r}\textsuperscript{2}=0.81, \textit{p}$<$0.001).  When the Mediterranean samples were removed, this correlation was much weaker, but still significant (Fig. \ref{fig:lobster_IBD}b; \textit{r}\textsuperscript{2}=0.17, \textit{p}=0.041).  However, the removal of Oosterschelde lobsters vastly increased the strength and significance of the correlation (Fig. \ref{fig:lobster_IBD}c, \textit{r}\textsuperscript{2}=0.49, \textit{p}$<$0.001).  Analysis with only the Mediterranean samples also produced a strong correlation, but this was not significant (Fig. \ref{fig:lobster_IBD}d; \textit{r}\textsuperscript{2}=0.88, \textit{p}=0.062).

%% IBD
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/IBD_composite_fig.pdf}
\caption[European lobster isolation-by-distance analyses]
{European lobster isolation-by-distance analyses: pairwise comparisons of geographic distances (km) and \textit{F}\textsubscript{st} between sampling sites were plotted using all sites (top-left), Atlantic sites only (top-right), Atlantic sites only and excluding Oosterschelde (bottom-left), and Mediterranean sites only (bottom-right). A linear regression line (blue line) and the standard error (dark grey shaded area) was added to each plot.  Asterisks denote significance levels: *<0.05, **<0.01, ***<0.001.}
\label{fig:lobster_IBD}
\end{figure}
%% end of figure

The RDA was globally significant (\textit{R}\textsuperscript{2}=0.43, \textit{p}$<$0.001) and the first two axes of the RDA accounted for 52.1 \% and 29.1 \%, respectively.  Of the eight dbMEMs constructed from the Euclidean distances, only vectors 1-2 were found to be significant for explaining variation in the allele frequencies.  RDA was re-run with these two dbMEMs and the results indicated that these two spatial variables explained 28.0 \% of the variation in the allele frequencies (Fig. \ref{fig:RDA}).

%% RDA
\begin{figure}[h!]
\centering
\includegraphics[width=0.95\textwidth]{Chapter_figures/chapter6/rda_IBD.pdf}
\caption[European lobster redundancy analysis]
{European lobster redundancy analysis. Each point represents a sampling site.  Blue lines show the distance-based Moran's eigenvector maps (dbMEMs) that significantly explained variation in the allele frequencies.}
\label{fig:RDA}
\end{figure}
%% end of figure

\clearpage
\subsubsection{6.3.6 Larval dispersal simulations}
Simulations of larval dispersal with and without the incorporation of biological parameters (i.e temperature-dependency and mortality) were vastly contrasting (Fig. \ref{fig:lobster_dispersal}).  Without any biological parameters considered, larval dispersal across much of the study area was extensive.  For example, many particles from Sula Sgeir (northwest Scotland) and Skagerrak were able to travel to the tip of northern Norway in one journey.  Moreover, there was considerable mixing in the Celtic and Irish Seas, within the North Sea, and between locations along the western coast of Ireland; however, less mixing was apparent between the western and eastern English Channel.  In contrast, when biological parameters were considered, larval dispersal across the study area was notably reduced.  In particular, there was very limited dispersal out of northern Scotland and the furthest drifting particles from Skagerrak were only able to travel to southwest Norway.  Moreover, dispersal was more limited from all study sites across the seas of Britain and Ireland. 

%% Larval dispersal trajectories
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=0.7\textwidth]{Chapter_figures/chapter6/Tom_traject.png}
\includegraphics[width=0.7\textwidth]{Chapter_figures/chapter6/Tom_traject_w_mortality.png}
\caption[Larval dispersal trajectories]
{Larval dispersal trajectories with the exclusion of biological parameters (top) and the inclusion of temperature-dependent PLD and mortality (bottom).}
\label{fig:lobster_dispersal}
\end{figure}
\clearpage
}
%% end of figure

%% Larval dispersal connectivity
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=0.85\textwidth]{Chapter_figures/chapter6/connectivity_heatmap_non_bio.pdf}
\includegraphics[width=0.85\textwidth]{Chapter_figures/chapter6/connectivity_heatmap.pdf}
\caption[Larval dispersal connectivity matrices]
{Larval dispersal connectivity matrices with the exclusion of biological parameters (top) and the inclusion of temperature-dependent PLD and mortality (bottom).}
\label{fig:lobster_connectivity}
\end{figure}
\clearpage
}
%% end of figure

The connectivity matrices generally supported results from the dispersal trajectories, whereby connectivity between sites was much higher with the exclusion of biological (non-bio) parameters compared to the inclusion of biological (bio) parameters (Fig. \ref{fig:lobster_connectivity}).  In the non-bio matrix, asymmetrical connectivity was apparent from western North Sea sites (Brd, Cro, Eye, and Oos) to eastern North Sea (Hel) and Skagerrak sites (Flo, Sin and Kav); this was not the case for the bio matrix in which no asymmetrical connectivity was found across the same spatial scales.  Moreover, in both connectivity matrices, Skagerrak sites did not act as a source for other study sites other than for those within Skagerrak, except for a few potential recruits into Sbs and Hel from Flo (Fig. \ref{fig:lobster_connectivity}).  Both matrices indicated that the strongest source-sink relationships were predominantly between the same sites, or between neighbouring sites situated spatially very closer to each other.  However, in the bio matrix, there was no intra or inter-site connectivity in three study sites (Ork, She and Sul), all of which are located in colder waters around the north of Scotland. 

\clearpage
\subsubsection{6.3.7 Individual assignment}
Assigning individuals to their basin of origin (Atlantic or Mediterranean) using the baseline data was extremely accurate, ranging from 98-100 \% depending on the proportion of individuals used in the training dataset and the number of loci used for the assignment tests (Fig. \ref{fig:Assignment}). When all 86 SNPs were used, the model successfully predicted the basin of origin of the test individuals at 100 \% accuracy, regardless of the proportion of individuals used in the training set.  This was then tested on the unknown dataset and the SVM and LDA functions both correctly predicted the basin of origin for all 76 individuals.  In comparison, assigning individuals back to their location of origin was highly inaccurate, with an overall assignment accuracy of $\sim$1 \% (Fig. \ref{fig:Assignment}).  However, assigning individuals to one of three regional groups (Skagerrak, North Sea, and remaining Atlantic sites) was much more accurate, with an overall assignment accuracy of $\sim$78 \% (Fig. \ref{fig:Assignment}).  The assignment tests suggested that assigning the remaining Atlantic sites back to the Atlantic group was approximately 94 \% accurate using all 86 SNPs; assigning Scandinavian (i.e. Skagerrak) sites back to the Skagerrak group was approximately 68 \% accurate using all 86 SNPs.  In contrast, the assignment tests showed that North Sea sites could not be assigned back to a North Sea group - they were either assigned to the Atlantic group (78 \%) or the Skagerrak group (22 \%).  

%% Assignment
\afterpage{
\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/Basin_assignment_accuracy.pdf}
\includegraphics[width=\textwidth]{Chapter_figures/chapter6/Region_assignment_accuracy.pdf}
\includegraphics[width=0.6\textwidth]{Chapter_figures/chapter6/Location_assignment_accuracy.pdf}
\caption[European lobster individual assignment]
{European lobster individual assignment: basin-wide analysis (top), regional analysis (middle), and sampling location analysis (bottom).}
\label{fig:Assignment}
\end{figure}
\clearpage
}
% end of figure

\subsection{6.4 Discussion}
This study used a small panel of informative SNPs to investigate population genetic patterns of \textit{H. gammarus} at sites sampled across most of its contemporary geographical range. Although this SNP panel composes loci that were chosen to capture genetic differentiation at different spatial scales, the overall panel includes loci across the spectrum of the differentiation statistics (Fig. \ref{fig:RADseq_gstperSNP}), thereby permitting the analysis of both loci potentially under high selection or drift and loci which are generally homogenous across certain spatial scales.  Sampling density reflected that of lobster abundance, being most concentrated around Britain and Ireland, where approximately 75 \% of the species total catch resides \citep{FAO2018}.  The patterns of population genetic structure detected in this study supports the general assertion of previous studies that regional lobster stocks are relatively well-connected.  However, this study provided much higher resolution of population structure than has been previously achieved, which has enabled the detection of fine-scale differences between sampling sites, particularly in the northeast Atlantic.  Crucially, the higher power offered by SNP markers, combined with biophysical modelling, allowed the potential drivers of these patterns to be explored with much greater precision.  Genetic and modelling analysis suggested that patterns of connectivity are driven, at least in part, by spatial distances between sites (IBD) which implies that \textit{H. gammarus} larval dispersal likely accords with a stepping-stone model, a finding consistent with a recent study that used 14 microsatellite markers to explore population structure \citep{Ellis2017}.  Individual assignment was shown to be highly accurate at a basin-wide scale and reasonably accurate at a regional scale, although marker power and/or underlying differentiation was not sufficient to accurately assign individuals to their sampling location of origin. The findings from this study should be valuable to fisheries management, particularly for providing estimates of stock connectivity and delineating management units \citep{Reiss2009}, and for defining the spatial bounds of ecologically responsible hatchery stock enhancement programmes. 

\subsubsection{6.4.1 Atlantic-Mediterranean transition}
Population structure analyses in this study found three distinct genetic groups, organised into sites from the northeast Atlantic, the middle Mediterranean, and the Aegean Sea (the eastern Mediterranean).  Divergence was particularly strong between sites originating from the northeast Atlantic and the Mediterranean, a pattern that was detected in two previous studies that used mtDNA RFLPs (Triantafyllidis et al. 2005) and allozymes (Jorstad et al. 2005).  A deep partition between the Atlantic and the Mediterranean basins has been found in previous studies for a diverse array of marine organisms including the European spiny lobster (mtDNA and microsatellites, Palero et al. 2008; \citeyear{Palero2011}), other decapods such as crabs (mtDNA, \citeauthor{Roman2004} 2004; \citeauthor{Garcia-Merchan2012} 2012) and shrimp (mtDNA, \citeauthor{Reuschel2010} 2010), molluscs (microsatellites, \citeauthor{Perez-Losada2002} 2002; mtDNA and allozymes, \citeauthor{Sa-Pinto2012} 2012), arrow worms (mtDNA and microsatellites, \citeauthor{Peijnenburg2006} 2006), seahorses (SNPs, \citeauthor{Riquet2017} 2017), demersal fish (mtDNA and allozymes, \citeauthor{Bargelloni2003} 2003, \citeyear{Bargelloni2005}; mtDNA and microsatellites, \citeauthor{Pita2010} 2010), and deep-sea fish (mtDNA, \citeauthor{Charrier2006} 2006).

The majority of these studies ascribe this partition to restricted gene flow between the Atlantic and Mediterranean basins, possibly due to IBD and/or an oceanographic barrier to connectivity.  For example, \citet{Castilho2017} found that genetic patterns in the intertidal peacock blenny (\textit{Salaria pavo}) were likely explained by a combination of IBD and asymmetrical gene flow from the Mediterranean to the Atlantic.  Conversely, a break in connectivity between the Atlantic and Mediterranean basins has been suggested to occur at the Almeria-Oran front \citep[e.g.][]{Patarnello2007} and/or the Strait of Gibraltar \citep[e.g.][]{Garcia-Merchan2012}.  For instance, \citet{Reuschel2010} reported a distinct phylogeographic break in the pan-European littoral prawn (\textit{Palaemon elegans}) across the Atlantic-Mediterranean boundary, a finding the authors linked to reduced larval dispersal across the Almeria-Oran front, which is a front formed by the convergence of two distinct water masses and moderated by the currents of the Eastern Alboran Gyre \citep{Tintore1988}.  Aside from this oceanographic barrier, historical vicariant events potentially associated with the Strait of Gibraltar have also been suggested to shape genetic patterns of marine taxa across this boundary.  For example, during the last glacial maxima, fluctuations in sea levels (up to 120 m below present-day levels) periodically reduced the width and depth of the Gibraltar Strait \citep{Rohling1998}, potentially resulting in recurrent impediments to dispersal across this boundary \citep{Charrier2006}.  In addition, some studies have proposed a scenario of secondary contact across this transition zone, possibly due to the re-opening of the Gibraltar Strait some 5.33 Ma following the theorised Zanclean flood \citep{Perez-Losada2002} or due to vicariance during Pleistocene glaciations \citep{Taboada2016}, in which secondary introgression occurs between previously isolated and divergent allopatric populations \citep{Bierne2013}.  This has been supported by studies that have reported: (i) clinal changes in allele frequencies across the transition zone (located generally west or east of Gibraltar) in several species (e.g.  cuttlefish, \citeauthor{Perez-Losada2002} 2002; seagrass, \citeauthor{Alberto2008} 2008; seahorses, Riquot et al. 2017); and (ii) the discovery of two distinct Atlantic-Mediterranean clades, of which one clade co-occurs in both basins (e.g. swordfish, \citeauthor{Alvarado-Bremer2005} 2005; brittle stars, Taboada et al. 2016).

The results of this study suggests that a neutral pattern of IBD likely explains some of the structure in \textit{H. gammarus} between the Atlantic and Mediterranean basins, supported by DAPC, STRUCTURE and IBD analyses with neutral SNPs.  It is also possible that the Almeria-Oran front or the Strait of Gibraltar have contributed to this pattern, but a lack of sampling in the western Mediterranean means it is difficult to make inferences about the potential role of these putative barriers.  Moreover, although the genetic cline (Fig. \ref{fig:lobster_str_maps}) from the eastern Mediterranean to northwest Spain potentially supports a scenario of secondary contact, sites from the southern Atlantic and western Mediterranean are needed to fully explore this hypothesis.  Alternatively, analyses with outlier SNPs, which are putatively non-neutral, revealed that some of the basin-wide differentiation detected is explained by divergence at SNPs potentially under the influence of divergent selection.  This may suggest that lobsters from these two basins are potentially locally adapted to specific environmental conditions, of which sea temperature and salinity are possible selective factors as they have been shown to drive adaptive divergence among populations of numerous marine invertebrate species \citep{Sanford2011,Dalongeville2018}.  For example, a temperature gradient (-1\textsuperscript{o}C to 26\textsuperscript{o}C) exists across the northwest Atlantic distribution of \textit{H. americanus}, and a recent study found evidence for thermal adaptation in this species \citep{Benestan2016b}.  A similarly extensive thermal gradient exists across the range of \textit{H. gammarus} populations sampled in this study --from the Aegean Sea (23.5-26.4\textsuperscript{o}C in August) to Skagerrak (1.1-6.3\textsuperscript{o}C in March)-- so future research may investigate temperature as a potential driver of adaptive variation.  A recent study on seahorses suggested that a gradient of introgression between Atlantic and Mediterranean lagoons is likely driven by parallel outlier loci, possibly from adaptive introgression or because of a shared history of divergence retained at outlier loci against secondary gene flow \citep{Riquet2017}; based on the patterns at outlier SNPs in this study, this could also be a potential explanation for the Atlantic-Mediterranean partition in \textit{H. gammarus}. 


\subsubsection{6.4.2 Differentiation within the Mediterranean}
In addition to the strong differentiation observed between the Atlantic and Mediterranean basins, this study detected differentiation (albeit slightly weaker) in \textit{H. gammarus} within the Mediterranean Sea, separated into the middle (Sardinia and Lazio) and eastern Mediterranean (Aegean Sea).  A similar pattern was found in a previous study of \textit{H. gammarus}, whereby samples from the Aegean Sea were differentiated from one sample in the Adriatic Sea (mid-Mediterranean) and one sample from the Columbretes Islands (western Mediterranean) \citep{Triantafyllidis2005}.  In the present study, the stronger differentiation observed at neutral SNPs infers restrictions to gene flow; the Mantel test using only Mediterranean sites (although non-significant) (Fig. \ref{fig:lobster_IBD}d) indicated that IBD may be a key driver of this pattern, which suggests that \textit{H. gammarus} larval dispersal between the middle and eastern Mediterranean follows a stepping-stone model of connectivity. In addition, as clinal patterns were found in the STRUCTURE analysis using outlier SNPs, this could also suggest that secondary contact may explain some of the differentiation observed, potentially from adaptive introgression \citep{Riquet2017}.  In any case, the evidence for divergence via drift implies that, despite the generally high \textit{N}\textsubscript{e} expected in marine invertebrates, \textit{N}\textsubscript{e} across these spatial scales is not sufficiently large to mitigate drift.  This may be due to the present-day lower abundance and patchy distribution of \textit{H. gammarus} in the Mediterranean (resulting from past over-exploitation), which is also supported by the overall lower genetic diversity (possibly due to bottlenecks) found in the Mediterranean sites in this study.  

\subsubsection{6.4.3 Northeast Atlantic connectivity}
In the northeast Atlantic, there appeared to be a genetic cline in the datasets that used all SNPs and outlier SNPs (and in the neutral SNPs using STRUCTURE \textit{K}=3), starting from the most southerly sampling site (Vigo, northwest Spain) to the most northerly sampling sites (Flo-Gul-Kav-Lys, Skagerrak).  Interestingly, however, this pattern was considerably weaker in the DAPC using neutral SNPs; instead, most northeast Atlantic sites generally clustered together, with the exception of Oosterschelde which was partially differentiated from the main cluster of Atlantic samples.  

Mantel tests conducted with only Atlantic sites (Fig. \ref{fig:lobster_IBD}b) and neutral SNPs found that geographical distances explain 17 \% of the variation in the \textit{F}\textsubscript{st} dissimilarity matrix, although this increased to 49 \% when Oosterschelde was removed (Fig. \ref{fig:lobster_IBD}c).  This suggests that IBD likely explains a considerable component of the genetic cline observed, which is also supported by the biophysical modelling as the dispersal trajectories and connectivity matrices generally imply that seascape hydrology (i.e. ocean currents) facilitates a stepping-stone model of connectivity between the study sites (Fig. \replaced{\ref{fig:lobster_dispersal}, \ref{fig:lobster_connectivity}}{modelling}).  Nevertheless, assuming that IBD is indeed involved in driving some of this genetic cline, over 50 \% of the variation in the \textit{F}\textsubscript{st} dissimilarity matrix (and 72 \% in the RDA using all sites) remains unexplained, which suggests other processes are also responsible for shaping this genetic cline. 

As alluded to previously, the most commonly proposed causes of clinal patterns in allele frequencies are (i) IBD caused by neutral drift, (ii) selection across an environment gradient, and (iii) secondary contact and introgression between previously isolated and genetically divergent populations \citep{Perez-Losada2002}.  In this study, Mantel tests with neutral SNPs provided support for IBD.  However, fine-scale local adaptation across an environmental gradient (e.g. temperature), similar to the explanation proposed for the differentiation between the Atlantic and Mediterranean basins, cannot be ruled out as a driver of this pattern.  This is supported by analysis of the 15 outlier loci, which showed a very clear genetic cline in both the DAPC and STRUCTURE analyses.  Yet, without the incorporation of sea temperature data, and further genomic resources that have reliable gene annotations (e.g. whole genome or transcriptome), local adaptation to sea temperatures and the mechanisms behind this process are speculative.  Alternatively, this clinal pattern could be explained by secondary contact following range expansions from refugia after the Last Glacial Maximum (or more ancient glaciations in the Pleistocene).  Putative refugia in the northeast Atlantic have been proposed in western France, the Iberian Peninsula, and west-southwest Britain and Ireland \citep{Maggs2008,Finnegan2013,Jenkins2018a}, evidenced by the high levels of genetic diversity found in populations inhabiting these areas \citep{Provan2008}.  Given that the highest genetic diversity in this study was found in the Bay of Biscay (Île de Ré) and northwest Spain (Vigo), it is possible these two sites served as glacial refugia, which preceded secondary introgression of northward dispersers after the ice retreated. 

Mantel tests in this study also revealed a reduced correlation (and a substantial decrease in significance) when lobsters from Oosterschelde were excluded from the analysis. Moreover, using neutral SNPs, there was evidence that the Oosterschelde sample was genetically differentiated from lobsters at all other Atlantic sites.  This pattern has been reported in a previous study on \textit{H. gammarus} using mtDNA RFLPs \citep{Triantafyllidis2005}; the authors attributed this differentiation in Oosterschelde to a combination of past bottlenecks and a lack of immigration. In 1962-1963, harsh winters brought extremely low water temperatures and salinities (from high localised river discharges) to Oosterschelde, causing mass mortality of lobsters and many other marine organisms \citep{Triantafyllidis2005}.  Moreover, construction of dams over the last century has created a semi-enclosed area virtually isolated from the North Sea \citep{Triantafyllidis2005}.  The differentiation and lower diversity of lobsters from Oosterschelde found in this SNP study are in line with the conclusions by \citet{Triantafyllidis2005}, that is, this pattern is likely a result of drift caused by past bottlenecks and a barrier to gene flow over the last century.

At much finer spatial scales (i.e. within and between neighbouring seas), and taking the genetic cline into account, the results from this study suggest high genetic connectivity between certain sites sampled in the northeast Atlantic.  For example, high gene flow was apparent within and between sites situated in the English Channel, the Celtic and Irish seas, the coast of western Ireland and up to northern Scotland (up to distances of 1,400 km); this was supported by the relatively low pairwise differentiation indices (\textit{F}\textsubscript{st} and \textit{D}) and the population structure analyses.  This was also supported, but to a lesser extent, by the biological modelling simulations. Although the non-bio dispersal trajectories and connectivity matrices indicated extensive dispersal potential across these scales, this is not likely to be realistic, or as representative of real-world transience as the bio projections of dispersal and connectivity.  When biologically realistic dynamic parameters were factored into the model, the results showed significantly less dispersal potential, with practically no dispersal from sites north of Scotland (Ork, Sul and She), where larvae encountered a median temperature of $<$14\textsuperscript{o}C for more than half of their drift time which meant they were considered dead.  However, these areas host lobster stocks which have long supported intensive commercial fisheries, into which recruitment must occur somehow. \citet{Quinn2013} showed that, when reared in cold water (10\textsuperscript{o}C), larval development times among \textit{H. americanus} clutches sourced from females at the northern extent of the range were significantly reduced compared to those of clutches sourced from more southerly latitudes. The authors interpreted this, and results showing the reverse trend when cold-water larvae were reared at higher temperatures, as a signal of local adaptation to their environment, and it seems plausible that \textit{H. gammarus} larvae originating from the northerly areas sampled in this study have undergone similar adaptations to their thermal niche. If this were the case, then the dispersal potential of larvae from northerly areas may well have been underestimated by our biological parameters, which were all based on the development of larvae sourced from females inhabiting warmer waters to the south (i.e. Helgoland and southwest England) \citep{Schmalenbach2010}.  

High genetic connectivity at similar fine-scales were also found in other regions; for example, between sites within the Skagerrak region, between sites in the middle Mediterranean, and between sites within the Aegean Sea.  For Skagerrak, this was supported by both (bio and non-bio) simulations of larval dispersal, which implies that connectivity between sites in Skagerrak can occur in a single dispersive event at distances of up to 168 km (Flo-Sin).  Moreover, these simulations suggest that larvae can disperse to sites in western Norway; though, previous studies have found northern Norway populations to be genetically distinct \citep{Triantafyllidis2005,Jorstad2005}, which indicates that sites in northern Norway may be more isolated than the simulations in this study suggest. The genetic isolation of northern Norway could not be tested in this SNP study; nevertheless, based on these previous studies, it appears that gene flow between northern Norway and other sites in the northeast Atlantic may be restricted or that selective forces occurring at the northern range edge may be mitigating the homogenising effect of gene flow.  In the Mediterranean, the high genetic similarity of sites (i) within the middle Mediterranean and (ii) within the Aegean Sea, suggests that genetic connectivity can potentially occur up to distances of 294 km (between Sardinia and Lazio) and 280 km (between Alexandroupoli and Thermaikos Bay), respectively.  However, because the spatial extent of the ocean model did not facilitate simulations of larval dispersal across the Mediterranean, it is difficult to ascertain whether these patterns are due to high gene flow or due to high \textit{N}\textsubscript{e} (or due to both).  Alternatively, the high genetic similarity between sampling sites that are situated closely together (e.g. sites within Skagerrak) may suggest that these sites are a single panmictic population.

\subsubsection{6.4.4 Implications for fisheries management and stock enhancement}
For managing fisheries, it is important to identify stock structure and connectivity to ensure that the spatial implementation of management is commensurate with that of biological population units \citep{Reiss2009}, and to pinpoint populations that may contribute colonisers to overfished or depleted stocks \citep{DaSilva2015}.  The latter typically relies on estimating demographic connectivity, which is difficult to measure, particularly with genetic data alone \citep{Lowe2010}.  In the northeast Atlantic, analysis of neutral SNPs and biophysical modelling from this study provided evidence for a stepping-stone model of connectivity, in which larvae have the potential to disperse up to 300 km from some coastal sites of the British Isles (Fig. \ref{fig:lobster_dispersal}).  This implies that site-specific recruitment may not always come from local sources, but from adjacent local or regional sources.  Thus, if multiple adjacent (LFU) stocks across Britain and Ireland collapsed simultaneously, this could have profound ramifications for stock recovery and local fisheries.  Recent research has found that temporary closures or prohibiting fishing in MPAs offers some respite to lobster populations \citep{Moland2013,Roach2018}, which may be a viable management option for lobster fisheries going forward to prevent over-exploitation. However, safeguarding lobster stocks in one area via temporary closures or MPAs (e.g. Isles of Scilly) may inadvertently benefit local and regional areas adjacent to the protected area, due to an increase in larval subsidy, rather than the area being protected.  Therefore, fisheries management should acknowledge these patterns of dispersal and recruitment when considering which source populations to protect during temporary closures and for designing holistic management programmes.

For lobster hatcheries, knowledge of stock structure is crucial to ensure that reared juveniles, which are usually reared from the egg clutches of wild females originating from the local area \citep{Ellis2015b}, demonstrate evolutionary compatibility with the targeted population being stocked \citep{Ward2006}.  Overall, the genetic profiles observed in this study using neutral and outlier SNPs suggests that stocking of a target population should ideally be implemented with juveniles whose parents originate from the same geographical area.  Furthermore, the use of broodstock originating from the northeast Atlantic to stock target populations in the Mediterranean, or vice versa, is to be highly discouraged; this also applies to broodstock originating from the middle Mediterranean with the aim to stock target populations in the Aegean Sea (and vice versa).  This is because of the potential to introduce maladapted traits into the target population that could also proliferate to neighbouring populations \citep{Araki2007}; this has the most deleterious consequences in stocks that are depleted or are highly adapted to local conditions \citep{Lorenzen2012a}.  In this study, a significant excess of heterozygotes were found at several sites, which may indicate the presence of heterozygote advantage at some loci \citep{Sellis2011}, or outbreeding depression, which can be caused by stocking a target population with individuals adapted to a completely different environment \citep{Frankham2011}.  For example, three of the five Scandinavian sites in Skagerrak showed this pattern, of which Kvitsoy (southwest Norway) $\sim$225 km away from Flodevigen has seen quite extensive stocking during 1990 and 1994 \citep{Agnalt2004,Ellis2015b}.  Although this result is potentially evidence for outbreeding depression at these Scandinavian sites, most of the broodstock used in the stock enhancement programme were reported to originate from Kvitsoy \citep{Agnalt2004}.

Individual assignment using genetic techniques has been shown to be a potentially useful tool for determining the origin of fished individuals and for tackling IUU fishing  \citep{Martinsohn2009,Nielsen2012,Bernatchez2017}. However, the power of the molecular markers employed is highly sensitive to the degree of genetic differentiation between sites \citep{Christie2017}.  This study demonstrated that a panel of 86 SNPs has adequate power to assign lobsters to either the Atlantic or Mediterranean basin at 100 \% accuracy.  This may have useful applications for management authorities, particularly those responsible for Mediterranean coasts, who wish to find out whether an individual lobster has been imported from somewhere in the northeast Atlantic or has been locally caught in the Mediterranean.  Moreover, managers could test for the introduction of Atlantic lobsters into the Mediterranean via escapees or incidental larval release (i.e. from storage facilities on the coast), or from inadvisable attempts to boost stocks through the intentional release of larvae, juveniles or adult \textit{H. gammarus} that originate from Atlantic populations.  Unfortunately, it is not currently possible to assign lobsters back to their location of origin using this SNP panel; however, it may be possible to assign lobsters with some confidence to a geographical region (e.g. Skagerrak).  The power and accuracy of these assignment tests at local and regional scales could be improved by incorporating more SNP markers and by using software that take patterns of IBD into account \citep[e.g.][]{Guillot2016,Drinan2018}.


\subsubsection{6.4.5 Limitations and future research}
The drivers of some of the genetic patterns found in this study, such as secondary contact, could be further explored with samples from sites in southern European Atlantic waters (i.e. southern Portugal and Spain) and Atlantic Morocco, and from sites in the western Mediterranean (i.e. Alboran Sea).  Moreover, although analysis of outlier SNPs in this study was linked to differentiation at the southern range limit (Aegean Sea), samples from northern Norway would be advantageous to explore the potential for local adaptation to sea temperatures at both the northern and the southern range limit of \textit{H. gammarus}.  Future research could also collate sea temperature (and salinity) data to test whether these data explain any of the variation in allele frequencies; such an approach was implemented in the American lobster to explore thermal adaptation \citep{Benestan2016b}.

Analysis to detect outlier SNPs in this study was stringent; yet, as the panel was composed of a subset of informative SNPs derived from RADseq data, there is a possibility some of these outlier SNPs are false-positives.  However, 13 out of 15 of the SNPs classified as outliers in this study were also classed as outliers in the raw RADseq dataset (Chapter 5), so it is likely that these 13 SNPs are genuine outlier markers potentially under divergent selection.  Moreover, although two outliers were not detected in the original RADseq dataset, both SNPs were still classed as outliers in this study because they could represent adaptive loci associated with sampling sites which were included in this study but not the RADseq study. All 15 SNPs could be further validated as outliers by aligning the raw RAD-tag loci to the American lobster or European lobster transcriptome, although at present only the American lobster transcriptome is publicly available.  \added{Additionally, the population allele frequency of one allele for each of the 15 outlier SNPs was visualised (Fig. \ref{fig:lobster_allele_freq}). This showed some interesting differences in the frequency of the allele investigated, with several SNPs completely fixed for one allele in some sampling sites from the Atlantic, the middle Mediterranean or the eastern Mediterranean. These loci could act as diagnostic SNPs that differentiate populations from the Atlantic, the middle Mediterranean and the eastern Mediterranean; moreover, these SNP loci may also be useful for future research to estimate admixture and migration.}

\added{
Interpreting genetic data using STRUCTURE analysis in the presence of IBD or hierarchical structuring can be challenging \citep{Frantz2009, Gilbert2016}. The STRUCTURE admixture model assumes that each of the \textit{K} ancestral populations existed at some point in the past and that modern individuals were produced by recent mixing (or no mixing) of these ancestral populations \citep{Lawson2018}. Incorrect inferences of \textit{K} may arise when hierarchical levels of population structure exist (subpopulations within populations) and there is uneven sampling across these levels \citep{Puechmaille2016}. However, the sampling regime in this study was robust, with regularly distributed samping locations in the northeast Atlantic and with the majority of sampling sites containing approximately 35-37 individuals; this mitigates the potential caveat of uneven sampling. This study also found strong spatial autocorrelation across the study area, which may limit the ability of STRUCTURE to accurately delimit clusters and assign individuals to ancestral populations.  For example, \citet{Frantz2009} showed that Bayesian clustering methods can overestimate genetic structure when analysing genetic data characterised by IBD (or clines of genetic variation across a landscape). This is a potential limitation of the underlying model when applied to these scenarios, in which the inferred value of \textit{K} and the corresponding allele frequencies in each cluster can sometimes be arbitrary, a limitation acknowledged by the authors of STRUCTURE \citep{Perez2018}. Although these caveats should be considered in this SNP study, the STRUCTURE results were consistent with other population structure analyses (e.g. genetic indices and DAPC). However, as IBD appears to be a characteristic of this dataset, future analyses such as PCA and tree-building based on unbiased genetic distances may be more appropriate to avoid such caveats.}


\subsubsection{6.4.6 Conclusion}
In conclusion, the results of this SNP study indicate that several factors have potentially contributed to shaping the genetic structure of \textit{H. gammarus} across its northeast Atlantic and Mediterranean distribution, including IBD, oceanographic barriers and secondary contact after a period of allopatric isolation.  How these patterns, and lobsters in general, will respond to projected increases in sea temperature with global climate change is uncertain.  Indeed, as the PLD of \textit{H. gammarus} is temperature-dependent \citep{Schmalenbach2010}, elevated sea temperatures could see future range expansions northward, perhaps into the Faroe Islands and Iceland; such a pattern of expansion has been suggested in the green crab (\textit{Carcinus maenas}) \citep{Roman2004}, whereby Shetland has potentially acted as a stepping-stone for range expansion.  In contrast, warming temperatures may act to decrease the PLD (possibly reducing dispersal distance and connectivity) and potentially increase larval mortality in the eastern Mediterranean where sea temperatures across the range of \textit{H. gammarus} are highest \citep{Schmalenbach2010}.  Consequently, managing and monitoring lobster stocks effectively, and preserving the distribution of genetic diversity across the range of \textit{H. gammarus}, will likely be critical in ensuring the future persistence and sustainability of lobster populations and the fisheries they support.


%--------------
%
% Chapter 7: General discussion
% 
%--------------
\newpage
\section{Chapter 7: General discussion}
\rule{\textwidth}{0.8pt}
\vspace{1pt}

\noindent This thesis has addressed three research components: (i) exploration of comparative phylogeography in a number of diverse taxa across the northeast Atlantic using meta-analysis; (ii) proposed criteria for selecting candidate species to assess connectivity between MPAs; and (iii) generated novel population genetic data for two benthic marine organisms, the pink sea fan and the European lobster, which has provided novel evidence to address connectivity between MPAs.  In this final chapter, the results of this thesis are summarised, followed by a discussion on the translation of genetic data into policy and speculation on how future developments may enhance studies of marine connectivity. 

\subsection{7.1 Summary and addressing research hypotheses}
The meta-analysis of coastal marine taxa (Chapter 2) represents one of the first meta-data studies to investigate comparative phylogeography in the northeast Atlantic; the other meta-analysis study in the north Atlantic explored the possibility of distinguishing signatures of periglacial refugia from southern refugia \citep{Maggs2008}.  The inclusion of many diverse taxa in the present meta-analysis allowed two main hypotheses to be tested: (i) whether there are common phylogeographic breaks among taxa; and (ii) whether demography was constant or variable among coastal species during the Pleistocene glaciations, and in particular during and after the LGM.  The results showed that marine taxa in the northeast Atlantic show a mixture of contemporary genealogical structure and that patterns of phylogeography are discordant, which supports the null hypothesis of contrasting patterns of phylogeography.  This finding generally accords with similar meta-analyses across other seas/oceans, which also report contrasting patterns of phylogeography across the southeast coast of the United States of America \citep{Pelc2009}, the northeast Pacific \citep{Marko2010}, and the northwest Pacific \citep{Ni2014}.   The results in the meta-analysis from this thesis also indicated that population expansions were common in northeast Atlantic coastal taxa and were mostly linked to the LGM, providing evidence to reject the null hypothesis and accept the alternative hypothesis; this agrees with the findings for five species studied by \citet{Marko2010} but is contrasting to the findings of \citet{Ni2014} in which the authors reported that most population expansions pre-dated the LGM.

Selection of candidate species for assessing genetic connectivity between MPAs (Chapter 3) was tailored towards species whose distribution spanned MPA boundaries designated in UK waters.  However, the criteria proposed to select taxa in this thesis could easily be applied to assessments of MPA network connectivity in other seas and oceans around the world.  Although no hypotheses were tested in this chapter, this research component was essential to this thesis because it facilitated the selection of appropriate taxa for addressing MPA connectivity across the British network, particularly across the English and Welsh network.  

To explore the population genetics and connectivity of the pink sea fan and the European lobster, genomic techniques that utilise NGS technology were employed; this is the first time RADseq or a similar approach has been used to isolate genome-wide SNPs in either of these two non-model organisms.  For pink sea fan, there was distinct population structure using both microsatellite and SNP markers, organised into sites from Britain-France, southern Portugal and northwest Ireland, with some evidence for weak differentiation between samples from Britain and France.  Therefore, the null hypothesis can be rejected and the second alternative hypothesis (H\textsubscript{1B}) can be accepted.  In addition, the results suggested that the MPA network in southwest Britain is likely sufficient for maintaining genetic connectivity in pink sea fans; thus, the null hypothesis can also be rejected and the first alternative hypothesis (H\textsubscript{1A}) can be accepted.  For European lobster, there was strong differentiation between lobsters from the northeast Atlantic, the middle Mediterranean and the eastern Mediterranean; therefore, the null hypothesis of no population structure can be rejected. In the northeast Atlantic, a genetic cline from northwest Spain to Skagerrak was evident, which could be explained by one or more processes such as IBD, fine-scale local adaptation and secondary contact. Consequently, depending on which sites are being compared, this provides potential support for the first alternative hypothesis (i.e. between sites spatially close together) and for the second alternative hypothesis (i.e. between sites spatially further away).  Moreover, further analysis of neutral SNPs suggests that Oosterschelde is genetically isolated from other sites in the northeast Atlantic, which supports a hypothesis of population structure driven by reduced gene flow and subsequent drift.  The connectivity matrix based on the simulation model that incorporated biological parameters suggested that larval dispersal can potentially occur at spatial scales up to hundreds of kilometres, which was supported by overall genetic homogeneity at these scales. Therefore, based on the recommended placement of a MPA every 80 km or less by \citet{Roberts2010}, the results for European lobster from this study suggest that the MPA network across England and Wales is likely sufficient to maintain genetic connectivity, but potentially also demographic connectivity.


\subsection{7.2 Assessing connectivity between MPAs: which taxa?}
Connectivity is identified as one of five key principles for designing an ecologically coherent network of MPAs in European waters \citep{OSPARCommission2017}. Yet, without the ability to demonstrate connectivity, it is impossible to be certain that sites designated within a MPA network do in fact constitute a network, when they may be in reality a set of unlinked habitats and associated species assemblages.  In this thesis, a set of criteria were developed to pinpoint certain taxa that may represent ideal surrogates for empirically assessing connectivity between and outside of MPA boundaries.  Particular attention was afforded to taxa which may fulfil the criteria of an umbrella, keystone or flagship species \citep{Simberloff1998,Kalinkat2017}.  For example, connectivity patterns observed in one species may be representative of other species with similar biology and dispersal capacity (i.e. similar PLDs); consequently, when species assemblages across an area of interest are well documented, designing a network based on known connectivity patterns from one species may extrapolate benefits to other organisms in the community \citep{Marti-Puig2013}.  As the biology and dispersal of European lobster is well understood, such an approach could be used to fill in gaps in the current MPA network around Britain.  However, species whose dispersal traits are poorly understood should not necessarily be dismissed, particularly if the species has high local or national conservation priority. These species may be advantageous for driving MPA proposals and designations that encompass populations of these species; pink sea fans in southwest Britain is such an example, whereby several MCZs have been specially designated to protect populations of pink sea fan, some of which may have added benefits to other (non-protected) species in the MCZs with similar life histories and dispersal traits \citep[e.g. dead man's fingers,][]{Holland2017}.  

\subsection{7.3 Translating genetic data into policy}
Content discussed in this section is based on a paper published in the journal \textit{Marine Policy} \citep[section 3,][]{Jenkins2018b}. A central premise of this thesis is to use the novel genetic data generated to inform practitioners in the fields of marine protected area design and fisheries management.  However, translating primary research into the language and terminology required by policymakers and conservation managers is not a trivial task.  Often, it may be more beneficial to present a few points that represent the key findings of a study, while trying to avoid unnecessary technical jargon, which could lead to misinterpretation or confusion.  Several papers have discussed the challenges of translating genetic data to inform management and have asserted the importance of strong collaboration and communication between scientists and practitioners \citep{Laikre2010,Gordon2014,Shafer2015,Galla2016,Garner2016,Hogg2017,Taylor2017}.  Some of the reasons put forward for the avoidance of genetic data in fisheries management include a lack of understanding of the potential value of genetic data, the assumption that genetic studies are expensive, and the suggestions that other data types are significantly more important than genetic information in management decisions \citep{Bernatchez2017}. One feature of genetic data is that they cannot be seen or measured without the use of specialist molecular techniques, meaning it can sometimes be difficult to articulate the level of variation and the importance of genetic diversity to non-scientists \citep{Laikre2010}. Moreover, in cases where research is carried out by non-academic bodies, these institutions often have little incentive to publish, or have internal deadlines or political/legal constraints that may delay scientific publication, so the findings may not be widely disseminated \citep{Garner2016}.

However, while some barriers to the dissemination of genetic research exist, there are examples across various taxa and systems where genetic data have successfully informed policy and conservation, and have led to improved management decisions.  This suggests that some barriers to the application of genetic data are starting to be overcome.  Some examples include the genetic restoration of Florida panthers \citep{Johnson2010}, the genetic management of salmonids \citep{Baerwald2011,Habicht2012}, the authenticity and monitoring of seafood in sushi bars \citep{Vandamme2016}, and the traceability of fisheries resources \citep[e.g. FishPopTrace,][]{Martinsohn2009}.

Yet, while there are a myriad of studies documenting the spatial genetic structure and genetic connectivity of benthic marine species, very few of these studies have been directly used as evidence to inform or support MPA designations and/or network connectivity.  This may be a consequence of ineffective dissemination of the key findings of research projects, but also likely relates to the availability of data at the time when large-scale MPA projects were commissioned and candidate lists were first drawn-up.  Nevertheless, as these data are becoming more available to practitioners, it is crucial that gaps between primary research (i.e. academic researchers) and applied science (i.e. policymakers) are overcome in order to realise the potential of genetic data to inform MPA design and conservation planning \citep{Shafer2015,Garner2016}.  

Genetic data are currently not (to the authors' knowledge) used by managers as evidence to inform MPA designation or network connectivity in England and Wales.  Discussions with national agencies suggest that the personnel and infrastructure are not in place to process, grade and assess the usefulness of spatially relevant genetic data \citep{Jenkins2018b}.  This may explain the lack of genetic data currently used as evidence to support existing MPA designation or to inform new designations around southwest Britain \citep{Jenkins2018b}.  However, genetic data from single-species can provide an estimate of realised connectivity within evolutionary timescales and, combined with biophysical modelling, these data would likely supplement the present methods used to assess network connectivity in Britain (section 1.3.4).  Moreover, genetic data can reveal distinct localised genetic diversity, otherwise undetectable using only presence/absence data or modelling, which can be of major importance for identifying populations or areas that should be prioritised for protection \citep{Funk2012}. 

The pink sea fan microsatellite study \citep{Holland2017} and the SNP study in this thesis has the potential to inform and support the designations of MPAs that include \textit{E. verrucosa} as a protected feature. The key finding from these studies which might constitute evidence for MPA manager is that, as it stands, the MPA network in southwest Britain appears to be adequate to maintain genetic connectivity in this protected species.  This also appears to be the case for European lobster, for which the MPA network may be able to maintain both genetic and demographic connectivity.  The integration of these data in future reviews or monitoring reports would likely serve as another piece of evidence to support the designation of these MPAs and to help demonstrate the ecological coherency of the network in southwest Britain. 

Although genetic data have much promise for informing marine conservation and fisheries management, there are some general limitations.  Firstly, managers are typically interested in demographic connectivity, which is difficult to quantify using genetic data, unless combined with other data such as population growth rates, reproductive success or biophysical modelling \citep{Lowe2010,Breusing2016}.  Moreover, as few as ten effective immigrants per generation may be sufficient to maintain genetic homogeneity (drift connectivity) \citep{Lowe2010}, meaning that, despite being genetically similar, some populations may have minimal larval exchange \citep{Botsford2009}.  However, genetic markers provide insights into realised connectivity, which may be useful for managers who are interested in the contribution of immigrants that survive, reproduce and add to the local gene pool.  Secondly, these approaches usually assume that populations are at gene flow-drift equilibrium; deviation from this assumption can lead to over-estimating the amount of contemporary gene flow \citep{Lowe2010}.  For example, inferring patterns of connectivity from marine species with overlapping generations or long-life spans (e.g. corals) can be challenging because genetic profiles can remain essentially unchanged for many decades, even after barriers to gene flow are introduced. Therefore, in some cases, interpretations of genetic homogeneity may represent historical and not contemporary gene flow and, vice versa, distinct population structure may represent historical and not present-day isolation \citep{Hedgecock2007}.  This difference in timescales is critical to consider in assessments of connectivity because MPA networks are generally established to protect and maintain present-day and future patterns of biodiversity and connectivity, or to facilitate recovery/restoration to a previous level of abundance and diversity \citep{Jenkins2018b}.

    
\subsection{7.4 Future developments in marine connectivity}
For exploring marine connectivity in benthic organisms with a pelagic larval phase there are three areas that may see pronounced developments in the coming decades: population genetics, biophysical modelling, and real-time tracking.  Genomics has begun to revolutionise conservation genetics \citep{Allendorf2010,Funk2018} and is likely to continue over the next decade; this will likely have benefits for marine connectivity by providing more accurate measures of population genetic parameters (e.g. gene flow) and by providing higher power for assignment approaches.  Of course, in the near future we may have complete genomes sequenced for thousands of species \citep{Allendorf2010,Fuentes-Pardo2017}, which could allow researchers to directly compare whole genomes of their study species.  In addition, future advances in computer power and memory may generate very high-resolution (i.e. $<$50 m) ocean circulation models; this may enable the development of IBMs capable of tracking of larvae at extremely fine-scales.  Lastly, although the technology does not yet exist, methods may emerge that allow researchers to track pelagic larvae in real-time from eggs to settlement; however, there is currently no evidence to suggest this is or will be feasible.  In any case, the development of molecular techniques and more powerful ocean models is almost certain, and will likely give unprecedented resolution into the marine connectivity of benthic marine organisms. 


%--------------
%
% Appendix
% 
%--------------
\newpage
\section{Appendix}

%%%% Chapter 2
\subsection{Chapter two}
\subsubsection{A1: Mutation rates}
Mutation rates used for each species are freely available from the supplementary material hosted online by \textit{PeerJ} (https://doi.org/10.7717/peerj.5684).
\subsubsection{A2: Haplotype networks for all species}
Haplotype networks for all species are freely available from the supplementary material hosted online by \textit{PeerJ} (https://doi.org/10.7717/peerj.5684).
\subsubsection{A3: Mismatch graphs for all species / lineages}
Mismatch graphs for all species are freely available from the supplementary material hosted online by \textit{PeerJ} (https://doi.org/10.7717/peerj.5684).

%%%% Chapter 4

%% DNA extraction protocol
% \subsubsection{A4: Detailed salting-out DNA extraction protocol}
\includepdf[pages=1,scale=0.9,pagecommand={\subsection{Chapter four} \subsubsection{A4: Detailed salting-out DNA extraction protocol}  \thispagestyle{plain}}]{Chapter_figures/Salting_Out_DNA_Extraction_Protocol.pdf}
\includepdf[pages=2-,scale=0.9,pagecommand={\thispagestyle{plain}}]{Chapter_figures/Salting_Out_DNA_Extraction_Protocol.pdf}

%% Pink sea fan gel images
\subsubsection{A5: Pink sea fan gel images}
\includegraphics[width=0.495\textwidth, height=5cm]{Chapter_figures/chapter4/Qiagen_vs_salt.jpg}
\includegraphics[width=0.495\textwidth, height=5cm]{Chapter_figures/chapter4/gel1.png}
\includegraphics[width=0.495\textwidth, height=5cm]{Chapter_figures/chapter4/gel2.png}
\includegraphics[width=0.495\textwidth, height=5cm]{Chapter_figures/chapter4/gel3.png}

\noindent Pink sea fan DNA run on a 1 \% agarose gel and stained with ethidium bromide. The top-left image compares the performance of the Qiagen Blood and Tissue kit and the salting-out protocol; the other three images show DNA samples extracted using the salting-out protocol.}

%% DAPC 8 random SNPs
\subsubsection{A6: Pink sea fan DAPC using eight random SNPs}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter4/Seafan_dapc_randomsnps1.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter4/Seafan_dapc_randomsnps2.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter4/Seafan_dapc_randomsnps3.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter4/Seafan_dapc_randomsnps4.png}

\noindent Pink sea fan discriminant analysis of principle components (DAPC) using eight randomly selected SNPs.


%%%% Chapter 6
\newpage
\subsection{Chapter six}
\subsubsection{A7: European lobster SNP dataset: interpreting \textit{K}}
\includegraphics[width=0.5\textwidth]{Chapter_figures/chapter6/evanno_allSNPs_38pops.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/evanno_71neutral_38pops.png}
\includegraphics[width=0.495\textwidth]{Chapter_figures/chapter6/evanno_15outliers_38pops.png}
\noindent Interpreting \textit{K} for the European lobster SNP dataset using (A) mean L(\textit{K}), (B) L$'$(\textit{K}), (C) L$''$(\textit{K}), (D) delta \textit{K}. Each plot represents analysis with all 86 SNPs (top), analysis with 71 putatively neutral SNPs (bottom-left), and analysis with 15 outlier SNPs (bottom-right).
\clearpage

%--------------
%
% Papers
% 
%--------------

% Increase page width and length
% \newgeometry{bottom=25mm}

% PeerJ 2018 paper
\phantomsection
\addcontentsline{toc}{subsection}{Research paper: \textit{PeerJ} 2018}
\includepdf[pages=1,scale=0.8]{Papers/Jenkins_2018_PeerJ.pdf}
\includepdf[pages=2-,scale=0.8,pagecommand={\thispagestyle{plain}}]{Papers/Jenkins_2018_PeerJ.pdf}


% Marine Policy 2018 paper
\phantomsection
\addcontentsline{toc}{subsection}{Research paper: \textit{Marine Policy} 2018}
\includepdf[pages=1,scale=0.8]{Papers/MarinePolicy2018.pdf}
\includepdf[pages=2-,scale=0.8,pagecommand={\thispagestyle{plain}}]{Papers/MarinePolicy2018.pdf}

% Heredity 2017 paper
\phantomsection
\addcontentsline{toc}{subsection}{Research paper: \textit{Heredity} 2017}
\includepdf[pages=1,scale=0.8]{Papers/Heredity_2017_Paper.pdf}
\includepdf[pages=2-,scale=0.8,pagecommand={\thispagestyle{plain}}]{Papers/Heredity_2017_Paper.pdf}

% Con Gen Res 2018 paper
\phantomsection
\addcontentsline{toc}{subsection}{Research paper: \textit{Conservation Genetics Resources} 2018}
\includepdf[pages=1,scale=0.8]{Papers/ConGenRes2018.pdf}
\includepdf[pages=2-,scale=0.8,pagecommand={\thispagestyle{plain}}]{Papers/ConGenRes2018.pdf}


% Restore default geometry
% \restoregeometry


%--------------
%
% Glossary
% 
%--------------
% \newpage
% \section{Glossary}
% A description of the most importance terms/concepts/software be go here. For example: Fst, coalescence theory, DAPC, STRUCTURE, HWE, LD, etc.


%--------------
%
% References
% 
%--------------
%% Three types: 
% \citep{Jones2010} = (Jones et al. 2010)
% \citet{Jones2010} = Jones et al. (2010)
% \citep[e.g.][]{Jones2010} = (e.g. Jones et al. 2010)

\small
\clearpage
\phantomsection
\addcontentsline{toc}{section}{References}
\singlespacing
\bibliographystyle{besjournals_mod}
\bibliography{Mendeley_refs_2018_10_24}

%=================
%
% End of thesis
%
%=================
\end{document}


